
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.4404d67389b1.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.484f3c634d37.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.859ea76d5b06.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs4.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Role for the Silencing Protein Dot1 in Meiotic Checkpoint Control - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939B27CEE34FF68500005A0C5708941C.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="mbc" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC15018/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC15018/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Molecular Biology of the Cell" /><meta name="citation_title" content="Role for the Silencing Protein Dot1 in Meiotic Checkpoint Control" /><meta name="citation_author" content="Pedro A. San-Segundo" /><meta name="citation_author_institution" content="Howard Hughes Medical Institute, Department of Molecular, Cellular and Developmental Biology and Department of Genetics, Yale University, New Haven, Connecticut 06520-8103" /><meta name="citation_author" content="G. Shirleen Roeder" /><meta name="citation_author_institution" content="Howard Hughes Medical Institute, Department of Molecular, Cellular and Developmental Biology and Department of Genetics, Yale University, New Haven, Connecticut 06520-8103" /><meta name="citation_publication_date" content="2000/10" /><meta name="citation_issue" content="10" /><meta name="citation_volume" content="11" /><meta name="citation_firstpage" content="3601" /><meta name="citation_doi" content="10.1091/mbc.11.10.3601" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC15018/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC15018/" /><meta name="citation_pmid" content="11029058" /><meta name="DC.Title" content="Role for the Silencing Protein Dot1 in Meiotic Checkpoint Control" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="American Society for Cell Biology" /><meta name="DC.Contributor" content="Pedro A. San-Segundo" /><meta name="DC.Contributor" content="G. Shirleen Roeder" /><meta name="DC.Date" content="2000 Oct" /><meta name="DC.Identifier" content="10.1091/mbc.11.10.3601" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Role for the Silencing Protein Dot1 in Meiotic Checkpoint Control" /><meta property="og:type" content="article" /><meta property="og:description" content="During the meiotic cell cycle, a surveillance mechanism called the&#10; &#x0201c;pachytene checkpoint&#x0201d; ensures proper chromosome segregation by&#10; preventing meiotic progression when recombination and chromosome&#10; synapsis are defective. The silencing ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC15018/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        The PMC website is updating on October 15, 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC15018/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/1/">Mol Biol Cell</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/451/">v.11(10); 2000 Oct</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC15018
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC15018/pdf/mk003601.pdf" class="int-view">PDF (1015K)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/15018/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/15018/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="15018" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC15018%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="1sNl94qPdQf0T5AA52St7B1dcYguBvNM0UqyFFUC3doxuFcIPnwvLAZHkEjPFYuV">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC15018%2F&amp;text=Role%20for%20the%20Silencing%20Protein%20Dot1%20in%20Meiotic%20Checkpoint%20Control" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC15018%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC15018/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC15018/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/11029058/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC15018/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/11029058/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC15018/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/15018/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/1/">Mol Biol Cell</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/451/">v.11(10); 2000 Oct</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC15018
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-mbc.gif" alt="Logo of mbc" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.molbiolcell.org/" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Mol Biol Cell.</span> 2000 Oct; 11(10): 3601–3615. </div><div class="part2">  <span class="doi"><span>doi: </span><a href="//doi.org/10.1091%2Fmbc.11.10.3601" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1091/mbc.11.10.3601</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC15018</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/11029058">11029058</a></div></div></div><h1 class="content-title">Role for the Silencing Protein Dot1 in Meiotic Checkpoint Control</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=San-Segundo%20PA%5BAuthor%5D" class="affpopup" co-rid="_co_idm139690866949216" co-class="co-affbox">Pedro A. San-Segundo</a><sup>&#x0002a;</sup><sup>&#x02020;</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Roeder%20GS%5BAuthor%5D" class="affpopup" co-rid="_co_idm139690866584256" co-class="co-affbox">G. Shirleen Roeder</a><sup>&#x0002a;</sup><sup>&#x02021;</sup><sup>&#x000a7;</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139690866949216"><h3 class="no_margin">Pedro A. San-Segundo</h3><p><sup>&#x0002a;</sup>Howard Hughes Medical Institute, Department of Molecular, Cellular
 and Developmental Biology and <sup>&#x02021;</sup>Department of Genetics,
 Yale University, New Haven, Connecticut 06520-8103</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=San-Segundo%20PA%5BAuthor%5D">Pedro A. San-Segundo</a></div></div><div id="_co_idm139690866584256"><h3 class="no_margin">G. Shirleen Roeder</h3><p><sup>&#x0002a;</sup>Howard Hughes Medical Institute, Department of Molecular, Cellular
 and Developmental Biology and <sup>&#x02021;</sup>Department of Genetics,
 Yale University, New Haven, Connecticut 06520-8103</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Roeder%20GS%5BAuthor%5D">G. Shirleen Roeder</a></div></div></div></div><div class="contrib-group half_rhythm fm-editor">Douglas Koshland, <span class="fm-role">Monitoring Editor</span></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm139690865177536_ai idm139690866949344_ai idm139690866756832_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139690865177536_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139690865177536_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm139690865177536_ai" style="display:none"><div class="fm-affl" id="N0x8a537a0.0x8cdc748"><sup>&#x0002a;</sup>Howard Hughes Medical Institute, Department of Molecular, Cellular
 and Developmental Biology and <sup>&#x02021;</sup>Department of Genetics,
 Yale University, New Haven, Connecticut 06520-8103</div><div id="FN151"><sup>&#x02020;</sup>Present address: Instituto de
 Microbiologia-Bioquimica CSIC/Universidad de Salamanca, Spain.</div><div id="FN153"><sup>&#x000a7;</sup>Corresponding author: E-mail address:
 <a href="mailto:dev@null" data-email="ude.elay@redeor.neelrihs" class="oemail">ude.elay@redeor.neelrihs</a>.</div></div><div class="fm-article-notes hide half_rhythm" id="idm139690865177536_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2000 May 10; Revised 2000 Jul 25; Accepted 2000 Jul 31.</div></div><div class="permissions half_rhythm hide" id="idm139690865177536_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  &#x000a9; 2000, The American Society for Cell Biology</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="abstract-a.p.b.q" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.p.b.qtitle" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p class="p p-first-last">During the meiotic cell cycle, a surveillance mechanism called the
 &#x0201c;pachytene checkpoint&#x0201d; ensures proper chromosome segregation by
 preventing meiotic progression when recombination and chromosome
 synapsis are defective. The silencing protein Dot1 (also known as Pch1)
 is required for checkpoint-mediated pachytene arrest of the
 <em>zip1</em> and <em>dmc1</em> mutants of
 <em>Saccharomyces cerevisiae</em>. In the absence of
 <em>DOT1</em>, the <em>zip1</em> and <em>dmc1</em>
 mutants inappropriately progress through meiosis, generating inviable
 meiotic products. Other components of the pachytene checkpoint include
 the nucleolar protein Pch2 and the heterochromatin component Sir2. In
 <em>dot1</em>, disruption of the checkpoint correlates with the
 loss of concentration of Pch2 and Sir2 in the nucleolus. In addition to
 its checkpoint function, Dot1 blocks the repair of meiotic
 double-strand breaks by a Rad54-dependent pathway of recombination
 between sister chromatids. In vegetative cells, mutation of
 <em>DOT1</em> results in delocalization of Sir3 from telomeres,
 accounting for the impaired telomeric silencing in <em>dot1</em>.</p></div></div><div id="sec-a.q.a" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.q.atitle" style="text-transform: uppercase;">INTRODUCTION</h2><p class="p p-first">During eukaryotic cell division, faithful transmission of genetic
 information is ensured by the operation of cell-cycle checkpoints.
 These surveillance mechanisms arrest or delay cell cycle progression in
 response to defects in cellular processes, thereby preventing the
 initiation of late events until earlier events have been successfully
 completed (<a href="#B30" rid="B30" class=" bibr popnode">Hartwell and Weinert, 1989</a>). In the mitotic cell cycle,
 several checkpoint controls have been characterized. For example, the
 &#x0201c;DNA damage checkpoint&#x0201d; responds to genome injuries by arresting
 the cell cycle at G1/S or G2/M, or slowing progression through S phase,
 thus allowing time to repair the damage and preventing the replication
 or segregation of damaged chromosomes (reviewed by <a href="#B68" rid="B68" class=" bibr popnode">Weinert, 1998</a>;
 <a href="#B36" rid="B36" class=" bibr popnode">Lowndes and Murguia, 2000</a>).</p><p>In <em>Saccharomyces cerevisiae</em>, several components of the DNA
 damage checkpoint have been identified. Rad9 and the Rad24 group of
 proteins (Rad24, Rad17, Mec3, and Ddc1) are thought to be involved in
 sensing damage and/or generating a signal in response to damage
 (reviewed by <a href="#B68" rid="B68" class=" bibr popnode">Weinert, 1998</a>; <a href="#B36" rid="B36" class=" bibr popnode">Lowndes and Murguia, 2000</a>). This signal is
 transduced through a cascade of protein kinases, including Mec1, Rad53,
 and Chk1, that act on downstream effectors, ultimately triggering cell
 cycle arrest (reviewed by <a href="#B68" rid="B68" class=" bibr popnode">Weinert, 1998</a>; <a href="#B36" rid="B36" class=" bibr popnode">Lowndes and Murguia, 2000</a>). A
 number of checkpoint proteins initially characterized in budding and
 fission yeasts have counterparts in mammals, demonstrating the
 conservation of these surveillance mechanisms (<a href="#B68" rid="B68" class=" bibr popnode">Weinert, 1998</a>).</p><p>Meiosis is a specialized form of cell division that generates haploid
 gametes from diploid parental cells in sexually reproducing organisms.
 Proper distribution of chromosomes to the haploid progeny depends on a
 coordinated series of interactions between homologous chromosomes
 (homologs) that occur during meiotic prophase, including reciprocal
 genetic exchange and the intimate association of homologs in the
 context of the synaptonemal complex (SC) (reviewed by <a href="#B51" rid="B51" class=" bibr popnode">Roeder, 1997</a>). As
 in the mitotic cell cycle, checkpoints operate in meiosis to ensure the
 accurate transmission of genetic information. The same checkpoint
 controls that arrest progression of the mitotic cell cycle in response
 to DNA damage, blocks in replication, or defects in spindle integrity
 also function in meiosis (reviewed by <a href="#B4" rid="B4" class=" bibr popnode">Bailis and Roeder, 2000a</a>). In
 addition, meiotic cells possess a surveillance mechanism called the
 &#x0201c;pachytene checkpoint&#x0201d; that monitors events specific to meiosis,
 such as recombination and chromosome synapsis (i.e., SC formation),
 that are critical for proper meiotic chromosome segregation (reviewed
 by <a href="#B51" rid="B51" class=" bibr popnode">Roeder, 1997</a>). In mutants of budding yeast, mouse, and
 <em>Caenorhabditis elegans</em> that are defective at intermediate
 steps in recombination and/or synapsis, the pachytene checkpoint
 triggers arrest at midmeiotic prophase (reviewed by <a href="#B4" rid="B4" class=" bibr popnode">Bailis and Roeder,
 2000a</a>). In the case of mouse and worm germ cells, arrest is followed by
 apoptotic death.</p><p>In yeast, meiotic recombination occurs concurrently with chromosome
 synapsis and is required for SC formation (reviewed by <a href="#B51" rid="B51" class=" bibr popnode">Roeder, 1997</a>).
 Recombination is initiated by double-strand breaks (DSBs) that occur
 before synapsis. The breaks are rapidly processed to expose
 single-stranded tails that then invade homologous sequences usually in
 a nonsister chromatid. Strand invasion is followed by repair synthesis
 and branch migration to form double Holliday junctions, around the time
 of SC formation. Mature recombinants are produced near the end of
 pachytene as the SC disassembles. During meiotic recombination, a
 number of intermediates are formed in which DNA molecules are not
 intact or are interlocked; thus, any attempt to segregate chromosomes
 before completion of recombination would be deleterious. The pachytene
 checkpoint prevents meiotic nuclear division in the presence of
 recombination intermediates (<a href="#B4" rid="B4" class=" bibr popnode">Bailis and Roeder, 2000a</a>).</p><p>In <em>S. cerevisiae</em>, <em>dmc1</em> and <em>zip1</em> are
 two well-characterized examples of mutants that undergo
 checkpoint-mediated arrest at pachytene. Dmc1 is a meiosis-specific
 homolog of the bacterial RecA strand-exchange enzyme (<a href="#B8" rid="B8" class=" bibr popnode">Bishop <em>et
 al.</em>, 1992</a>). In the <em>dmc1</em> mutant, synapsis is delayed and
 DSBs remain unrepaired (<a href="#B8" rid="B8" class=" bibr popnode">Bishop <em>et al.</em>, 1992</a>; <a href="#B50" rid="B50" class=" bibr popnode">Rockmill
 <em>et al.</em>, 1995</a>). Zip1 is a major structural component of the
 central region of the SC (<a href="#B62" rid="B62" class=" bibr popnode">Sym <em>et al.</em>, 1993</a>; <a href="#B64" rid="B64" class=" bibr popnode">Sym and Roeder,
 1995</a>; <a href="#B66" rid="B66" class=" bibr popnode">Tung and Roeder, 1998</a>; <a href="#B21" rid="B21" class=" bibr popnode">Dong and Roeder, 2000</a>). The
 <em>zip1</em> mutant arrests in meiosis with unsynapsed chromosomes
 and unresolved Holliday junctions (<a href="#B62" rid="B62" class=" bibr popnode">Sym <em>et al.</em>, 1993</a>;
 <a href="#B59" rid="B59" class=" bibr popnode">Storlazzi <em>et al.</em>, 1996</a>).</p><p>Several components of the pachytene checkpoint have been identified in
 <em>S. cerevisiae</em>. During meiotic recombination, DSBs and
 regions of single-stranded DNA occur; checkpoint proteins that
 respond to these types of lesions in vegetative cells also monitor
 recombination in meiosis. Thus, a subset of DNA damage checkpoint
 proteins, including Rad17, Rad24, Mec3, Ddc1, and Mec1, function in the
 pachytene checkpoint (<a href="#B38" rid="B38" class=" bibr popnode">Lydall <em>et al.</em>, 1996</a>; <a href="#B52" rid="B52" class=" bibr popnode">San-Segundo and
 Roeder, 1999</a>; Hudson, San-Segundo, and Roeder, unpublished data). The
 pachytene checkpoint also requires the SC-associated proteins Red1 and
 Mek1 (<a href="#B69" rid="B69" class=" bibr popnode">Xu <em>et al.</em>, 1997</a>; <a href="#B5" rid="B5" class=" bibr popnode">Bailis and Roeder, 2000b</a>; <a href="#B6" rid="B6" class=" bibr popnode">Bailis
 <em>et al.</em>, 2000</a>). Checkpoint-dependent cell-cycle arrest at
 pachytene is achieved by accumulation of hyperphosphorylated Swe1,
 which presumably inactivates Cdc28 (<a href="#B34" rid="B34" class=" bibr popnode">Leu and Roeder, 1999</a>), and by
 inhibiting the activity of the Ndt80 transcription factor, which is
 required for transcription of the <em>CLB1</em> gene encoding the
 major cyclin active at meiosis I (MI) (<a href="#B16" rid="B16" class=" bibr popnode">Chu and Herskowitz, 1998</a>;
 <a href="#B31" rid="B31" class=" bibr popnode">Hepworth <em>et al.</em>, 1998</a>).</p><p>A genetic screen for pachytene checkpoint mutants uncovered a
 meiosis-specific protein, Pch2, required for <em>zip1</em> meiotic
 arrest (<a href="#B52" rid="B52" class=" bibr popnode">San-Segundo and Roeder, 1999</a>). Pch2 localizes to the ribosomal
 DNA region (rDNA) and this nucleolar localization depends on the
 silencing factor Sir2, which is also necessary for pachytene checkpoint
 function. Silencing is a position-dependent, gene-independent form of
 repressed chromatin structure that affects large chromosomal domains.
 In yeast, three regions are subjected to silencing: the telomeres, the
 <em>HML/HMR</em> silent mating-type loci, and the rDNA array. Whereas
 telomeric and <em>HML/HMR</em> silencing involves the silent
 information regulators Sir2, Sir3, and Sir4, only the Sir2 protein is
 required for rDNA silencing (reviewed by <a href="#B37" rid="B37" class=" bibr popnode">Lustig, 1998</a>).</p><p class="p p-last">Presented here is the characterization of another protein, Dot1,
 identified in our screen for components of the pachytene checkpoint
 (<a href="#B52" rid="B52" class=" bibr popnode">San-Segundo and Roeder, 1999</a>). The <em>DOT1</em> gene was
 independently isolated in a screen for high-copy disruptors of
 telomeric silencing and shown also to affect <em>HML/HMR</em> and
 rDNA silencing (<a href="#B56" rid="B56" class=" bibr popnode">Singer <em>et al.</em>, 1998</a>), further implicating
 chromatin silencing in the pachytene checkpoint. In the absence of
 Dot1, the <em>zip1</em> and <em>dmc1</em> mutants fail to arrest;
 they proceed through meiosis and sporulation to produce inviable
 spores. In addition to its checkpoint function, Dot1 inhibits the
 operation of a Rad54-dependent intersister recombination pathway that
 repairs DSBs in the absence of Dmc1. The nucleolar Pch2 and Sir2
 proteins, as well as the telomeric Sir3 protein, are mislocalized in
 the absence of Dot1.</p></div><div id="sec-a.q.b" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.q.btitle" style="text-transform: uppercase;">MATERIALS AND METHODS</h2><div id="sec-a.q.b.b" class="sec sec-first"><h3 id="sec-a.q.b.btitle">Strains and Plasmids</h3><p class="p p-first-last">Yeast strain genotypes are listed in Table
 <a href="/pmc/articles/PMC15018/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table1.</span></span><span>1</span></a>. <em>DOT1</em> was cloned as an
 &#x0223c;6.5-kb <em>Eco</em>RI-<em>Bgl</em>II fragment from <em>S.
 cerevisiae</em> &#x003bb; clone 5513 (ATCC 70580) into the
 <em>Eco</em>RI-<em>Bam</em>HI sites of Bluescript
 SK<sup>&#x0002b;</sup> to generate pSS24. <em>DOT1</em> was
 disrupted by transformation with pSS30
 (<em>dot1::URA3</em>) or pSS44
 (<em>dot1::TRP1</em>). To generate the disruption plasmids,
 an &#x0223c;2.8-kb fragment containing <em>DOT1</em> was amplified by
 polymerase chain reaction (PCR) by using oligonucleotides ORF26&#x00023;1
 (5&#x02032;-GGG<span class="underline">GGATCC</span>AGGAACACTGAAGAACGGG-3&#x02032;) and ORF26&#x00023;2
 (5&#x02032;-GGG<span class="underline">GGGCCC</span>AGGTACCTGGTCCACGGCGC-3&#x02032;) as primers and DNA
 from &#x003bb; clone 5513 as template. <em>Bam</em>HI and <em>Apa</em>I
 sites added in the primers are underlined. The PCR fragment was cut
 with <em>Bam</em>HI and <em>Apa</em>I and cloned into the same
 sites of Bluescript SK<sup>&#x0002b;</sup> to generate pSS25. pSS30
 was constructed by replacing a <em>Hin</em>dIII fragment of pSS25
 containing most of the <em>DOT1</em> coding region (from nucleotides
 &#x02212; 414 to &#x0002b;1373) with a <em>Hin</em>dIII fragment (from pR1333)
 containing <em>URA3</em>. pSS44 was constructed by replacing the
 <em>Bst</em>BI-<em>Nhe</em>I fragment of pSS25 containing most of
 the <em>DOT1</em> coding region (from nucleotides &#x02212;109 to &#x0002b;1278)
 with a <em>Cla</em>I-<em>Spe</em>I fragment (from pSS43) containing
 <em>TRP1</em>. pSS43 was made by cloning an &#x0223c;0.85-kb
 <em>Eco</em>RI-<em>Bgl</em>II fragment containing <em>TRP1</em>
 (but not <em>ARS1</em>) from YRp7 (<a href="#B61" rid="B61" class=" bibr popnode">Struhl <em>et al.</em>, 1979</a>)
 between the <em>Eco</em>RI and <em>Bam</em>HI of Bluescript
 SK<sup>&#x0002b;</sup>. Both pSS30 and pSS44 were cut with
 <em>Bam</em>HI and <em>Kpn</em>I before transformation into yeast.
 Although <em>dot1</em> mutants have been shown to be defective in
 transcriptional silencing of a reporter gene inserted at the
 <em>HM</em> loci (<a href="#B56" rid="B56" class=" bibr popnode">Singer <em>et al.</em>, 1998</a>), we have not
 detected a mating defect in <em>dot1</em> haploids; therefore,
 <em>dot1</em> homozygous diploids were made directly by mating the
 haploid parents. A <em>rad54::LEU2</em> disruption plasmid
 (pSM31) was provided by D. Schild (Lawrence Berkeley Laboratories,
 Berkeley). Plasmids used for other gene disruptions have been described
 previously: <em>zip1::LEU2</em> (<a href="#B62" rid="B62" class=" bibr popnode">Sym <em>et al.</em>,
 1993</a>), <em>zip1::LYS2</em> (<a href="#B63" rid="B63" class=" bibr popnode">Sym and Roeder, 1994</a>),
 <em>zip1::URA3</em> (<a href="#B64" rid="B64" class=" bibr popnode">Sym and Roeder, 1995</a>),
 <em>dmc1::ARG4</em> (<a href="#B8" rid="B8" class=" bibr popnode">Bishop <em>et al.</em>, 1992</a>), and
 <em>rad24::TRP1</em> (<a href="#B40" rid="B40" class=" bibr popnode">Lydall and Weinert, 1997</a>). Strains
 containing <em>SIR3-HA</em> integrated at the <em>ura3-1</em> locus
 were provided by E. Hong and B. Rockmill (Yale University). Plasmids
 pSS63 and pSS64 containing <em>DOT1-HA</em> and <em>DOT1-GFP</em>,
 respectively, in the high-copy vector pRS424 (<a href="#B14" rid="B14" class=" bibr popnode">Christianson <em>et
 al.</em>, 1992</a>) were constructed by cloning in frame a
 <em>Not</em>I-<em>Not</em>I fragment carrying three copies of the
 hemagglutinin (HA) epitope or green fluorescent protein (GFP)-coding
 sequences (from plasmids p264 and p266, respectively; provided by B.
 Santos; Yale University) into a <em>Not</em>I site generated by PCR
 after the second codon of <em>DOT1</em>. Additional details of
 plasmid and strain constructions are available upon request. Media,
 sporulation conditions, and determination of the frequency of
 sporulation and meiotic nuclear division have been described (<a href="#B62" rid="B62" class=" bibr popnode">Sym
 <em>et al.</em>, 1993</a>; <a href="#B63" rid="B63" class=" bibr popnode">Sym and Roeder, 1994</a>; <a href="#B52" rid="B52" class=" bibr popnode">San-Segundo and Roeder,
 1999</a>).
 
 </p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p>Yeast strains
 </p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Strain</th><th rowspan="1" colspan="1">Genotype</th></tr></thead><tbody align="center"><tr><td rowspan="1" colspan="1">BR2495</td><td align="left" rowspan="1" colspan="1"><span class="underline"><em>MAT</em><strong>a</strong><em>&#x02003;&#x02002;leu2-27&#x02003;his4-280</em></span>&#x02002;&#x02002;<span class="underline"><em>trp1-1</em></span>&#x02002;&#x02002;<span class="underline"><em>arg4-8&#x02002;thr1-1</em></span>&#x02002;<span class="underline"><em>ura3-1</em></span>&#x02002;<span class="underline"><em>ade2-1</em></span>&#x02002;&#x02009;<span class="underline"><em>cyh10</em></span></td></tr><tr><td rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1"><em>MAT&#x003b1;&#x02002;leu2-3,112&#x02002;his4-260&#x02002;trp1-289&#x02002;ARG4&#x02002;thr1-4&#x02002;ura3-1&#x02002;ade2-1&#x02002;CYH10</em></td></tr><tr><td rowspan="1" colspan="1">MY63</td><td align="left" rowspan="1" colspan="1">BR2495 but homozygous <em>zip1&#x02237;LEU2</em></td></tr><tr><td rowspan="1" colspan="1">DP138</td><td align="left" rowspan="1" colspan="1">BR2495 but homozygous <em>dot1&#x02237;URA3</em></td></tr><tr><td rowspan="1" colspan="1">DP139</td><td align="left" rowspan="1" colspan="1">BR2495 but homozygous <em>zip1&#x02237;LEU2 dot1&#x02237;URA3</em></td></tr><tr><td rowspan="1" colspan="1">MY152</td><td align="left" rowspan="1" colspan="1">BR2495 but homozygous <em>zip1&#x02237;URA3</em></td></tr><tr><td rowspan="1" colspan="1">DP381</td><td align="left" rowspan="1" colspan="1">BR2495 but homozygous <em>zip1&#x02237;URA3 dot1&#x02237;TRP1</em></td></tr><tr><td rowspan="1" colspan="1">DP382</td><td align="left" rowspan="1" colspan="1">BR2495 but homozygous <em>zip1&#x02237;URA3 dot1&#x02237;TRP1
 rad54&#x02237;LEU2</em></td></tr><tr><td rowspan="1" colspan="1">DP201</td><td align="left" rowspan="1" colspan="1">BR2495 but homozygous <em>zip1&#x02237;LEU2
 PCH2-HA</em></td></tr><tr><td rowspan="1" colspan="1">DP285</td><td align="left" rowspan="1" colspan="1">BR2495 but homozygous <em>zip1&#x02237;LEU2
 dot1&#x02237;TRP1 PCH2-HA</em></td></tr><tr><td rowspan="1" colspan="1">BR1919-2N</td><td align="left" rowspan="1" colspan="1"><span class="underline"><em>MAT</em><strong>a</strong><em>&#x02002;leu2-3,112&#x02002;his4-260</em></span>&#x02002;<span class="underline"><em>trp1-289</em></span>&#x02002;<span class="underline"><em>ARG4&#x02002;thr1-4</em></span>&#x02002;<span class="underline"><em>ura3-1</em></span>&#x02002;<span class="underline"><em>ade2-1</em></span>&#x02002;<span class="underline"><em>CYH10</em></span></td></tr><tr><td rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1"><em>MAT&#x003b1;
 leu2-3,112&#x02002;his4-260&#x02002;trp1-289&#x02002;ARG4&#x02002;thr1-4&#x02002;ura3-1&#x02002;ade2-1&#x02002;CYH10</em></td></tr><tr><td rowspan="1" colspan="1">DP365</td><td align="left" rowspan="1" colspan="1">BR1919-2N but homozygous <em>sir3&#x02237;TRP1
 SIR3-HA&#x02237;URA3</em></td></tr><tr><td rowspan="1" colspan="1">DP366</td><td align="left" rowspan="1" colspan="1">BR1919-2N but homozygous
 <em>sir3&#x02237;TRP1 SIR3-HA&#x02237;URA3 dot1&#x02237;TRP1</em></td></tr><tr><td rowspan="1" colspan="1">DP185</td><td align="left" rowspan="1" colspan="1"><span class="underline"><em>MAT</em><strong>a</strong><em>&#x02002;&#x02002;&#x02002;CENIII&#x02003;&#x02003;&#x02003;leu2-hisG&#x02003;&#x02003;HIS4&#x02003;&#x02002;</em></span>&#x02002;<span class="underline"><em>lys2</em></span>&#x02002;<span class="underline"><em>ho&#x02237;LYS2&#x02002;trp1-H3</em></span>&#x02002;<span class="underline"><em>ura3</em></span></td></tr><tr><td rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1"><em>MAT&#x003b1;&#x02002;CENIII&#x02237;TRP1&#x02002;leu2-hisG&#x02002;his4-B-LEU2&#x02002;lys2&#x02002;ho&#x02237;lys2&#x02002;trp1-H3&#x02002;ura3</em></td></tr><tr><td rowspan="1" colspan="1">DP184</td><td align="left" rowspan="1" colspan="1">DP185 but homozygous <em>dot1&#x02237;URA3</em></td></tr><tr><td rowspan="1" colspan="1">DP246</td><td align="left" rowspan="1" colspan="1">DP185&#x02002;but&#x02002;<span class="underline"><em>zip1&#x02237;LEU2</em></span></td></tr><tr><td rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">&#x02002;&#x02002;&#x02002;&#x02002;&#x02002;&#x02002;&#x02002;&#x02002;&#x02002;&#x02002;&#x02002;<em>zip1&#x02237;LYS2</em></td></tr><tr><td rowspan="1" colspan="1">DP371</td><td align="left" rowspan="1" colspan="1">DP185&#x02002;but&#x02002;<span class="underline"><em>zip1&#x02237;LEU2</em></span> and
 homozygous <em>dot1&#x02237;URA3</em></td></tr><tr><td rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">&#x02002;&#x02002;&#x02002;&#x02002;&#x02002;&#x02002;&#x02002;&#x02002;&#x02002;&#x02002;&#x02002;<em>zip1&#x02237;LYS2</em></td></tr><tr><td rowspan="1" colspan="1">DP338</td><td align="left" rowspan="1" colspan="1"><span class="underline"><em>MAT</em><strong>a</strong><em>&#x02002;leu2</em></span>&#x02002;<span class="underline"><em>lys2</em></span>&#x02002;<span class="underline"><em>ho&#x02237;LYS2&#x02002;trp1</em></span>&#x02002;<span class="underline"><em>ura3</em></span>&#x02002;<span class="underline"><em>arg4</em></span></td></tr><tr><td rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1"><em>MAT&#x003b1;&#x02002;leu2&#x02002;lys2&#x02002;ho&#x02237;LYS2&#x02002;trp1&#x02002;ura3&#x02002;arg4</em></td></tr><tr><td rowspan="1" colspan="1">DP339</td><td align="left" rowspan="1" colspan="1">DP338 but homozygous <em>dmc1&#x02237;ARG4</em></td></tr><tr><td rowspan="1" colspan="1">DP341</td><td align="left" rowspan="1" colspan="1">DP338 but homozygous <em>dmc1&#x02237;ARG4 dot1&#x02237;TRP1</em></td></tr><tr><td rowspan="1" colspan="1">DP343</td><td align="left" rowspan="1" colspan="1">DP338 but homozygous <em>dmc1&#x02237;ARG4 dot1&#x02237;TRP1
 rad54&#x02237;LEU2</em></td></tr><tr><td rowspan="1" colspan="1">DP352</td><td align="left" rowspan="1" colspan="1">DP338 but homozygous <em>dmc1&#x02237;ARG4
 rad24&#x02237;TRP1</em></td></tr><tr><td rowspan="1" colspan="1">DP342</td><td align="left" rowspan="1" colspan="1">DP338 but homozygous <em>dmc1&#x02237;ARG4
 rad54&#x02237;LEU2</em></td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139690859469792"><a target="object" rel="noopener" href="/pmc/articles/PMC15018/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="fn-a.r.a.b.d.a"><p class="p p-first-last">Strains DP185, DP184, DP246, DP371, DP338, DP339, DP341, DP343,
 DP352 and DP342 are isogenic with SK1.&#x000a0;</p></div></div></div></div><div id="sec-a.q.b.c" class="sec"><h3 id="sec-a.q.b.ctitle">Cytology</h3><p class="p p-first-last">Chromosome spreads and immunostaining of Pch2-HA, Sir2, Zip1,
 and Red1 were performed as described previously (<a href="#B52" rid="B52" class=" bibr popnode">San-Segundo and
 Roeder, 1999</a>). To detect Sir3-HA and Dot1-HA, mouse monoclonal anti-HA
 antibody (HA.11; Babco, Richmond, CA) was used at 1:100 and 1:150
 dilution, respectively. Rat antitubulin antibody (YOL 1/34; Sera-Lab,
 Crawley Down, Sussex, United Kingdom) was used at 1:100
 dilution. Rabbit anti-Rad51 antibody (a gift of D. Bishop, University
 of Chicago) was used at 1:400 dilution. Anti-Nsr1 antibodies were
 described previously (<a href="#B52" rid="B52" class=" bibr popnode">San-Segundo and Roeder, 1999</a>). To observe the
 Dot1-GFP fluorescent signal combined with 4&#x02032;-6-diamidino-2-phenylindole
 (DAPI) staining in whole cells, 0.1 ml of 37&#x00025; formaldehyde was added
 to aliquots (0.9 ml) of vegetative or sporulating cultures. After
 fixation for 15&#x02013;20 min (mitotic cells) or 45&#x02013;60 min (meiotic cells)
 at 30&#x000b0;C with rotation, cells were washed twice with
 phosphate-buffered saline (PBS) and incubated for 15 min in PBS
 containing 0.1&#x00025; Triton X-100 and DAPI (2 &#x003bc;g/ml). Cells were washed
 twice with PBS, resuspended in a small volume of PBS, and observed at
 the fluorescence microscope (Nikon Eclipse E800, Plan Apo 100&#x000d7;/1.4 oil
 objective) equipped with an FITC HYQ filter (<em class="company">Chroma Technology
 Corporation</em>, Brattleboro, VT) to visualize the GFP signal.
 Images were captured with a Sensys charge-coupled device camera
 (Photometrics, Tucson, AZ).</p></div><div id="sec-a.q.b.d" class="sec sec-last"><h3 id="sec-a.q.b.dtitle">Dityrosine Fluorescence Assay</h3><p class="p p-first-last">To examine dityrosine fluorescence as an indicator of
 sporulation, patches of cells were grown on YPAD plates and
 replica-plated to sporulation plates overlaid with nitrocellulose
 filters (Protran BA85; Schleicher &#x00026; Schuell, Keene, NH). After
 1&#x02013;3 d incubation at 30&#x000b0;C (depending on strain background),
 dityrosine fluorescence was visualized by illuminating the plates from
 the top (without the lid) with 302-nm UV light. Photographs were taken
 using 667 Polaroid film and a Wratten gelatin filter (No. 47B; Kodak,
 Rochester, NY).</p></div></div><div id="sec-a.q.c" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.q.ctitle" style="text-transform: uppercase;">RESULTS</h2><div id="sec-a.q.c.b" class="sec sec-first"><h3 id="sec-a.q.c.btitle">Identification of <em>DOT1</em></h3><p class="p p-first">To identify genes involved in the pachytene checkpoint in <em>S.
 cerevisiae</em>, a screen for mutations that alleviate the meiotic
 arrest of the <em>zip1</em> mutant was performed; mutations were
 introduced by transformation with a transposon-mutagenized yeast
 genomic library (<a href="#B52" rid="B52" class=" bibr popnode">San-Segundo and Roeder, 1999</a>). In addition to
 insertions in three other genes (<em>RAD24</em>, <em>DDC1</em>, and
 <em>PCH2</em>), two transposon insertions were found in the open
 reading frame YDR440w, which we initially called <em>PCH1</em> (for
 pachytene checkpoint). During the course of this study, the same gene
 was isolated in a screen for high-copy disruptors of telomeric
 silencing and named <em>DOT1</em> (<a href="#B56" rid="B56" class=" bibr popnode">Singer <em>et al.</em>, 1998</a>).
 Therefore, the name <em>DOT1</em> is used throughout this article.</p><p><em>DOT1</em> encodes a basic (predicted pI 9.7) 582-amino-acid
 protein with no other homologs in the yeast genome. The N-terminal half
 of Dot1 is unique and has a high content of lysine residues (Figure
 <a href="/pmc/articles/PMC15018/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1A).</span></span><span>1</span></a>A). This lysine-rich domain contains at
 least two consensus monopartite nuclear localization signals (<a href="#B13" rid="B13" class=" bibr popnode">Chelsky
 <em>et al.</em>, 1989</a>; Figure <a href="/pmc/articles/PMC15018/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1A).</span></span><span>1</span></a>A). The C terminus of Dot1 displays
 significant homology over a region of &#x0223c;200 amino acids with putative
 proteins identified in systematic sequencing of the human and <em>C.
 elegans</em> genomes (Figure <a href="/pmc/articles/PMC15018/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1).</span></span><span>1</span></a>). One of the mutant alleles of
 <em>DOT1</em> recovered from the screen encodes a truncated protein
 lacking only the last 55 amino acids, thus removing the last conserved
 block (Figure <a href="/pmc/articles/PMC15018/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1,</span></span><span>1</span></a>, A and B). This allele displays a phenotype
 indistinguishable from that of the <em>dot1</em> null mutant (see
 below; our unpublished results).
 
 </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC15018/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139690855675488"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=15018_mk1001328001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is mk1001328001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC15018/bin/mk1001328001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139690855675488"><a target="object" rel="noopener" href="/pmc/articles/PMC15018/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC15018/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p>DOT1 sequence features. (A) Diagram of the Dot1
 protein (rectangle) and <em>DOT1</em> promoter (discontinuous
 line). Indicated are a lysine-rich domain (striped box; 21.5&#x00025; of
 residues between amino acids 35 and 169 are lysines), two putative NLSs
 starting at positions 126 and 162, and the region of homology to human
 and <em>C. elegans</em> ORFs (gray area). The sequence
 <span class="underline">ACGCGT</span>CA at positions &#x02212;172 to &#x02212;165 in the
 <em>DOT1</em> promoter corresponds to a consensus MCB element
 (MluI cell cycle box site is underlined). (B) Alignment of the regions
 of homology from <em>S. cerevisiae</em> Dot1, human ORF <span>R29381</span>_1
 and <em>C. elegans</em> ORFs W06D11.4, F54F7.7, F55G7.2, and
 ZC53.6. Identities are highlighted in black and conservative
 substitutions in gray. Numbers at the left represent amino acid
 positions. Alignment and shading were performed with ClustalW 1.7 and
 Boxshade 3.21, respectively, at the BCM Search Launcher
 (<a href="http://dot.imgen.bcm.tmc.edu:9331/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://dot.imgen.bcm.tmc.edu:9331/</a>). In A and B, the position of the
 C-terminal transposon insertion recovered from the mutant screen is
 marked with a triangle. (C) Percentage identity and similarity of Dot1
 to the human and nematode homologs over the indicated length. The
 significance of the homology is indicated by the BLAST E-values.</p></div></div></div><p class="p p-last">Examination of the <em>DOT1</em> promoter revealed the presence of a
 &#x0201c;<em>Mlu</em>I cell cycle box&#x0201d; (Figure <a href="/pmc/articles/PMC15018/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1A).</span></span><span>1</span></a>A). This element
 is the binding site for the MBF transcription factor, which
 activates transcription of genes at the G1/S transition in the mitotic
 cell cycle (<a href="#B32" rid="B32" class=" bibr popnode">Johnston and Lowndes, 1992</a>; <a href="#B33" rid="B33" class=" bibr popnode">Koch and Nasmyth, 1994</a>).
 Indeed, genome-wide analysis of gene expression during the mitotic cell
 cycle has shown that transcription of <em>DOT1</em> is cell-cycle
 regulated with the peak of expression at G1 (<a href="#B58" rid="B58" class=" bibr popnode">Spellman <em>et
 al.</em>, 1998</a>). In contrast, <em>DOT1</em> transcription is not
 regulated during the meiotic cell cycle (<a href="#B15" rid="B15" class=" bibr popnode">Chu <em>et al.</em>, 1998</a>).</p></div><div id="sec-a.q.c.c" class="sec"><h3 id="sec-a.q.c.ctitle">Dot1 Is Required for Meiotic Prophase Arrest or Delay in the
 <em>zip1</em> Mutant</h3><p class="p p-first">To characterize the role of Dot1 in meiosis, null mutants were
 constructed and analyzed in diploid strains. The <em>dot1</em> single
 mutant displays the wild-type level of sporulation (Figure
 <a href="/pmc/articles/PMC15018/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2A)</span></span><span>2</span></a>A) and high levels of spore viability
 and crossing over (Table <a href="/pmc/articles/PMC15018/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table2).</span></span><span>2</span></a>). Thus, Dot1
 is not normally required for meiotic progression, recombination,
 chromosome segregation and spore formation.
 
 
 
 </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC15018/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139690851609696"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=15018_mk1001328002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is mk1001328002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC15018/bin/mk1001328002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139690851609696"><a target="object" rel="noopener" href="/pmc/articles/PMC15018/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC15018/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p>Dot1 is required for checkpoint-induced
 meiotic arrest (or delay) of <em>zip1</em> and
 <em>dmc1</em> mutants. (A) Mutation of <em>DOT1</em>
 alleviates the meiotic arrest of <em>zip1</em> in BR2495 isogenic
 strains. Time course of sporulation of strains BR2495 (wild type), MY63
 (<em>zip1</em>), DP138 (<em>dot1</em>), and DP139
 (<em>zip1 dot1</em>). (B) Mutation of <em>DOT1</em>
 bypasses the meiotic delay of <em>zip1</em> in the SK1 strain
 background. Dityrosine fluorescence, an indicator of sporulation, was
 examined after 1 and 2 d on sporulation plates. Strains are DP185
 (wild type), DP246 (<em>zip1</em>), and DP371 (<em>zip1
 dot1</em>). (C) Distribution of tetrad types. The percentages of
 tetrads with 4, 3, 2, 1 and 0 viable spores (4-sv, 3-sv, 2-sv, 1-sv,
 and 0-sv, respectively) are represented. Tetrads (389 and 513) were
 dissected from DP246 (<em>zip1</em>) and DP371 (<em>zip1
 dot1</em>), respectively. (D) Bypass of <em>dmc1</em> arrest
 by <em>dot1</em> and effects of the <em>rad54</em>
 mutation. Dityrosine fluorescence was examined after 2 d on
 sporulation plates. (&#x00025;) MI &#x0002b; MII represents the fraction of cells that
 have undergone at least one meiotic nuclear division (i.e., containing
 two or more DAPI-stained bodies) after 24 h in liquid sporulation
 medium. (&#x00025;) Spo. represents the fraction of cells that formed mature
 or immature asci. The asterisks indicate that most asci are
 immature. At least 300 cells were scored for each strain. Strains are
 DP338 (wild type), DP339 (<em>dmc1</em>), DP341 (<em>dmc1
 dot1</em>), DP343 (<em>dmc1 dot1 rad54</em>), DP352
 (<em>dmc1 rad24</em>), and DP342 (<em>dmc1 rad54</em>).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p>Spore viability and crossing over in SK1 strains
 </p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Genotype</th><th rowspan="1" colspan="1">Spore viability
 (&#x00025;)</th><th rowspan="1" colspan="1">Crossing over (cM)</th><th rowspan="1" colspan="1">Tetrads dissected</th><th rowspan="1" colspan="1">Strains</th></tr></thead><tbody align="center"><tr><td rowspan="1" colspan="1">Wild
 type</td><td rowspan="1" colspan="1">98.0</td><td rowspan="1" colspan="1">50.8</td><td rowspan="1" colspan="1">215</td><td rowspan="1" colspan="1">DP185</td></tr><tr><td rowspan="1" colspan="1"><em>dot1</em></td><td rowspan="1" colspan="1">93.5</td><td rowspan="1" colspan="1">45.0</td><td rowspan="1" colspan="1">128</td><td rowspan="1" colspan="1">DP184</td></tr><tr><td rowspan="1" colspan="1"><em>zip1</em></td><td rowspan="1" colspan="1">52.3</td><td rowspan="1" colspan="1">15.2</td><td rowspan="1" colspan="1">389</td><td rowspan="1" colspan="1">DP246</td></tr><tr><td rowspan="1" colspan="1"><em>zip1
 dot1</em></td><td rowspan="1" colspan="1">31.5</td><td rowspan="1" colspan="1">13.6</td><td rowspan="1" colspan="1">513</td><td rowspan="1" colspan="1">DP371</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139690844830512"><a target="object" rel="noopener" href="/pmc/articles/PMC15018/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="fn-a.r.a.e.d.a"><p class="p p-first-last">Crossover data are the sum of the map distances in the
 <em>MAT-CENIII</em> and <em>CENIII-HIS4</em> intervals on
 chromosome III. Calculations are based on the following numbers of
 4-spore-viable tetrads: 203 for wild type, 112 for <em>dot1</em>, 129
 for <em>zip1</em> and 84 for <em>zip1 dot1</em>.&#x000a0;</p></div></div></div><p>The sporulation phenotype of the <em>zip1</em> mutant depends on
 strain background, though chromosomes fail to synapse in all
 <em>zip1</em> strains tested (<a href="#B62" rid="B62" class=" bibr popnode">Sym <em>et al.</em>, 1993</a>; <a href="#B63" rid="B63" class=" bibr popnode">Sym and
 Roeder, 1994</a>). In the BR2495 strain background, <em>zip1</em> arrests
 at pachytene, and no mature recombinants are produced (<a href="#B62" rid="B62" class=" bibr popnode">Sym <em>et
 al.</em>, 1993</a>; <a href="#B51" rid="B51" class=" bibr popnode">Roeder, 1997</a>). Deletion of <em>DOT1</em> completely
 alleviates the <em>zip1</em> meiotic arrest; the kinetics and
 frequency of sporulation in <em>zip1 dot1</em> are similar to wild
 type (Table <a href="/pmc/articles/PMC15018/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table3</span></span><span>3</span></a> and Figure <a href="/pmc/articles/PMC15018/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure2A).</span></span><span>2</span></a>A). However,
 spore viability is reduced to &#x0223c;37&#x00025; (Table <a href="/pmc/articles/PMC15018/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table3),</span></span><span>3</span></a>), suggesting that some
 or all of the defects conferred by <em>zip1</em> persist in the
 double mutant. In fact, chromosomes do not synapse in <em>zip1
 dot1</em> (see DISCUSSION).
 
 </p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p>Sporulation and spore viability in BR2495 strains
 </p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Genotype</th><th rowspan="1" colspan="1">Sporulation
 efficiency<sup>a</sup> (&#x00025;)</th><th rowspan="1" colspan="1">Spore
 viability<sup>b</sup> (&#x00025;)</th><th rowspan="1" colspan="1">Strains</th></tr></thead><tbody align="center"><tr><td rowspan="1" colspan="1">Wild
 type<sup>c</sup></td><td rowspan="1" colspan="1">51</td><td rowspan="1" colspan="1">95.3</td><td rowspan="1" colspan="1">BR2495</td></tr><tr><td rowspan="1" colspan="1"><em>zip1</em></td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">&#x02014;</td><td rowspan="1" colspan="1">MY152</td></tr><tr><td rowspan="1" colspan="1"><em>zip1
 dot1</em></td><td rowspan="1" colspan="1">57</td><td rowspan="1" colspan="1">36.9</td><td rowspan="1" colspan="1">DP381<sup>d</sup></td></tr><tr><td rowspan="1" colspan="1"><em>zip1 dot1
 rad54</em></td><td rowspan="1" colspan="1">54</td><td rowspan="1" colspan="1">13.7</td><td rowspan="1" colspan="1">DP382<sup>e</sup></td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139690856364176"><a target="object" rel="noopener" href="/pmc/articles/PMC15018/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TF3-a"><sup>a</sup>&#x02009;Sporulation efficiency was
 determined by counting the fraction of cells forming asci after 3 days
 on sporulation plates; 400 cells were scored for each strain.&#x000a0;</div><div id="TF3-b"><sup>b</sup>&#x02009;Spore viability was assessed by
 tetrad dissection.&#x000a0;</div><div id="TF3-c"><sup>c</sup>&#x02009;Viability data of wild type was
 presented by <a href="#B52" rid="B52" class=" bibr popnode">San-Segundo and Roeder (1999)</a>, and is based
 on the dissection of 256 tetrads.&#x000a0;</div><div id="TF3-d"><sup>d</sup>&#x02009;Eighty-eight tetrads were dissected
 for DP381.&#x000a0;</div><div id="TF3-e"><sup>e</sup>&#x02009;Eighty-seven tetrads were dissected
 for DP382.&#x000a0;</div></div></div><p class="p p-last">In the SK1 strain background, the <em>zip1</em> mutant sporulates,
 but MI is delayed and Holliday junctions persist longer than in wild
 type (<a href="#B63" rid="B63" class=" bibr popnode">Sym and Roeder, 1994</a>; <a href="#B59" rid="B59" class=" bibr popnode">Storlazzi <em>et al.</em>, 1996</a>);
 however, recombination intermediates are eventually resolved, and
 &#x0223c;50&#x00025; of the spores produced are viable (Table <a href="/pmc/articles/PMC15018/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table2;</span></span><span>2</span></a>; <a href="#B63" rid="B63" class=" bibr popnode">Sym and Roeder,
 1994</a>). <em>DOT1</em> is required for the meiotic delay of
 <em>zip1</em> in the SK1 strain background; <em>zip1 dot1</em>
 sporulates with wild-type kinetics (Figure <a href="/pmc/articles/PMC15018/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2B;</span></span><span>2</span></a>B; our unpublished
 results). SK1 strains were used to compare recombination and chromosome
 segregation in the <em>zip1</em> and <em>zip1 dot1</em> mutants.
 The reduction in spore viability in <em>zip1</em> (Table <a href="/pmc/articles/PMC15018/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table2;</span></span><span>2</span></a>; <a href="#B63" rid="B63" class=" bibr popnode">Sym and
 Roeder, 1994</a>) is mostly due to pairs of homologous chromosomes
 segregating to the same pole at MI because of a failure to cross over
 (<a href="#B63" rid="B63" class=" bibr popnode">Sym and Roeder, 1994</a>). Relief of the meiotic delay of <em>zip1</em>
 by a <em>dot1</em> mutation leads to a further decrease in spore
 viability from &#x0223c;52 to &#x0223c;31&#x00025; (Table <a href="/pmc/articles/PMC15018/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table2).</span></span><span>2</span></a>). However, crossing over in
 <em>zip1 dot1</em> is not significantly different from
 <em>zip1</em>, at least for the chromosome III intervals examined
 (Table <a href="/pmc/articles/PMC15018/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table2).</span></span><span>2</span></a>). Homolog nondisjunction at meiosis I in <em>zip1</em> is
 manifested by a nonrandom pattern of spore death (<a href="#B63" rid="B63" class=" bibr popnode">Sym and Roeder,
 1994</a>), with 0-, 2-, and 4-spore-viable tetrads predominating (Figure
 <a href="/pmc/articles/PMC15018/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2C).</span></span><span>2</span></a>C). The same pattern of spore death is exhibited by <em>zip1
 dot1</em> (Figure <a href="/pmc/articles/PMC15018/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2C),</span></span><span>2</span></a>C), but the proportion of 0-spore-viable tetrads is
 increased at the expense of tetrads with four viable spores (Figure
 <a href="/pmc/articles/PMC15018/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2C).</span></span><span>2</span></a>C). Thus, in the absence of <em>DOT1</em>, <em>zip1</em>
 fails to arrest or delay in meiosis, resulting in chromosome
 missegregation and the formation of aneuploid meiotic products.</p></div><div id="sec-a.q.c.d" class="sec"><h3 id="sec-a.q.c.dtitle">Dot1 Prevents Cell Cycle Progression and Rad54-dependent
 Intersister Recombination in the <em>dmc1</em> Mutant</h3><p class="p p-first">In SK1 strains, the <em>dmc1</em> mutant arrests in meiosis with
 unrepaired DSBs. To determine whether Dot1 is required for this
 checkpoint-dependent arrest, <em>dmc1 dot1</em> double mutants were
 examined. Deletion of <em>DOT1</em> relieves the meiotic arrest of
 <em>dmc1</em>. The <em>dmc1 dot1</em> double mutant undergoes high
 levels of meiotic nuclear division and sporulation (Figure <a href="/pmc/articles/PMC15018/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2D);
</span></span><span>2</span></a>D);
 however, spore viability is very low (&#x0223c;5&#x00025; for <em>dmc1 dot1</em>
 vs. &#x0003e;95&#x00025; for wild type; 110 and 227 tetrads dissected, respectively).</p><p>Strikingly, a high proportion of the <em>dmc1 dot1</em> meiotic
 products are enclosed in asci-containing mature spores similar to those
 of wild type, as evidenced by microscopic examination (our unpublished
 results) and formation of dityrosine (Figure <a href="/pmc/articles/PMC15018/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2D),</span></span><span>2</span></a>D), a fluorescent
 component of spore walls (<a href="#B11" rid="B11" class=" bibr popnode">Briza <em>et al.</em>, 1990</a>). In contrast,
 the <em>dmc1 rad24</em> double mutant proceeds through the meiotic
 divisions with unrepaired DSBs, resulting in the formation of nuclei
 with fragmented DNA and few asci with mature spores (<a href="#B38" rid="B38" class=" bibr popnode">Lydall <em>et
 al.</em>, 1996</a>), manifested by the almost complete absence of
 dityrosine fluorescence (Figure <a href="/pmc/articles/PMC15018/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2D).</span></span><span>2</span></a>D). The fact that <em>dmc1 dot1</em>
 forms morphologically normal spores suggests that a significant
 fraction of meiotic DSBs in <em>dmc1</em> are repaired in the absence
 of Dot1.</p><p>To examine the presence of recombination intermediates relative to
 meiotic progression, the cytological assay described by <a href="#B38" rid="B38" class=" bibr popnode">Lydall <em>et
 al.</em> (1996)</a> was used. This method is based on the use of anti-Rad51
 antibodies to detect recombination intermediates (presumably unrepaired
 DSBs) in combination with tubulin staining to identify cells that have
 initiated MI. In wild type, all Rad51 foci have disappeared when the MI
 spindle is formed (<a href="#B38" rid="B38" class=" bibr popnode">Lydall <em>et al.</em>, 1996</a>; Figure
 <a href="/pmc/articles/PMC15018/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure3A).</span></span><span>3</span></a>A). In the arrested <em>dmc1</em>
 mutant, a single tubulin-stained focus characteristic of duplicated but
 unseparated spindle pole bodies is present, and numerous Rad51 foci
 persist (Table <a href="/pmc/articles/PMC15018/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table4</span></span><span>4</span></a> and Figure <a href="/pmc/articles/PMC15018/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure3B).</span></span><span>3</span></a>B). Nuclei
 in the <em>dmc1 dot1</em> mutant fall into three classes (Table <a href="/pmc/articles/PMC15018/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table4).
</span></span><span>4</span></a>).
 Some nuclei (38&#x00025;) are similar to those of wild type (i.e., MI spindle
 and no Rad51 foci), indicating that all DSBs have been repaired (Figure
 <a href="/pmc/articles/PMC15018/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3C).</span></span><span>3</span></a>C). In other nuclei (56&#x00025;), a few Rad51 foci persist concomitantly
 with the elongated spindle (Figure <a href="/pmc/articles/PMC15018/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure3D).</span></span><span>3</span></a>D). Occasionally (6&#x00025;), MI nuclei
 containing 20&#x02013;35 Rad51 foci are observed (Table <a href="/pmc/articles/PMC15018/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table4;</span></span><span>4</span></a>; our unpublished
 results). Thus, whereas &#x0223c;63 Rad51 foci accumulate in the
 <em>dmc1</em> single mutant, only approximately four foci (on
 average) persist in <em>dmc1 dot1</em> nuclei undergoing MI (Table
 <a href="/pmc/articles/PMC15018/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table4).</span></span><span>4</span></a>). These results indicate that most DSBs are repaired when
 <em>dmc1</em> arrest is bypassed by mutation of <em>DOT1</em>; in
 &#x0223c;40&#x00025; of cells, all DSBs are repaired. Because most <em>dmc1
 dot1</em> spores are dead, this repair presumably occurs using sister
 chromatids as donors instead of homologs (interhomolog recombination
 would ensure proper disjunction at meiosis I).
 
 
 
 </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC15018/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139690860437664"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=15018_mk1001328003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is mk1001328003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC15018/bin/mk1001328003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139690860437664"><a target="object" rel="noopener" href="/pmc/articles/PMC15018/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC15018/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p>Cytological analysis of unrepaired DSBs and
 spindle elongation. Spread meiotic nuclei from strains DP338 (wild
 type; A), DP339 (<em>dmc1</em>; B), DP341 (<em>dmc1
 dot1</em>; C and D), DP343 (<em>dmc1 dot1 rad54</em>; E), and
 DP352 (<em>dmc1 rad24</em>; F), stained with DAPI (left) and
 anti-Rad51 (red) and anti-tubulin (green) antibodies (right). Spreads
 were prepared after 6 h in sporulation medium. Bar, 2 &#x003bc;m.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p>Quantitation of Rad51 foci associated with MI
 spindle </p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Genotype</th><th rowspan="1" colspan="1">Nuclei (&#x00025;)</th><th rowspan="1" colspan="1">Rad51 foci (No.)</th><th rowspan="1" colspan="1">MI spindle</th><th rowspan="1" colspan="1">Nuclei
 scored</th><th rowspan="1" colspan="1">Strain</th></tr></thead><tbody align="center"><tr><td rowspan="1" colspan="1"><em>dmc1</em></td><td rowspan="1" colspan="1">100</td><td rowspan="1" colspan="1">62.7&#x02009;
 &#x000b1;&#x02009;7.0</td><td rowspan="1" colspan="1">&#x02212;</td><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">DP339</td></tr><tr><td rowspan="1" colspan="1"><em>dmc1
 dot1</em><sup>a</sup></td><td rowspan="1" colspan="1">38</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1"><sup>&#x0002b;</sup></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">DP341</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">56</td><td rowspan="1" colspan="1">4.0&#x02009;
 &#x000b1;&#x02009;3.0</td><td rowspan="1" colspan="1"><sup>&#x0002b;</sup></td><td rowspan="1" colspan="1">50</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">25.6&#x02009; &#x000b1;&#x02009;6.0</td><td rowspan="1" colspan="1"><sup>&#x0002b;</sup></td></tr><tr><td rowspan="1" colspan="1">&#x02003;Average<sup>a</sup></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">3.8&#x02009; &#x000b1;&#x02009;6.4</td></tr><tr><td rowspan="1" colspan="1"><em>dmc1 dot1 rad54</em></td><td rowspan="1" colspan="1">100</td><td rowspan="1" colspan="1">56.7&#x02009;
 &#x000b1;&#x02009;10.5</td><td rowspan="1" colspan="1"><sup>&#x0002b;</sup></td><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">DP343</td></tr><tr><td rowspan="1" colspan="1"><em>dmc1 rad24</em></td><td rowspan="1" colspan="1">100</td><td rowspan="1" colspan="1">58.0&#x02009;
 &#x000b1;&#x02009;12.9</td><td rowspan="1" colspan="1"><sup>&#x0002b;</sup></td><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">DP352</td></tr><tr><td rowspan="1" colspan="1"><em>zip1</em></td><td rowspan="1" colspan="1">100</td><td rowspan="1" colspan="1">25.1&#x02009;
 &#x000b1;&#x02009;4.3</td><td rowspan="1" colspan="1">&#x02212;</td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">MY152</td></tr><tr><td rowspan="1" colspan="1"><em>zip1
 dot1</em></td><td rowspan="1" colspan="1">100</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1"><sup>&#x0002b;</sup></td><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">DP381</td></tr><tr><td rowspan="2" colspan="1"><em>zip1 dot1
 rad54</em><sup>b</sup></td><td rowspan="1" colspan="1">43</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1"><sup>&#x0002b;</sup></td><td rowspan="2" colspan="1">30</td><td rowspan="2" colspan="1">DP382</td></tr><tr><td rowspan="1" colspan="1">56</td><td rowspan="1" colspan="1">2.3&#x02009; &#x000b1;&#x02009;1.6</td><td rowspan="1" colspan="1"><sup>&#x0002b;</sup></td></tr><tr><td rowspan="1" colspan="1">&#x02003;Average<sup>b</sup></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">1.3&#x02009;
 &#x000b1;&#x02009;1.6</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139690860435024"><a target="object" rel="noopener" href="/pmc/articles/PMC15018/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TF4-a"><sup>a</sup>&#x02009;MI Nuclei of <em>dmc1
 dot1</em> are grouped into three classes. The overall number of Rad51
 foci per nucleus is also represented (average).&#x000a0;</div><div id="TF4-b"><sup>b</sup>&#x02009;MI Nuclei of <em>zip1 dot1
 rad54</em> are grouped into two classes. The overall number of Rad51
 foci per nucleus is also represented (average).&#x000a0;</div></div></div><p class="p p-last">Several studies indicate that meiotic DSBs in the <em>dmc1</em>
 mutant can be repaired by a Rad54-dependent pathway of recombination
 between sister chromatids, if <em>RED1</em> is mutated or cells are
 returned to growth medium (<a href="#B53" rid="B53" class=" bibr popnode">Schwacha and Kleckner, 1997</a>; <a href="#B69" rid="B69" class=" bibr popnode">Xu <em>et
 al.</em>, 1997</a>; <a href="#B2" rid="B2" class=" bibr popnode">Arbel <em>et al.</em>, 1999</a>; <a href="#B7" rid="B7" class=" bibr popnode">Bishop <em>et
 al.</em>, 1999</a>). To test whether this alternative pathway is active in
 <em>dmc1 dot1</em>, the effects of a <em>rad54</em> mutation were
 examined. The <em>dmc1 dot1 rad54</em> triple mutant proceeds through
 the meiotic nuclear divisions (Figure <a href="/pmc/articles/PMC15018/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2D);</span></span><span>2</span></a>D); however, the triple mutant
 is unable to make mature spores (Figure <a href="/pmc/articles/PMC15018/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2D;</span></span><span>2</span></a>D; our unpublished results).
 Furthermore, numerous Rad51 foci are detected in <em>dmc1 dot1
 rad54</em> cells that have exited prophase and elongated the MI spindle
 (Table <a href="/pmc/articles/PMC15018/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table4</span></span><span>4</span></a> and Figure <a href="/pmc/articles/PMC15018/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure3E).</span></span><span>3</span></a>E). In this respect, <em>dmc1 dot1 rad54</em> is
 similar to <em>dmc1 rad24</em>, which enters MI with unrepaired DSBs
 (Table <a href="/pmc/articles/PMC15018/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table4</span></span><span>4</span></a> and Figure <a href="/pmc/articles/PMC15018/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure3F;</span></span><span>3</span></a>F; <a href="#B38" rid="B38" class=" bibr popnode">Lydall <em>et al.</em>, 1996</a>). Thus, the
 Rad54-dependent pathway is active in <em>dmc1 dot1</em>, but blocking
 repair of DSBs by a <em>rad54</em> mutation does not prevent meiotic
 progression.</p></div><div id="sec-a.q.c.e" class="sec"><h3 id="sec-a.q.c.etitle">Repair of DSBs in <em>zip1 dot1</em> Is Largely Independent
 of Rad54</h3><p class="p p-first">In contrast to <em>dmc1</em>, which accumulates DSBs, the
 <em>zip1</em> mutant accumulates primarily unresolved Holliday
 junctions; a small fraction (&#x0223c;10&#x00025;) of DSBs also remain unrepaired
 (<a href="#B59" rid="B59" class=" bibr popnode">Storlazzi <em>et al.</em>, 1996</a>). Although the spore viability of
 the <em>zip1 dot1</em> double mutant is significantly lower than that
 of wild type, the moderate fraction of viable spores produced (Tables <a href="/pmc/articles/PMC15018/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Tables2
</span></span><span>2</span></a>
 and <a href="/pmc/articles/PMC15018/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and3)</span></span><span>3</span></a>) indicates that DSBs have been repaired. To test whether the
 repair of DSBs in <em>zip1 dot1</em> depends on Rad54, a <em>zip1
 dot1 rad54</em> strain was analyzed. The triple mutant sporulates with
 the same efficiency as wild type or <em>zip1 dot1</em> (Table <a href="/pmc/articles/PMC15018/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table3)</span></span><span>3</span></a>) and
 generates mature spores (our unpublished results), suggesting that most
 (if not all) DSBs are repaired even in the absence of Rad54.</p><p class="p p-last">To confirm this conclusion, Rad51 and tubulin staining was carried out.
 In the arrested <em>zip1</em> mutant, no spindle is formed and &#x0223c;25
 Rad51 foci are present (Table <a href="/pmc/articles/PMC15018/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table4</span></span><span>4</span></a> and Figure
 <a href="/pmc/articles/PMC15018/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure4A).</span></span><span>4</span></a>A). In <em>zip1 dot1</em> nuclei that
 have entered MI, all Rad51 foci have disappeared (Table <a href="/pmc/articles/PMC15018/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table4</span></span><span>4</span></a> and Figure
 <a href="/pmc/articles/PMC15018/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure4B).</span></span><span>4</span></a>B). In the <em>zip1 dot1 rad54</em> triple mutant, two kinds of MI
 nuclei are found: &#x0223c;43&#x00025; are like <em>zip1 dot1</em> (i.e., no Rad51
 foci coexist with the spindle; Table <a href="/pmc/articles/PMC15018/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table4;</span></span><span>4</span></a>; our unpublished results),
 whereas &#x0223c;56&#x00025; nuclei still contain a few Rad51 in conjunction with a
 spindle (Table <a href="/pmc/articles/PMC15018/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table4</span></span><span>4</span></a> and Figure <a href="/pmc/articles/PMC15018/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure4,</span></span><span>4</span></a>, C and D). Consistent with the small
 number of unrepaired DSBs present in <em>zip1 dot1 rad54</em> (Table
 <a href="/pmc/articles/PMC15018/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table4),</span></span><span>4</span></a>), the triple mutant exhibits a further decrease in spore viability
 compared with <em>zip1 dot1</em> (Table <a href="/pmc/articles/PMC15018/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table3)</span></span><span>3</span></a>) and loss of the
 distinctive 4-, 2-, and 0-spore-viable pattern of spore death (Figure
 <a href="/pmc/articles/PMC15018/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4E).</span></span><span>4</span></a>E). Thus, unlike the situation in <em>dmc1 dot1</em>, only a minor
 fraction of the DSBs accumulated by <em>zip1</em> requires Rad54 for
 repair when meiotic arrest is relieved by a <em>dot1</em> mutation.
 
 </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC15018/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139690855378320"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=15018_mk1001328004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is mk1001328004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC15018/bin/mk1001328004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139690855378320"><a target="object" rel="noopener" href="/pmc/articles/PMC15018/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC15018/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p>Repair of DSBs in <em>zip1 dot1</em>
 is largely independent of Rad54. (A&#x02013;D) Cytological analysis of ongoing
 recombination and entry into MI, assessed by the presence of Rad51 foci
 and spindle elongation, respectively. Spread meiotic nuclei from
 strains MY152 (<em>zip1</em>; A), DP381 (<em>zip1
 dot1</em>; B), and DP382 (<em>zip1 dot1 rad54</em>; C and D),
 stained with DAPI (left) and anti-Rad51 (red) and anti-tubulin (green)
 antibodies (right). Spread nuclei were prepared after 16 h in
 sporulation medium. Bar, 2 &#x003bc;m. (E) Distribution of tetrad types. The
 percentages of tetrads with 4, 3, 2, 1, and 0 viable spores (4-sv,
 3-sv, 2-sv, 1-sv, and 0-sv, respectively) are represented. Tetrads (88
 and 87) were dissected from strains DP381 (<em>zip1 dot1</em>)
 and DP382 (<em>zip1 dot1 rad54</em>), respectively.</p></div></div></div></div><div id="sec-a.q.c.f" class="sec"><h3 id="sec-a.q.c.ftitle">Dot1 Is a Nuclear Protein That Associates with Chromosomes</h3><p class="p p-first">To study the cellular localization of Dot1, the protein was tagged
 at the N terminus with GFP or with three copies of the HA epitope. Both
 fusion proteins are fully functional (our unpublished results).</p><p>In mitotic diploid cells containing a high-copy plasmid carrying
 <em>DOT1-GFP</em>, the Dot1-GFP signal is detected in the nucleus,
 but is often enriched toward one side of the nucleus in a region that
 stains faintly with the DNA-binding dye DAPI (Figure
 <a href="/pmc/articles/PMC15018/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5A)</span></span><span>5</span></a>A) and corresponds to the nucleolus
 (<a href="#B46" rid="B46" class=" bibr popnode">Pintard <em>et al.</em>, 2000</a>). Dot1-GFP is found in cells at
 different stages of the mitotic cell cycle. The same pattern of nuclear
 localization with nucleolar accumulation is present in vegetative cells
 expressing Dot1-GFP from a low-copy plasmid, suggesting that this
 localization is not an artifact of overexpression (our unpublished
 results). In cells at different stages of meiotic progression, Dot1-GFP
 is also found in the nucleus, but accumulation in the nucleolus is not
 evident until meiosis has been completed and spores are being formed
 (Figure <a href="/pmc/articles/PMC15018/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5B).</span></span><span>5</span></a>B). In meiotic cells, visualization of Dot1-GFP depends on the
 use of a high-copy plasmid.
 
 </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC15018/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139690863941648"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=15018_mk1001328005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is mk1001328005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC15018/bin/mk1001328005.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139690863941648"><a target="object" rel="noopener" href="/pmc/articles/PMC15018/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC15018/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a7--><div class="caption"><p>Nuclear localization of Dot1. (A and B) Cells from
 strain DP138 (<em>dot1</em>) transformed with pSS64 (<em>2 &#x003bc;
 DOT1-GFP</em>) growing vegetatively (A) or at different times in
 sporulation medium (B) were fixed, stained with DAPI, and observed at
 the fluorescence microscope. The merged images of DAPI (red) and
 Dot1-GFP (green) are shown in the bottom panels. Overlap appears
 yellow. The corresponding differential interference contrast images are
 also presented (top). The approximate cell-cycle stage was estimated
 based on bud size and nuclear morphology and position. In B, only the
 large mother cells have entered meiosis; the small buds should not be
 considered. (C) Spread nuclei from strain DP138 (<em>dot1</em>)
 transformed with pSS63 (<em>2 &#x003bc; DOT1-HA</em>) growing vegetatively,
 stained with DAPI (blue) and anti-HA antibodies (red). (D) Spread
 leptotene/zygotene (top row) and pachytene (bottom row) nuclei from
 strain DP138 (<em>dot1</em>) containing pSS63 (<em>2 &#x003bc;
 DOT1-HA</em>) stained with DAPI (blue) and anti-Zip1 (green) and
 anti-HA (red) antibodies. Bar, 2 &#x003bc;m.</p></div></div></div><p class="p p-last">To determine whether Dot1 is free in the nucleoplasm or associated with
 chromosomes, nuclei from vegetative and meiotic cells carrying
 <em>DOT1-HA</em> on a high-copy plasmid were surface spread and
 analyzed by indirect immunofluorescence by using anti-HA antibodies. In
 spread mitotic nuclei, two patterns of Dot1-HA localization are found.
 In &#x0223c;29&#x00025; of nuclei, Dot1 is homogeneously distributed throughout
 chromatin (Figure <a href="/pmc/articles/PMC15018/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5C,</span></span><span>5</span></a>C, bottom). In the remaining nuclei, Dot1 is clearly
 enriched in the region of the nucleus corresponding to the nucleolus
 (Figure <a href="/pmc/articles/PMC15018/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5C,</span></span><span>5</span></a>C, top). Nucleolar localization of Dot1 was confirmed by
 double staining for Dot1-HA and the nucleolar protein Nsr1 (our
 unpublished results). In spread meiotic nuclei, Dot1 is uniformly
 dispersed throughout the whole DAPI-stained area during all stages of
 meiotic prophase (Figure <a href="/pmc/articles/PMC15018/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5D;</span></span><span>5</span></a>D; our unpublished results). These results
 indicate that Dot1 binds directly or indirectly to chromosomes.
 However, only a small fraction (&#x0223c;5&#x00025;) of spread nuclei (especially
 from meiotic cells) displays Dot1 staining, suggesting that the
 interaction of Dot1 with chromosomes is weak and/or sensitive to the
 spreading procedure.</p></div><div id="sec-a.q.c.g" class="sec"><h3 id="sec-a.q.c.gtitle">Dot1 Is Required for Localization of Sir2 and Pch2</h3><p class="p p-first">The checkpoint defect of <em>zip1 dot1</em> (Tables <a href="/pmc/articles/PMC15018/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Tables2</span></span><span>2</span></a> and <a href="/pmc/articles/PMC15018/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and3)</span></span><span>3</span></a>) is
 similar to that of <em>zip1 pch2</em> and <em>zip1 sir2</em>
 (<a href="#B52" rid="B52" class=" bibr popnode">San-Segundo and Roeder, 1999</a>; our unpublished results). Furthermore,
 like Sir2, Dot1 is involved in chromatin silencing (<a href="#B56" rid="B56" class=" bibr popnode">Singer <em>et
 al.</em>, 1998</a>). These observations suggest that Pch2, Sir2, and Dot1
 act at the same step in the pachytene checkpoint pathway and raise the
 possibility that proper localization of Pch2 and/or Sir2 requires Dot1.</p><p class="p p-last">To address this question, localization of Pch2 was examined in spread
 meiotic chromosomes of the <em>zip1 dot1</em> mutant. In
 <em>zip1</em>, the Pch2 protein is detected only in the rDNA region
 (Figure <a href="/pmc/articles/PMC15018/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6A,</span></span><span>6</span></a>A, top; <a href="#B52" rid="B52" class=" bibr popnode">San-Segundo and Roeder,
 1999</a>). In contrast, in the <em>zip1 dot1</em> double mutant, Pch2 is
 fairly evenly distributed throughout chromatin (Figure <a href="/pmc/articles/PMC15018/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6A,</span></span><span>6</span></a>A, bottom).
 Because Pch2 localization depends on Sir2, the possibility that Sir2 is
 also mislocalized in the absence of Dot1 was explored. In wild-type
 meiotic chromosomes, Sir2 accumulates in the nucleolus and is also
 present in foci mostly located at telomeric positions (Figure <a href="/pmc/articles/PMC15018/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6B,</span></span><span>6</span></a>B, top;
 <a href="#B52" rid="B52" class=" bibr popnode">San-Segundo and Roeder, 1999</a>). In contrast, in the <em>dot1</em>
 mutant, a significant fraction of the Sir2 protein is dispersed to
 other chromosomal locations (Figure <a href="/pmc/articles/PMC15018/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6B,</span></span><span>6</span></a>B, bottom). The localization
 pattern of Dot1 in mitotic cells is unaffected by a <em>sir2</em>
 mutation (our unpublished results). The localization dependence implies
 the following order of action for these proteins: Dot1 &#x02192; Sir2 &#x02192;
 Pch2.
 
 </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC15018/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139690864518672"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=15018_mk1001328006.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is mk1001328006.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC15018/bin/mk1001328006.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139690864518672"><a target="object" rel="noopener" href="/pmc/articles/PMC15018/figure/F6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC15018/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6</a></div><!--caption a7--><div class="caption"><p>Pch2, Sir2, and Sir3 are mislocalized in the
 absence of <em>DOT1</em>. (A) Spread meiotic nuclei from DP201
 (<em>zip1 PCH2-HA</em>, top) and DP285 (<em>zip1 dot1
 PCH2-HA</em>, bottom) stained with DAPI (blue) and anti-HA (red) and
 anti-Red1 (green) antibodies. (B) Spread pachytene nuclei from BR2495
 (wild type, top) and DP138 (<em>dot1</em>, bottom) stained with
 DAPI (blue) and anti-Sir2 (red) and anti-Zip1 (green) antibodies. (C)
 Spread mitotic nuclei from DP365 (<em>SIR3-HA</em>, top) and
 DP366 (<em>dot1 SIR3-HA</em>, bottom) stained with DAPI (blue)
 and anti-HA (red) antibodies. In all panels, arrows point to the
 nucleolar region. Bar, 2 &#x003bc;m.</p></div></div></div></div><div id="sec-a.q.c.h" class="sec sec-last"><h3 id="sec-a.q.c.htitle">Sir3 Is Mislocalized in the <em>dot1</em> Mutant</h3><p class="p p-first-last">In addition to the meiotic checkpoint function described here,
 Dot1 has been shown to be important for telomeric silencing in
 vegetative cells (<a href="#B56" rid="B56" class=" bibr popnode">Singer <em>et al.</em>, 1998</a>). The observation that
 Sir2 localization is altered in the <em>dot1</em> mutant raised the
 possibility that the defect in telomeric silencing is caused by the
 failure to recruit and/or stabilize the Sir protein complex at
 telomeres. To explore this hypothesis, the localization of Sir3 was
 studied in spread mitotic nuclei of wild type and <em>dot1</em>,
 using a functional HA-tagged version of Sir3. In wild type, a few very
 strong Sir3 foci are present that likely represent clusters of
 telomeres (<a href="#B25" rid="B25" class=" bibr popnode">Gotta <em>et al.</em>, 1996</a>; Figure <a href="/pmc/articles/PMC15018/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure6C,</span></span><span>6</span></a>C, top). Also, some
 Sir3-HA is detected in the rDNA region (the possibility of low-level
 binding of Sir3 to the rDNA has been previously reported; <a href="#B26" rid="B26" class=" bibr popnode">Gotta
 <em>et al.</em>, 1997</a>). However, in the <em>dot1</em> mutant, Sir3
 is dispersed all over chromatin (Figure <a href="/pmc/articles/PMC15018/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure6C,</span></span><span>6</span></a>C, bottom). A few stronger
 foci can be observed that perhaps correspond to telomeric clusters, but
 they clearly contain less Sir3 protein than those in wild type. Sir3 is
 also mislocalized in meiotic chromosomes from the <em>dot1</em>
 mutant (our unpublished results). Thus, localization of Sir3 (and Sir2)
 is altered in the absence of the Dot1 protein, presumably accounting
 for the defect in telomeric silencing in the <em>dot1</em> mutant.</p></div></div><div id="sec-a.q.d" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.q.dtitle" style="text-transform: uppercase;">DISCUSSION</h2><div id="sec-a.q.d.b" class="sec sec-first"><h3 id="sec-a.q.d.btitle">Dot1 Is Required for the Pachytene Checkpoint and for Preventing
 Repair of DSBs by Intersister Recombination</h3><p class="p p-first">The results presented here indicate that the product of the
 <em>DOT1</em> gene of <em>S. cerevisiae</em> is necessary for
 function of the pachytene checkpoint. The <em>dot1</em> single mutant
 shows no apparent defect in spore viability, recombination, or
 sporulation, arguing that <em>DOT1</em> is not required during an
 unperturbed meiosis. In contrast, <em>DOT1</em> is essential to
 trigger meiotic prophase arrest (or delay) when the SC fails to
 assemble due to lack of Zip1. The defects in chromosome synapsis (see
 Red1 staining in Figure <a href="/pmc/articles/PMC15018/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure6A)</span></span><span>6</span></a>A) and interhomolog exchange characteristic of
 <em>zip1</em> are still manifest, or even enhanced, in the <em>zip1
 dot1</em> double mutant. Thus, relief of the meiotic block by the
 <em>dot1</em> mutation is not due to suppression of the defects that
 trigger checkpoint-induced arrest, but rather is due to disruption of
 the checkpoint per se.</p><p>The inappropriate meiotic progression of <em>zip1 dot1</em> results
 in chromosome missegregation and the formation of aneuploid spores. The
 pattern of spore death in <em>zip1 dot1</em> resembles that of the
 <em>zip1</em> single mutant, suggesting that the cause of spore death
 in both cases is homolog nondisjunction at MI (<a href="#B63" rid="B63" class=" bibr popnode">Sym and Roeder, 1994</a>).
 The fraction of viable spores produced by <em>zip1 dot1</em>
 (&#x0223c;35&#x00025;) is higher than expected if chromosomes segregate before
 recombination intermediates are resolved (segregation before completion
 of recombination would result in chromosome breakage and extensive
 spore death). Thus, most or all Holliday junctions and DSBs that
 accumulate in <em>zip1</em> must become resolved/repaired as the
 arrest is bypassed by a <em>dot1</em> mutation. A similar situation
 (&#x0223c;40&#x00025; spore viability) is found when <em>zip1</em> meiotic arrest
 is relieved by mutation of <em>SWE1</em>, the major downstream target
 of the pachytene checkpoint (<a href="#B34" rid="B34" class=" bibr popnode">Leu and Roeder, 1999</a>). Swe1 inhibits
 cyclin-dependent kinase activity by phosphorylating tyrosine 19 of
 Cdc28 (<a href="#B9" rid="B9" class=" bibr popnode">Booher <em>et al.</em>, 1993</a>). It is unlikely that Swe1
 function directly affects the recombination machinery; however,
 recombination is completed in <em>zip1 swe1</em> (<a href="#B34" rid="B34" class=" bibr popnode">Leu and Roeder,
 1999</a>) as in <em>zip1 dot1</em>. These observations raise the
 possibility that the accumulation of recombination intermediates in
 <em>zip1</em> is the consequence, rather than the cause, of the
 meiotic arrest. The defect in SC assembly may be the primary cause of
 checkpoint-induced arrest in <em>zip1</em> strains.</p><p>Checkpoint-dependent arrest in the <em>dmc1</em> mutant also requires
 <em>DOT1</em>. Physical assays of recombination have demonstrated
 that the <em>dmc1</em> mutant accumulates processed DSBs (<a href="#B8" rid="B8" class=" bibr popnode">Bishop
 <em>et al.</em>, 1992</a>). Cytologically, the presence of recombination
 intermediates is manifested by the persistence of multiple
 Rad51-containing complexes on chromosomes. Although most spores
 produced by <em>dmc1 dot1</em> are dead, immunostaining with
 anti-Rad51 antibodies revealed that a significant fraction of
 <em>dmc1 dot1</em> cells that progress into MI have repaired the DSBs
 (i.e., Rad51 foci have disappeared). The high levels of spore
 inviability of <em>dmc1 dot1</em> can be explained if the resolution
 of recombination does not involve homologous chromosomes and,
 therefore, does not ensure proper segregation.</p><p>It has been recently shown that <em>dmc1</em> cells can use a
 Rad54-dependent pathway for repairing DSBs by using sister chromatids
 instead of homologs as templates for repair (<a href="#B2" rid="B2" class=" bibr popnode">Arbel <em>et al.</em>,
 1999</a>; <a href="#B7" rid="B7" class=" bibr popnode">Bishop <em>et al.</em>, 1999</a>). Indeed, Rad51 foci persist in
 the <em>dmc1 dot1 rad54</em> triple mutant, indicating that repair of
 DSBs in <em>dmc1 dot1</em> requires <em>RAD54</em>. However,
 despite the presence of unresolved recombination intermediates, the
 triple mutant proceeds through meiosis. The spores produced are
 immature and probably contain fragmented chromosomes, similar to the
 situation in <em>dmc1 rad24</em>. The <em>dmc1 rad54</em> double
 mutant does arrest (Figure <a href="/pmc/articles/PMC15018/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2D),</span></span><span>2</span></a>D), demonstrating that the failure of
 <em>dmc1 dot1 rad54</em> to arrest is due to the lack of Dot1 and not
 the lack of Rad54 function. These data argue that Dot1 performs two
 distinct functions. Dot1 executes a genuine checkpoint function,
 preventing meiotic cell cycle progression when recombination is
 incomplete. In addition, Dot1 blocks a Rad54-dependent pathway that
 repairs DSBs by intersister recombination when Dmc1 is absent.</p><p class="p p-last">Unlike <em>dmc1</em>, the formation and disappearance of meiotic DSBs
 in the <em>zip1</em> mutant is essentially normal, except that a
 small fraction of DSBs (&#x0223c;10&#x00025;) remains unrepaired (<a href="#B59" rid="B59" class=" bibr popnode">Storlazzi <em>et
 al.</em>, 1996</a>; <a href="#B69" rid="B69" class=" bibr popnode">Xu <em>et al.</em>, 1997</a>). In agreement with these
 findings, &#x0223c;25 foci of Rad51 persist in nuclei of the arrested
 <em>zip1</em> mutant. When <em>zip1</em> arrest is relieved by the
 <em>dot1</em> mutation, Rad51 foci disappear, consistent with the
 notion that recombination intermediates are resolved (see above).
 However, for the most part, this repair does not require Rad54, because
 only a very small number of Rad51 foci persist when <em>zip1 dot1
 rad54</em> cells enter MI. Assuming that the Rad54-dependent pathway is
 mostly involved in intersister recombination (<a href="#B2" rid="B2" class=" bibr popnode">Arbel <em>et al.</em>,
 1999</a>), this result suggests that most DSBs accumulated in
 <em>zip1</em> are already &#x0201c;committed&#x0201d; to undergo repair by using
 a nonsister chromatid. In contrast, in the <em>dmc1</em> mutant, the
 preference for interhomolog recombination is lost (<a href="#B53" rid="B53" class=" bibr popnode">Schwacha and
 Kleckner, 1997</a>), and DSBs can be repaired by sister-sister
 recombination when the block to the pathway is relieved by mutation of
 <em>DOT1</em> (this work) or <em>RED1</em> (<a href="#B69" rid="B69" class=" bibr popnode">Xu <em>et al.</em>,
 1997</a>; <a href="#B7" rid="B7" class=" bibr popnode">Bishop <em>et al.</em>, 1999</a>). The small number of
 Rad54-dependent (likely intersister) recombination events that occur in
 <em>zip1 dot1</em> may account for the further reduction in spore
 viability compared with <em>zip1</em>.</p></div><div id="sec-a.q.d.c" class="sec"><h3 id="sec-a.q.d.ctitle">Comparison of Dot1 Function with That of Other Pachytene Checkpoint
 Proteins</h3><p class="p p-first">The meiosis-specific proteins Red1 and Mek1 are also involved in
 the pachytene checkpoint (<a href="#B69" rid="B69" class=" bibr popnode">Xu <em>et al.</em>, 1997</a>; <a href="#B5" rid="B5" class=" bibr popnode">Bailis and
 Roeder, 2000b</a>). Red1 is a component of the lateral elements of the SC
 (<a href="#B57" rid="B57" class=" bibr popnode">Smith and Roeder, 1997</a>), and Mek1 is a kinase that phosphorylates Red1
 (<a href="#B3" rid="B3" class=" bibr popnode">Bailis and Roeder, 1998</a>; <a href="#B20" rid="B20" class=" bibr popnode">de los Santos and Hollingsworth, 1999</a>).
 Mutation of <em>DOT1</em> or <em>RED1</em> has similar effects when
 combined with <em>dmc1</em>: both bypass meiotic arrest, and both
 channel the repair of DSBs into a Rad54-dependent pathway (this work;
 <a href="#B69" rid="B69" class=" bibr popnode">Xu <em>et al.</em>, 1997</a>; <a href="#B7" rid="B7" class=" bibr popnode">Bishop <em>et al.</em>, 1999</a>). It has
 been proposed that Red1 (and Mek1) promote a meiosis-specific
 chromosomal context in which interhomolog recombination is monitored
 (<a href="#B69" rid="B69" class=" bibr popnode">Xu <em>et al.</em>, 1997</a>; <a href="#B6" rid="B6" class=" bibr popnode">Bailis <em>et al.</em>, 2000</a>). Recent
 findings indicate that Mek1-dependent phosphorylation of Red1,
 triggered by the presence of recombination intermediates, activates the
 pachytene checkpoint (<a href="#B5" rid="B5" class=" bibr popnode">Bailis and Roeder, 2000b</a>). Dot1 appears to be a
 chromatin-associated protein that might also contribute to chromosomal
 context. However, Dot1 is not meiosis specific; in addition, the
 phenotypes of <em>dot1</em> and <em>red1</em> single mutants are
 clearly different, suggesting that the proteins perform different
 functions. Whereas the <em>red1</em> mutant is severely defective in
 chromosome synapsis, recombination, chromosome segregation, and the
 production of viable spores (<a href="#B48" rid="B48" class=" bibr popnode">Rockmill and Roeder, 1988</a>, <a href="#B49" rid="B49" class=" bibr popnode">1990</a>),
 <em>dot1</em> is not defective in these processes.</p><p>Another set of proteins required for checkpoint-dependent arrest of
 <em>zip1</em> and <em>dmc1</em> are the DNA damage checkpoint
 proteins Rad24, Rad17, Mec1, and Ddc1 (<a href="#B38" rid="B38" class=" bibr popnode">Lydall <em>et al.</em>, 1996</a>;
 <a href="#B52" rid="B52" class=" bibr popnode">San-Segundo and Roeder, 1999</a>). It is generally believed that these
 proteins are involved in the detection of broken DNA both in mitotic
 and meiotic cells (<a href="#B39" rid="B39" class=" bibr popnode">Lydall and Weinert, 1995</a>; <a href="#B38" rid="B38" class=" bibr popnode">Lydall <em>et al.</em>,
 1996</a>; <a href="#B35" rid="B35" class=" bibr popnode">Longhese <em>et al.</em>, 1998</a>; <a href="#B68" rid="B68" class=" bibr popnode">Weinert, 1998</a>). The
 distribution of Dot1 throughout chromatin is consistent with such a
 role. However, the meiotic phenotypes of <em>dot1</em> and DNA damage
 checkpoint mutants differ in several aspects. Whereas <em>DOT1</em>
 is not required during an unperturbed meiosis, the <em>rad24</em>,
 <em>rad17</em>, and <em>mec1-1</em> single mutants show decreased
 crossing over, increased ectopic recombination, increased unequal
 sister-chromatid exchange, defective chromosome synapsis, and reduced
 spore viability (<a href="#B39" rid="B39" class=" bibr popnode">Lydall and Weinert, 1995</a>; <a href="#B38" rid="B38" class=" bibr popnode">Lydall <em>et al.</em>,
 1996</a>; <a href="#B28" rid="B28" class=" bibr popnode">Grushcow <em>et al.</em>, 1999</a>; <a href="#B65" rid="B65" class=" bibr popnode">Thompson and Stahl, 1999</a>). Both
 <em>DOT1</em> and <em>RAD24</em> are required for <em>dmc1</em>
 arrest, but the outcome of bypassing arrest is different. The
 <em>dmc1 dot1</em> mutant completes meiosis with most DSBs repaired
 via a Rad54-dependent pathway generating mature, though largely
 inviable, spores. In contrast, the Rad54 pathway appears to remain
 inactive in <em>dmc1 rad24</em>, resulting in progression through
 meiosis with unrepaired DSBs and the formation of immature spores.</p><p class="p p-last">Recent evidence indicates that the ultimate consequence of activation
 of the pachytene checkpoint is inhibition of cyclin-dependent kinase
 activity by Swe1-mediated phosphorylation of Cdc28 and limiting
 Ndt80-dependent transcription of <em>CLB1</em> (<a href="#B16" rid="B16" class=" bibr popnode">Chu and Herskowitz,
 1998</a>; <a href="#B31" rid="B31" class=" bibr popnode">Hepworth <em>et al.</em>, 1998</a>; <a href="#B34" rid="B34" class=" bibr popnode">Leu and Roeder, 1999</a>). A
 <em>zip1 swe1</em> mutant sporulates to wild-type levels but after a
 significant delay; this delay is overcome by simultaneous
 overexpression of <em>CLB1</em> (<a href="#B34" rid="B34" class=" bibr popnode">Leu and Roeder, 1999</a>). Because
 <em>zip1 dot1</em> sporulates with wild-type efficiency and kinetics,
 this implies that Dot1 function lies upstream of the bifurcation of the
 checkpoint pathway into the two regulatory branches of cyclin-dependent
 kinase activity.</p></div><div id="sec-a.q.d.d" class="sec"><h3 id="sec-a.q.d.dtitle">Localization of Pch2, Sir2, and Sir3 Depends on Dot1</h3><p class="p p-first-last">The predominantly nucleolar proteins Pch2 and Sir2 are also
 required for the pachytene checkpoint (<a href="#B52" rid="B52" class=" bibr popnode">San-Segundo and Roeder, 1999</a>),
 revealing an unexpected link between chromatin silencing and control of
 meiotic progression. Like Dot1, Pch2 and Sir2 are dispensable during a
 normal meiosis, but they are essential to prevent progression of
 meiosis in the absence of Zip1. Several lines of evidence suggest that
 these three proteins act at the same step in the pachytene checkpoint
 pathway. First, the meiotic phenotypes of <em>zip1 dot1</em> are
 similar to those of <em>zip1 pch2</em> or <em>zip1 sir2</em>; all
 three double mutants sporulate with wild-type kinetics, resulting in
 similar levels and patterns of spore viability (<a href="#B52" rid="B52" class=" bibr popnode">San-Segundo and Roeder,
 1999</a>; this work; our unpublished results). Second, like Sir2, Dot1 is
 involved in chromatin silencing (<a href="#B56" rid="B56" class=" bibr popnode">Singer <em>et al.</em>, 1998</a>).
 Third, proper localization of Pch2 and Sir2 depends on Dot1. <a href="#B56" rid="B56" class=" bibr popnode">Singer
 <em>et al.</em> (1998)</a> have shown that both Dot1 overproduction and
 <em>DOT1</em> deletion disrupt telomeric silencing, suggesting that
 Dot1 is part of a protein complex in which the proper balance of
 components is critical for silencing. Indeed, the absence of Dot1
 results in mislocalization of the Sir3 protein, which is an essential
 component of a multiprotein complex required for telomeric silencing
 (<a href="#B27" rid="B27" class=" bibr popnode">Grunstein, 1998</a>; <a href="#B37" rid="B37" class=" bibr popnode">Lustig, 1998</a>). Furthermore, overexpression of
 <em>DOT1</em> in <em>zip1</em> cells results in partial bypass of
 the meiotic arrest (our unpublished results). These considerations lead
 to the speculation that Dot1 is a critical component of protein
 complexes assembled in heterochromatic regions (nucleolus and/or
 telomeres) with functions in checkpoint and/or silencing processes. In
 vegetative cells, Dot1 accumulates in the nucleolus. In meiotic nuclei,
 Dot1 is also present, but not enriched, in the rDNA region. However,
 the localization data for Dot1 in spread nuclei should be considered
 with caution because they are based on overproduction of the protein.
 The fact that Dot1 does not seem to be localized exclusively in
 heterochromatic regions is not incompatible with a role in silencing;
 other essential components of silenced chromatin, such as histones H3
 and H4, and Rap1, are also present in both silenced and nonsilenced
 regions (<a href="#B54" rid="B54" class=" bibr popnode">Shore, 1994</a>; <a href="#B27" rid="B27" class=" bibr popnode">Grunstein, 1998</a>).</p></div><div id="sec-a.q.d.e" class="sec sec-last"><h3 id="sec-a.q.d.etitle">Considerations about Heterochromatin, Checkpoints, and DNA Damage
 Responses</h3><p class="p p-first">The silencing factor Sir2 is required for meiotic arrest of the
 <em>zip1</em> mutant, demonstrating a connection between chromatin
 silencing and the pachytene checkpoint. The checkpoint defect of
 <em>sir2</em> is presumably due to the failure to localize the
 meiosis-specific Pch2 protein to the nucleolus (<a href="#B52" rid="B52" class=" bibr popnode">San-Segundo and Roeder,
 1999</a>). These findings are extended by the results presented here
 demonstrating a role for another silencing protein, Dot1, in meiotic
 checkpoint control. In the absence of Dot1, the nucleolar confinement
 of Sir2 and Pch2 is lost, and the pachytene checkpoint is inactive. The
 involvement of nucleolar proteins in checkpoint mechanisms is not an
 exclusive feature of meiotic cells. The human Rad17 protein (homolog of
 the <em>S. cerevisiae</em> Rad24 checkpoint protein) localizes to the
 nucleolus in mitotic cells and is redistributed throughout the nucleus
 upon UV irradiation (<a href="#B12" rid="B12" class=" bibr popnode">Chang <em>et al.</em>, 1999</a>). In budding yeast,
 the cellular localization of most DNA damage checkpoint proteins,
 including Rad24, remains to be established.</p><p>The precise role of the nucleolus in the pachytene checkpoint is not
 yet understood. However, recent studies have demonstrated a role for
 the nucleolus in another cell-cycle regulatory process, namely the exit
 from mitosis (reviewed by <a href="#B23" rid="B23" class=" bibr popnode">Garcia and Pillus, 1999</a>). The Cdc14
 phosphatase is sequestered in the nucleolus during most of the mitotic
 cell cycle by association with the Net1/Cfi1 protein, but Cdc14 is
 released from the nucleolus in late anaphase, dispersing throughout the
 cell and triggering mitotic exit (<a href="#B55" rid="B55" class=" bibr popnode">Shou <em>et al.</em>, 1999</a>;
 <a href="#B67" rid="B67" class=" bibr popnode">Visintin <em>et al.</em>, 1999</a>). Importantly, localization of Sir2 to
 the rDNA also requires Net1, and the <em>net1</em> mutant is
 defective in rDNA silencing (<a href="#B60" rid="B60" class=" bibr popnode">Straight <em>et al.</em>, 1999</a>). It will
 be of interest to investigate whether Net1, and perhaps Cdc14, are
 implicated in the pachytene checkpoint.</p><p>Several observations also point to a link between telomeric
 heterochromatin and cellular responses to DSBs/DNA damage in both
 mitotic and meiotic cells. The Pch2 protein is normally found in the
 nucleolus, but it localizes to telomeres under some circumstances
 (<em>SIR4</em> overexpression or rDNA deletion). Under these
 conditions, Pch2 still provides at least partial checkpoint function
 (<a href="#B52" rid="B52" class=" bibr popnode">San-Segundo and Roeder, 1999</a>), arguing that some feature of
 heterochromatin is important for the pachytene checkpoint. A physical
 interaction between Mec3 (a DNA damage checkpoint protein) and Set1 (a
 protein required for telomeric silencing) has been reported (<a href="#B17" rid="B17" class=" bibr popnode">Corda
 <em>et al.</em>, 1999</a>). In addition, analysis of telomeric silencing,
 telomere length and DNA repair revealed genetic interactions between
 <em>mec3</em> and <em>set1</em> mutants (<a href="#B17" rid="B17" class=" bibr popnode">Corda <em>et al.</em>,
 1999</a>). In fission yeast, mutations in the
 <em>rad26</em><sup>&#x0002b;</sup>, <em>rad1</em><sup>&#x0002b;</sup>,
 <em>rad17</em><sup>&#x0002b;</sup>, and
 <em>rad3</em><sup>&#x0002b;</sup> checkpoint genes (the last three
 genes are homologs of <em>S. cerevisiae RAD17</em>, <em>RAD24</em>,
 and <em>MEC1</em>, respectively), result in telomere shortening
 (<a href="#B19" rid="B19" class=" bibr popnode">Dahlen <em>et al.</em>, 1998</a>) and, in the case of the
 <em>rad3</em> mutant, reduced telomeric silencing (<a href="#B42" rid="B42" class=" bibr popnode">Matsuura <em>et
 al.</em>, 1999</a>). A mutation in <em>MEC1</em> also leads to shorter
 telomeres (<a href="#B47" rid="B47" class=" bibr popnode">Ritchie <em>et al.</em>, 1999</a>) and impaired telomeric
 silencing (<a href="#B18" rid="B18" class=" bibr popnode">Craven and Petes, 2000</a>). The Sir proteins have not been
 implicated in the DNA damage checkpoint, but they do facilitate
 (directly or indirectly) the repair of DSBs (reviewed by <a href="#B29" rid="B29" class=" bibr popnode">Haber, 1999</a>).
 The Sir proteins, as well as the Ku complex, are redistributed from
 telomeres to the sites of DSBs. This relocalization requires the
 function of DNA damage checkpoint proteins and results in decreased
 telomeric silencing (<a href="#B41" rid="B41" class=" bibr popnode">Martin <em>et al.</em>, 1999</a>; <a href="#B43" rid="B43" class=" bibr popnode">McAnish <em>et
 al.</em>, 1999</a>; <a href="#B45" rid="B45" class=" bibr popnode">Mills <em>et al.</em>, 1999</a>). The Sir2-related Hst3
 and Hst4 proteins of <em>S. cerevisiae</em> and Hst4 of
 <em>Schizosaccharomyces pombe</em> also contribute to radiation
 resistance (<a href="#B10" rid="B10" class=" bibr popnode">Brachmann <em>et al.</em>, 1995</a>; <a href="#B22" rid="B22" class=" bibr popnode">Freeman-Cook <em>et
 al.</em>, 1999</a>). In consonance with all these observations, the
 <em>dot1</em> mutant, which shows reduced telomeric silencing and
 altered telomere length (<a href="#B56" rid="B56" class=" bibr popnode">Singer <em>et al.</em>, 1998</a>), also displays
 defects in the localization of Sir proteins and in the meiotic
 checkpoint that responds to unrepaired DSBs. Moreover, mutation of
 <em>DOT1</em> increases the DNA damage sensitivity of the
 <em>rad24</em> checkpoint mutant (San-Segundo and Roeder, unpublished
 observations), suggesting that Dot1 also participates in the DNA damage
 response in vegetative cells.</p><p class="p p-last">The connection between telomere function and cell-cycle checkpoints is
 not restricted to yeast. Human cells deficient in the ATM
 checkpoint protein undergo telomere shortening (<a href="#B44" rid="B44" class=" bibr popnode">Metcalfe <em>et
 al.</em>, 1996</a>). Mutation of the <em>mrt-2</em> gene of <em>C.
 elegans</em> (homolog of <em>RAD17</em> in <em>S. cerevisiae</em>)
 results in defects in telomere maintenance and defective responses to
 the accumulation of meiotic recombination intermediates and DNA damage
 (<a href="#B1" rid="B1" class=" bibr popnode">Ahmed and Hodgkin, 2000</a>; <a href="#B24" rid="B24" class=" bibr popnode">Gartner <em>et al.</em>, 2000</a>). It is
 tempting to speculate that the metazoan proteins that share homology
 with Dot1 (Figure <a href="/pmc/articles/PMC15018/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1)</span></span><span>1</span></a>) have roles in heterochromatin assembly, telomere
 metabolism, meiotic checkpoint control, and/or cellular responses to
 damaged DNA.</p></div></div><div id="ack-a.r.b" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.r.btitle" style="text-transform: uppercase;">ACKNOWLEDGMENTS</h2><div class="sec"><p>We thank Julie Bailis, Erica Hong, Janet Novak, and Beth
 Rockmill for comments on the manuscript. We also thank Elisa Stone and
 Lorraine Pillus for anti-Sir2 serum, Doug Bishop for anti-Rad51
 antibody, David Schild for plasmid pSM31, and Ted Weinert for a
 <em>rad24</em> disruption plasmid. P.A.S. was supported in part by a
 postdoctoral fellowship from the Ministerio de Educaci&#x000f3;n y
 Ciencia of Spain. This work was supported by National Institutes of
 Health grant GM-28904 (to G.S.R.) and by the Howard Hughes Medical
 Institute.</p></div></div><div id="glossary-a.r.c" class="tsec sec"><h2 class="head no_bottom_margin" id="glossary-a.r.ctitle" style="text-transform: uppercase;">Abbreviations used:</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td id="G1">DAPI</td><td>4&#x02032;-6-diamidino-2-phenylindole</td></tr><tr><td id="G2">DSB</td><td>double-strand break</td></tr><tr><td id="G3">GFP</td><td>green fluorescent protein</td></tr><tr><td id="G4">HA</td><td>hemagglutinin</td></tr><tr><td id="G5">MI</td><td>meiosis I</td></tr><tr><td id="G6">NLS</td><td>nuclear localization signal</td></tr><tr><td id="G7">PBS</td><td>phosphate-buffered saline</td></tr><tr><td id="G8">rDNA</td><td>ribosomal DNA</td></tr><tr><td id="G9">SC</td><td>synaptonemal
 complex</td></tr></tbody></table></div></div></div><div id="ref-list-a.r.d" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.r.dtitle" style="text-transform: uppercase;">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><ul class="first-line-outdent"><li id="B1"><span class="citation">Ahmed S, Hodgkin J. MRT-2 checkpoint protein is required for germline immortality and telomere replication in <em>C. elegans</em>. <span><span class="ref-journal">Nature. </span>2000;<span class="ref-vol">403</span>:159–164.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10646593" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=MRT-2+checkpoint+protein+is+required+for+germline+immortality+and+telomere+replication+in+C.+elegans&amp;author=S+Ahmed&amp;author=J+Hodgkin&amp;volume=403&amp;publication_year=2000&amp;pages=159-164&amp;pmid=10646593&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B2"><span class="citation">Arbel A, Zenvirth D, Simchen G. Sister chromatid-based DNA repair is mediated by <em>RAD54</em>, not by <em>DMC1</em> or. TID1. <span><span class="ref-journal">EMBO J. </span>1999;<span class="ref-vol">18</span>:2648–2658.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1171344/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10228176" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J&amp;title=Sister+chromatid-based+DNA+repair+is+mediated+by+RAD54,+not+by+DMC1+or.+TID1&amp;author=A+Arbel&amp;author=D+Zenvirth&amp;author=G+Simchen&amp;volume=18&amp;publication_year=1999&amp;pages=2648-2658&amp;pmid=10228176&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B3"><span class="citation">Bailis JM, Roeder GS. Synaptonemal complex morphogenesis and sister-chromatid cohesion require Mek1-dependent phosphorylation of a meiotic chromosomal protein. <span><span class="ref-journal">Genes Dev. </span>1998;<span class="ref-vol">12</span>:3551–3563.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC317243/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9832507" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;title=Synaptonemal+complex+morphogenesis+and+sister-chromatid+cohesion+require+Mek1-dependent+phosphorylation+of+a+meiotic+chromosomal+protein&amp;author=JM+Bailis&amp;author=GS+Roeder&amp;volume=12&amp;publication_year=1998&amp;pages=3551-3563&amp;pmid=9832507&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B4"><span class="citation">Bailis JM, Roeder GS. The pachytene checkpoint. <span><span class="ref-journal">Trends Genet. </span>2000a;<span class="ref-vol">16</span>:395–403.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10973068" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet&amp;title=The+pachytene+checkpoint&amp;author=JM+Bailis&amp;author=GS+Roeder&amp;volume=16&amp;publication_year=2000a&amp;pages=395-403&amp;pmid=10973068&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B5"><span class="citation">Bailis JM, Roeder GS. Pachytene exit controlled by reversal of Mek1-dependent phosphorylation. <span><span class="ref-journal">Cell. </span>2000b;<span class="ref-vol">101</span>:211–221.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10786836" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Pachytene+exit+controlled+by+reversal+of+Mek1-dependent+phosphorylation&amp;author=JM+Bailis&amp;author=GS+Roeder&amp;volume=101&amp;publication_year=2000b&amp;pages=211-221&amp;pmid=10786836&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B6"><span class="citation">Bailis JM, Smith AV, Roeder GS. Bypass of a meiotic checkpoint by overproduction of meiotic chromosomal proteins. <span><span class="ref-journal">Mol Cell Biol. </span>2000;<span class="ref-vol">20</span>:4838–4848.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC85935/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10848609" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=Bypass+of+a+meiotic+checkpoint+by+overproduction+of+meiotic+chromosomal+proteins&amp;author=JM+Bailis&amp;author=AV+Smith&amp;author=GS+Roeder&amp;volume=20&amp;publication_year=2000&amp;pages=4838-4848&amp;pmid=10848609&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B7"><span class="citation">Bishop DK, Nikolski Y, Oshiro J, Chon J, Shinohara M, Chen X. High copy number suppression of the meiotic arrest caused by a <em>dmc1</em> mutation: <em>REC114</em> imposes an early recombination block and <em>RAD54</em> promotes a <em>DMC1-</em>independent DSB repair pathway. <span><span class="ref-journal">Genes Cell. </span>1999;<span class="ref-vol">4</span>:425–443.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10526232" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Cell&amp;title=High+copy+number+suppression+of+the+meiotic+arrest+caused+by+a+dmc1+mutation:+REC114+imposes+an+early+recombination+block+and+RAD54+promotes+a+DMC1-independent+DSB+repair+pathway&amp;author=DK+Bishop&amp;author=Y+Nikolski&amp;author=J+Oshiro&amp;author=J+Chon&amp;author=M+Shinohara&amp;volume=4&amp;publication_year=1999&amp;pages=425-443&amp;pmid=10526232&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B8"><span class="citation">Bishop DK, Park D, Xu L, Kleckner N. <em>DMC1</em>: a meiosis-specific yeast homolog of <em>E. coli recA</em> required for recombination, synaptonemal complex formation, and cell cycle progression. <span><span class="ref-journal">Cell. </span>1992;<span class="ref-vol">69</span>:439–456.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1581960" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=DMC1:+a+meiosis-specific+yeast+homolog+of+E.+coli+recA+required+for+recombination,+synaptonemal+complex+formation,+and+cell+cycle+progression&amp;author=DK+Bishop&amp;author=D+Park&amp;author=L+Xu&amp;author=N+Kleckner&amp;volume=69&amp;publication_year=1992&amp;pages=439-456&amp;pmid=1581960&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B9"><span class="citation">Booher RN, Deshaies RJ, Kirschner MW. Properties of <em>Saccharomyces cerevisiae</em> wee1 and its differential regulation of p34<sup>CDC28</sup> in response to G<sub>1</sub> and G<sub>2</sub> cyclins. <span><span class="ref-journal">EMBO J. </span>1993;<span class="ref-vol">12</span>:3417–3426.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC413617/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8253069" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J&amp;title=Properties+of+Saccharomyces+cerevisiae+wee1+and+its+differential+regulation+of+p34CDC28+in+response+to+G1+and+G2+cyclins&amp;author=RN+Booher&amp;author=RJ+Deshaies&amp;author=MW+Kirschner&amp;volume=12&amp;publication_year=1993&amp;pages=3417-3426&amp;pmid=8253069&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B10"><span class="citation">Brachmann CB, Sherman JM, Devine SE, Cameron EE, Pillus L, Boeke JD. The <em>SIR2</em> gene family, conserved from bacteria to humans, functions in silencing, cell cycle progression, and chromosome stability. <span><span class="ref-journal">Genes Dev. </span>1995;<span class="ref-vol">9</span>:2888–2902.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7498786" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;title=The+SIR2+gene+family,+conserved+from+bacteria+to+humans,+functions+in+silencing,+cell+cycle+progression,+and+chromosome+stability&amp;author=CB+Brachmann&amp;author=JM+Sherman&amp;author=SE+Devine&amp;author=EE+Cameron&amp;author=L+Pillus&amp;volume=9&amp;publication_year=1995&amp;pages=2888-2902&amp;pmid=7498786&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B11"><span class="citation">Briza P, Breitenbach M, Ellinger A, Segall J. Isolation of two developmentally regulated genes involved in spore wall maturation in <em>Saccharomyces cerevisiae</em>. <span><span class="ref-journal">Genes Dev. </span>1990;<span class="ref-vol">4</span>:1775–1789.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2249774" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;title=Isolation+of+two+developmentally+regulated+genes+involved+in+spore+wall+maturation+in+Saccharomyces+cerevisiae&amp;author=P+Briza&amp;author=M+Breitenbach&amp;author=A+Ellinger&amp;author=J+Segall&amp;volume=4&amp;publication_year=1990&amp;pages=1775-1789&amp;pmid=2249774&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B12"><span class="citation">Chang M-S, Sasaki H, Campbell MS, Kraeft S-K, Sutherland R, Yang C-Y, Liu Y, Auclair D, Hao L, Sonoda H, Ferland LH, Chen LB. HRad17 colocalizes with NHP2L1 in the nucleolus and redistributes after UV irradiation. <span><span class="ref-journal">J Biol Chem. </span>1999;<span class="ref-vol">274</span>:36544–36549.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10593953" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=HRad17+colocalizes+with+NHP2L1+in+the+nucleolus+and+redistributes+after+UV+irradiation&amp;author=M-S+Chang&amp;author=H+Sasaki&amp;author=MS+Campbell&amp;author=S-K+Kraeft&amp;author=R+Sutherland&amp;volume=274&amp;publication_year=1999&amp;pages=36544-36549&amp;pmid=10593953&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B13"><span class="citation">Chelsky D, Ralph R, Jonak G. Sequence requirements for synthetic peptide-mediated translocation to the nucleus. <span><span class="ref-journal">Mol Cell Biol. </span>1989;<span class="ref-vol">9</span>:2487–2492.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC362321/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2668735" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=Sequence+requirements+for+synthetic+peptide-mediated+translocation+to+the+nucleus&amp;author=D+Chelsky&amp;author=R+Ralph&amp;author=G+Jonak&amp;volume=9&amp;publication_year=1989&amp;pages=2487-2492&amp;pmid=2668735&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B14"><span class="citation">Christianson TW, Sikorski RS, Dante M, Shero JH, Hieter P. Multifunctional yeast high-copy number shuttle vectors. <span><span class="ref-journal">Gene. </span>1992;<span class="ref-vol">110</span>:119–122.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1544568" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Multifunctional+yeast+high-copy+number+shuttle+vectors&amp;author=TW+Christianson&amp;author=RS+Sikorski&amp;author=M+Dante&amp;author=JH+Shero&amp;author=P+Hieter&amp;volume=110&amp;publication_year=1992&amp;pages=119-122&amp;pmid=1544568&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B15"><span class="citation">Chu S, DeRisi J, Eisen M, Mulholland J, Botstein D, Brown PO, Herskowitz I. The transcriptional program of sporulation in budding yeast. <span><span class="ref-journal">Science. </span>1998;<span class="ref-vol">282</span>:699–703.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9784122" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+transcriptional+program+of+sporulation+in+budding+yeast&amp;author=S+Chu&amp;author=J+DeRisi&amp;author=M+Eisen&amp;author=J+Mulholland&amp;author=D+Botstein&amp;volume=282&amp;publication_year=1998&amp;pages=699-703&amp;pmid=9784122&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B16"><span class="citation">Chu S, Herskowitz I. Gametogenesis in yeast is regulated by a transcriptional cascade dependent on Ndt80. <span><span class="ref-journal">Mol Cell. </span>1998;<span class="ref-vol">1</span>:685–696.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9660952" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell&amp;title=Gametogenesis+in+yeast+is+regulated+by+a+transcriptional+cascade+dependent+on+Ndt80&amp;author=S+Chu&amp;author=I+Herskowitz&amp;volume=1&amp;publication_year=1998&amp;pages=685-696&amp;pmid=9660952&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B17"><span class="citation">Corda Y, Schramke V, Longhese MP, Smokvina T, Paciotti V, Brevet V, Gilson E, Geli V. Interaction between Set1p and checkpoint protein Mec3p in DNA repair and telomere functions. <span><span class="ref-journal">Nat Genet. </span>1999;<span class="ref-vol">21</span>:204–208.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9988274" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Interaction+between+Set1p+and+checkpoint+protein+Mec3p+in+DNA+repair+and+telomere+functions&amp;author=Y+Corda&amp;author=V+Schramke&amp;author=MP+Longhese&amp;author=T+Smokvina&amp;author=V+Paciotti&amp;volume=21&amp;publication_year=1999&amp;pages=204-208&amp;pmid=9988274&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B18"><span class="citation">Craven RJ, Petes TD. Involvement of the checkpoint protein Mec1p in silencing of gene expression at telomeres in <em>Saccharomyces cerevisiae</em>. <span><span class="ref-journal">Mol Cell Biol. </span>2000;<span class="ref-vol">20</span>:2378–2384.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC85413/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10713162" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=Involvement+of+the+checkpoint+protein+Mec1p+in+silencing+of+gene+expression+at+telomeres+in+Saccharomyces+cerevisiae&amp;author=RJ+Craven&amp;author=TD+Petes&amp;volume=20&amp;publication_year=2000&amp;pages=2378-2384&amp;pmid=10713162&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B19"><span class="citation">Dahlen M, Olsson T, Kanter-Smoler G, Ramne A, Sunnerhagen P. Regulation of telomere length by checkpoint genes in <em>Schizosaccharomyces pombe</em>. <span><span class="ref-journal">Mol Biol Cell. </span>1998;<span class="ref-vol">9</span>:611–621.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC25290/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9487130" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Cell&amp;title=Regulation+of+telomere+length+by+checkpoint+genes+in+Schizosaccharomyces+pombe&amp;author=M+Dahlen&amp;author=T+Olsson&amp;author=G+Kanter-Smoler&amp;author=A+Ramne&amp;author=P+Sunnerhagen&amp;volume=9&amp;publication_year=1998&amp;pages=611-621&amp;pmid=9487130&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B20"><span class="citation">de los Santos T, Hollingsworth NM. Red1p, a <em>MEK1</em>-dependent phosphoprotein that physically interacts with Hop1p during meiosis in yeast. <span><span class="ref-journal">J Biol Chem. </span>1999;<span class="ref-vol">274</span>:1783–1790.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9880561" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Red1p,+a+MEK1-dependent+phosphoprotein+that+physically+interacts+with+Hop1p+during+meiosis+in+yeast&amp;author=T+de+los+Santos&amp;author=NM+Hollingsworth&amp;volume=274&amp;publication_year=1999&amp;pages=1783-1790&amp;pmid=9880561&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B21"><span class="citation">Dong H, Roeder GS. Organization of the yeast Zip1 protein within the central region of the synaptonemal complex. <span><span class="ref-journal">J Cell Biol. </span>2000;<span class="ref-vol">148</span>:417–426.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2174805/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10662769" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Biol&amp;title=Organization+of+the+yeast+Zip1+protein+within+the+central+region+of+the+synaptonemal+complex&amp;author=H+Dong&amp;author=GS+Roeder&amp;volume=148&amp;publication_year=2000&amp;pages=417-426&amp;pmid=10662769&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B22"><span class="citation">Freeman-Cook LL, Sherman JM, Brachmann CB, Allshire RC, Boeke JD, Pillus L. The <em>Schizosaccharomyces pombe hst4</em><sup>&#x0002b;</sup> gene is a <em>SIR2</em> homologue with silencing and centromeric functions. <span><span class="ref-journal">Mol Biol Cell. </span>1999;<span class="ref-vol">10</span>:3171–3186.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC25575/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10512858" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Cell&amp;title=The+Schizosaccharomyces+pombe+hst4&#x0002b;+gene+is+a+SIR2+homologue+with+silencing+and+centromeric+functions&amp;author=LL+Freeman-Cook&amp;author=JM+Sherman&amp;author=CB+Brachmann&amp;author=RC+Allshire&amp;author=JD+Boeke&amp;volume=10&amp;publication_year=1999&amp;pages=3171-3186&amp;pmid=10512858&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B23"><span class="citation">Garcia SN, Pillus L. Net results of nucleolar dynamics. <span><span class="ref-journal">Cell. </span>1999;<span class="ref-vol">97</span>:825–828.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10399910" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Net+results+of+nucleolar+dynamics&amp;author=SN+Garcia&amp;author=L+Pillus&amp;volume=97&amp;publication_year=1999&amp;pages=825-828&amp;pmid=10399910&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B24"><span class="citation">Gartner A, Milstein S, Ahmed S, Hodgkin J, Hengartner MO. A conserved checkpoint pathway mediates DNA damage-induced apoptosis and cell cycle arrest in <em>C. elegans</em>. <span><span class="ref-journal">Mol Cell. </span>2000;<span class="ref-vol">5</span>:435–443.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10882129" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell&amp;title=A+conserved+checkpoint+pathway+mediates+DNA+damage-induced+apoptosis+and+cell+cycle+arrest+in+C.+elegans&amp;author=A+Gartner&amp;author=S+Milstein&amp;author=S+Ahmed&amp;author=J+Hodgkin&amp;author=MO+Hengartner&amp;volume=5&amp;publication_year=2000&amp;pages=435-443&amp;pmid=10882129&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B25"><span class="citation">Gotta M, Laroche T, Formenton A, Maillet L, Scherthan H, Gasser SM. The clustering of telomeres and colocalization with Rap1, Sir3, and Sir4 proteins in wild-type <em>Saccharomyces cerevisiae</em>. <span><span class="ref-journal">J Cell Biol. </span>1996;<span class="ref-vol">134</span>:1349–1363.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2121006/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8830766" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Biol&amp;title=The+clustering+of+telomeres+and+colocalization+with+Rap1,+Sir3,+and+Sir4+proteins+in+wild-type+Saccharomyces+cerevisiae&amp;author=M+Gotta&amp;author=T+Laroche&amp;author=A+Formenton&amp;author=L+Maillet&amp;author=H+Scherthan&amp;volume=134&amp;publication_year=1996&amp;pages=1349-1363&amp;pmid=8830766&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B26"><span class="citation">Gotta M, Strahl-Bolsinger S, Renauld H, Laroche T, Kennedy BK, Grunstein M, Gasser SM. Localization of Sir2p: the nucleolus as a compartment for silent information regulators. <span><span class="ref-journal">EMBO J. </span>1997;<span class="ref-vol">16</span>:3243–3255.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1169941/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9214640" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J&amp;title=Localization+of+Sir2p:+the+nucleolus+as+a+compartment+for+silent+information+regulators&amp;author=M+Gotta&amp;author=S+Strahl-Bolsinger&amp;author=H+Renauld&amp;author=T+Laroche&amp;author=BK+Kennedy&amp;volume=16&amp;publication_year=1997&amp;pages=3243-3255&amp;pmid=9214640&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B27"><span class="citation">Grunstein M. Yeast heterochromatin: regulation of its assembly and inheritance by histones. <span><span class="ref-journal">Cell. </span>1998;<span class="ref-vol">93</span>:325–328.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9590166" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Yeast+heterochromatin:+regulation+of+its+assembly+and+inheritance+by+histones&amp;author=M+Grunstein&amp;volume=93&amp;publication_year=1998&amp;pages=325-328&amp;pmid=9590166&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B28"><span class="citation">Grushcow JM, Holzen TM, Park KJ, Weinert T, Lichten M, Bishop DK. <em>Saccharomyces cerevisiae</em> checkpoint genes <em>MEC1, RAD17</em> and <em>RAD24</em> are required for normal meiotic recombination partner choice. <span><span class="ref-journal">Genetics. </span>1999;<span class="ref-vol">153</span>:607–620.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1460798/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10511543" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Saccharomyces+cerevisiae+checkpoint+genes+MEC1,+RAD17+and+RAD24+are+required+for+normal+meiotic+recombination+partner+choice&amp;author=JM+Grushcow&amp;author=TM+Holzen&amp;author=KJ+Park&amp;author=T+Weinert&amp;author=M+Lichten&amp;volume=153&amp;publication_year=1999&amp;pages=607-620&amp;pmid=10511543&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B29"><span class="citation">Haber JE. Sir-Ku-itous routes to make ends meet. <span><span class="ref-journal">Cell. </span>1999;<span class="ref-vol">97</span>:829–832.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10399911" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Sir-Ku-itous+routes+to+make+ends+meet&amp;author=JE+Haber&amp;volume=97&amp;publication_year=1999&amp;pages=829-832&amp;pmid=10399911&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B30"><span class="citation">Hartwell LH, Weinert TA. Checkpoints: controls that ensure the order of cell cycle events. <span><span class="ref-journal">Science. </span>1989;<span class="ref-vol">246</span>:629–634.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2683079" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Checkpoints:+controls+that+ensure+the+order+of+cell+cycle+events&amp;author=LH+Hartwell&amp;author=TA+Weinert&amp;volume=246&amp;publication_year=1989&amp;pages=629-634&amp;pmid=2683079&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B31"><span class="citation">Hepworth SR, Friesen H, Segall J. <em>NDT80</em> and the meiotic recombination checkpoint regulate expression of middle sporulation-specific genes in <em>Saccharomyces cerevisiae</em>. <span><span class="ref-journal">Mol Cell Biol. </span>1998;<span class="ref-vol">18</span>:5750–5761.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC109161/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9742092" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=NDT80+and+the+meiotic+recombination+checkpoint+regulate+expression+of+middle+sporulation-specific+genes+in+Saccharomyces+cerevisiae&amp;author=SR+Hepworth&amp;author=H+Friesen&amp;author=J+Segall&amp;volume=18&amp;publication_year=1998&amp;pages=5750-5761&amp;pmid=9742092&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B32"><span class="citation">Johnston LH, Lowndes NF. Cell cycle control of DNA synthesis in budding yeast. <span><span class="ref-journal">Nucleic Acids Res. </span>1992;<span class="ref-vol">20</span>:2403–2410.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC312371/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1598198" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Cell+cycle+control+of+DNA+synthesis+in+budding+yeast&amp;author=LH+Johnston&amp;author=NF+Lowndes&amp;volume=20&amp;publication_year=1992&amp;pages=2403-2410&amp;pmid=1598198&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B33"><span class="citation">Koch C, Nasmyth K. Cell cycle regulated transcription in yeast. <span><span class="ref-journal">Curr Opin Cell Biol. </span>1994;<span class="ref-vol">6</span>:451–459.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7917338" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Cell+Biol&amp;title=Cell+cycle+regulated+transcription+in+yeast&amp;author=C+Koch&amp;author=K+Nasmyth&amp;volume=6&amp;publication_year=1994&amp;pages=451-459&amp;pmid=7917338&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B34"><span class="citation">Leu J-Y, Roeder GS. The pachytene checkpoint in <em>S. cerevisiae</em> depends on Swe1-mediated phosphorylation of the cyclin-dependent kinase Cdc28. <span><span class="ref-journal">Mol Cell. </span>1999;<span class="ref-vol">4</span>:805–814.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10619027" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell&amp;title=The+pachytene+checkpoint+in+S.+cerevisiae+depends+on+Swe1-mediated+phosphorylation+of+the+cyclin-dependent+kinase+Cdc28&amp;author=J-Y+Leu&amp;author=GS+Roeder&amp;volume=4&amp;publication_year=1999&amp;pages=805-814&amp;pmid=10619027&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B35"><span class="citation">Longhese MP, Foiani M, Muzi-Falconi M, Lucchini G, Plevani P. DNA damage checkpoint in budding yeast. <span><span class="ref-journal">EMBO J. </span>1998;<span class="ref-vol">17</span>:5525–5528.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1170880/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9755152" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J&amp;title=DNA+damage+checkpoint+in+budding+yeast&amp;author=MP+Longhese&amp;author=M+Foiani&amp;author=M+Muzi-Falconi&amp;author=G+Lucchini&amp;author=P+Plevani&amp;volume=17&amp;publication_year=1998&amp;pages=5525-5528&amp;pmid=9755152&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B36"><span class="citation">Lowndes NF, Murguia JR. Sensing and responding to DNA damage. <span><span class="ref-journal">Curr Opin Genet Dev. </span>2000;<span class="ref-vol">10</span>:17–25.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10679395" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Genet+Dev&amp;title=Sensing+and+responding+to+DNA+damage&amp;author=NF+Lowndes&amp;author=JR+Murguia&amp;volume=10&amp;publication_year=2000&amp;pages=17-25&amp;pmid=10679395&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B37"><span class="citation">Lustig AJ. Mechanisms of silencing in <em>Saccharomyces cerevisiae</em>. <span><span class="ref-journal">Curr Opin Genet Dev. </span>1998;<span class="ref-vol">8</span>:233–239.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9610415" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Genet+Dev&amp;title=Mechanisms+of+silencing+in+Saccharomyces+cerevisiae&amp;author=AJ+Lustig&amp;volume=8&amp;publication_year=1998&amp;pages=233-239&amp;pmid=9610415&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B38"><span class="citation">Lydall D, Nikolsky Y, Bishop DK, Weinert T. A meiotic recombination checkpoint controlled by mitotic checkpoint genes. <span><span class="ref-journal">Nature. </span>1996;<span class="ref-vol">383</span>:840–843.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8893012" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+meiotic+recombination+checkpoint+controlled+by+mitotic+checkpoint+genes&amp;author=D+Lydall&amp;author=Y+Nikolsky&amp;author=DK+Bishop&amp;author=T+Weinert&amp;volume=383&amp;publication_year=1996&amp;pages=840-843&amp;pmid=8893012&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B39"><span class="citation">Lydall D, Weinert T. Yeast checkpoint genes in DNA damage processing: implications for repair and arrest. <span><span class="ref-journal">Science. </span>1995;<span class="ref-vol">270</span>:1488–1491.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7491494" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Yeast+checkpoint+genes+in+DNA+damage+processing:+implications+for+repair+and+arrest&amp;author=D+Lydall&amp;author=T+Weinert&amp;volume=270&amp;publication_year=1995&amp;pages=1488-1491&amp;pmid=7491494&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B40"><span class="citation">Lydall D, Weinert T. G2/M checkpoint genes of <em>Saccharomyces cerevisiae</em>: further evidence for roles in DNA replication and/or repair. <span><span class="ref-journal">Mol Gen Genet. </span>1997;<span class="ref-vol">256</span>:638–651.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9435789" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Gen+Genet&amp;title=G2/M+checkpoint+genes+of+Saccharomyces+cerevisiae:+further+evidence+for+roles+in+DNA+replication+and/or+repair&amp;author=D+Lydall&amp;author=T+Weinert&amp;volume=256&amp;publication_year=1997&amp;pages=638-651&amp;pmid=9435789&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B41"><span class="citation">Martin SG, Laroche T, Suka N, Grunstein M, Gasser SM. Relocalization of telomeric Ku and SIR proteins in response to DNA strand breaks in yeast. <span><span class="ref-journal">Cell. </span>1999;<span class="ref-vol">97</span>:621–633.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10367891" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Relocalization+of+telomeric+Ku+and+SIR+proteins+in+response+to+DNA+strand+breaks+in+yeast&amp;author=SG+Martin&amp;author=T+Laroche&amp;author=N+Suka&amp;author=M+Grunstein&amp;author=SM+Gasser&amp;volume=97&amp;publication_year=1999&amp;pages=621-633&amp;pmid=10367891&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B42"><span class="citation">Matsuura A, Naito T, Ishikawa F. Genetic control of telomere integrity in <em>Schizosaccharomyces pombe</em>: <em>rad3</em><sup>&#x0002b;</sup> and <em>tel1</em><sup>&#x0002b;</sup> are parts of two regulatory networks independent of the downstream protein kinases <em>chk1</em><sup>&#x0002b;</sup> and <em>cds1</em><sup>&#x0002b;</sup> <span><span class="ref-journal">Genetics. </span>1999;<span class="ref-vol">152</span>:1501–1512.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1460706/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10430579" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genetic+control+of+telomere+integrity+in+Schizosaccharomyces+pombe:+rad3&#x0002b;+and+tel1&#x0002b;+are+parts+of+two+regulatory+networks+independent+of+the+downstream+protein+kinases+chk1&#x0002b;+and+cds1&#x0002b;&amp;author=A+Matsuura&amp;author=T+Naito&amp;author=F+Ishikawa&amp;volume=152&amp;publication_year=1999&amp;pages=1501-1512&amp;pmid=10430579&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B43"><span class="citation">McAnish AD, Scott-Drew S, Murray JA, Jackson SP. DNA damage triggers disruption of telomeric silencing and Mec1p-dependent relocation of Sir3p. <span><span class="ref-journal">Curr Biol. </span>1999;<span class="ref-vol">9</span>:963–966.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10508591" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Biol&amp;title=DNA+damage+triggers+disruption+of+telomeric+silencing+and+Mec1p-dependent+relocation+of+Sir3p&amp;author=AD+McAnish&amp;author=S+Scott-Drew&amp;author=JA+Murray&amp;author=SP+Jackson&amp;volume=9&amp;publication_year=1999&amp;pages=963-966&amp;pmid=10508591&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B44"><span class="citation">Metcalfe JA, Parkhill J, Campbell L, Stacey M, Biggs P, Byrd PJ, Taylor AM. Accelerated telomere shortening in ataxia telangiectasia. <span><span class="ref-journal">Nat Genet. </span>1996;<span class="ref-vol">13</span>:350–353.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8673136" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Accelerated+telomere+shortening+in+ataxia+telangiectasia&amp;author=JA+Metcalfe&amp;author=J+Parkhill&amp;author=L+Campbell&amp;author=M+Stacey&amp;author=P+Biggs&amp;volume=13&amp;publication_year=1996&amp;pages=350-353&amp;pmid=8673136&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B45"><span class="citation">Mills KD, Sinclair DA, Guarente L. <em>MEC1</em>-dependent redistribution of the Sir3 silencing protein from telomeres to DNA double-strand breaks. <span><span class="ref-journal">Cell. </span>1999;<span class="ref-vol">97</span>:609–620.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10367890" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=MEC1-dependent+redistribution+of+the+Sir3+silencing+protein+from+telomeres+to+DNA+double-strand+breaks&amp;author=KD+Mills&amp;author=DA+Sinclair&amp;author=L+Guarente&amp;volume=97&amp;publication_year=1999&amp;pages=609-620&amp;pmid=10367890&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B46"><span class="citation">Pintard L, Kressler D, Lapeyre B. Spb1p is a yeast nucleolar protein associated with Nop1p and Nop58p that is able to bind <em>S</em>-adenoysyl-<span class="small-caps">l</span>-methionine in vitro. <span><span class="ref-journal">Mol Cell Biol. </span>2000;<span class="ref-vol">20</span>:1370–1381.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC85287/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10648622" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=Spb1p+is+a+yeast+nucleolar+protein+associated+with+Nop1p+and+Nop58p+that+is+able+to+bind+S-adenoysyl-l-methionine+in+vitro&amp;author=L+Pintard&amp;author=D+Kressler&amp;author=B+Lapeyre&amp;volume=20&amp;publication_year=2000&amp;pages=1370-1381&amp;pmid=10648622&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B47"><span class="citation">Ritchie KB, Mallory JC, Petes TD. Interactions of <em>TLC1</em> (which encodes the RNA subunit of telomerase), <em>TEL1</em>, and <em>MEC1</em> in regulating telomere length in the yeast <em>Saccharomyces cerevisiae</em>. <span><span class="ref-journal">Mol Cell Biol. </span>1999;<span class="ref-vol">19</span>:6065–6075.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC84515/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10454554" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=Interactions+of+TLC1+(which+encodes+the+RNA+subunit+of+telomerase),+TEL1,+and+MEC1+in+regulating+telomere+length+in+the+yeast+Saccharomyces+cerevisiae&amp;author=KB+Ritchie&amp;author=JC+Mallory&amp;author=TD+Petes&amp;volume=19&amp;publication_year=1999&amp;pages=6065-6075&amp;pmid=10454554&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B48"><span class="citation">Rockmill B, Roeder GS. <em>RED1</em>: a yeast gene required for the segregation of chromosomes during the reductional division of meiosis. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1988;<span class="ref-vol">85</span>:6057–6061.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC281904/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3413075" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=RED1:+a+yeast+gene+required+for+the+segregation+of+chromosomes+during+the+reductional+division+of+meiosis&amp;author=B+Rockmill&amp;author=GS+Roeder&amp;volume=85&amp;publication_year=1988&amp;pages=6057-6061&amp;pmid=3413075&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B49"><span class="citation">Rockmill B, Roeder GS. Meiosis in asynaptic yeast. <span><span class="ref-journal">Genetics. </span>1990;<span class="ref-vol">126</span>:563–574.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1204212/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2249756" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Meiosis+in+asynaptic+yeast&amp;author=B+Rockmill&amp;author=GS+Roeder&amp;volume=126&amp;publication_year=1990&amp;pages=563-574&amp;pmid=2249756&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B50"><span class="citation">Rockmill B, Sym M, Scherthan H, Roeder GS. Roles for two RecA homologs in promoting meiotic chromosome synapsis. <span><span class="ref-journal">Genes Dev. </span>1995;<span class="ref-vol">9</span>:2684–2695.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7590245" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;title=Roles+for+two+RecA+homologs+in+promoting+meiotic+chromosome+synapsis&amp;author=B+Rockmill&amp;author=M+Sym&amp;author=H+Scherthan&amp;author=GS+Roeder&amp;volume=9&amp;publication_year=1995&amp;pages=2684-2695&amp;pmid=7590245&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B51"><span class="citation">Roeder GS. Meiotic chromosomes: it takes two to tango. <span><span class="ref-journal">Genes Dev. </span>1997;<span class="ref-vol">11</span>:2600–2621.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9334324" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;title=Meiotic+chromosomes:+it+takes+two+to+tango&amp;author=GS+Roeder&amp;volume=11&amp;publication_year=1997&amp;pages=2600-2621&amp;pmid=9334324&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B52"><span class="citation">San-Segundo P, Roeder GS. Pch2 links chromatin silencing to meiotic checkpoint control. <span><span class="ref-journal">Cell. </span>1999;<span class="ref-vol">97</span>:313–324.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10319812" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Pch2+links+chromatin+silencing+to+meiotic+checkpoint+control&amp;author=P+San-Segundo&amp;author=GS+Roeder&amp;volume=97&amp;publication_year=1999&amp;pages=313-324&amp;pmid=10319812&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B53"><span class="citation">Schwacha A, Kleckner N. Interhomolog bias during meiotic recombination: meiotic functions promote a highly differentiated interhomolog-only pathway. <span><span class="ref-journal">Cell. </span>1997;<span class="ref-vol">90</span>:1123–1136.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9323140" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Interhomolog+bias+during+meiotic+recombination:+meiotic+functions+promote+a+highly+differentiated+interhomolog-only+pathway&amp;author=A+Schwacha&amp;author=N+Kleckner&amp;volume=90&amp;publication_year=1997&amp;pages=1123-1136&amp;pmid=9323140&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B54"><span class="citation">Shore D. RAP1: a protein regulator in yeast. <span><span class="ref-journal">Trends Genet. </span>1994;<span class="ref-vol">10</span>:408–412.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7809947" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet&amp;title=RAP1:+a+protein+regulator+in+yeast&amp;author=D+Shore&amp;volume=10&amp;publication_year=1994&amp;pages=408-412&amp;pmid=7809947&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B55"><span class="citation">Shou W, Seol JH, Shevchenko A, Baskerville C, Moazed D, Chen ZWS, Jang J, Shevchenko A, Charbonneau H, Deshaies RJ. Exit from mitosis is triggered by Tem1-dependent release of the protein phosphatase Cdc14 from nucleolar RENT complex. <span><span class="ref-journal">Cell. </span>1999;<span class="ref-vol">97</span>:233–244.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10219244" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Exit+from+mitosis+is+triggered+by+Tem1-dependent+release+of+the+protein+phosphatase+Cdc14+from+nucleolar+RENT+complex&amp;author=W+Shou&amp;author=JH+Seol&amp;author=A+Shevchenko&amp;author=C+Baskerville&amp;author=D+Moazed&amp;volume=97&amp;publication_year=1999&amp;pages=233-244&amp;pmid=10219244&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B56"><span class="citation">Singer MS, Kahana A, Wolf AJ, Meisinger LL, Peterson SE, Goggin C, Mahowald M, Gottschling DE. Identification of high-copy disruptors of telomeric silencing in <em>Saccharomyces cerevisiae</em>. <span><span class="ref-journal">Genetics. </span>1998;<span class="ref-vol">150</span>:613–632.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1460361/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9755194" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Identification+of+high-copy+disruptors+of+telomeric+silencing+in+Saccharomyces+cerevisiae&amp;author=MS+Singer&amp;author=A+Kahana&amp;author=AJ+Wolf&amp;author=LL+Meisinger&amp;author=SE+Peterson&amp;volume=150&amp;publication_year=1998&amp;pages=613-632&amp;pmid=9755194&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B57"><span class="citation">Smith AV, Roeder GS. The yeast Red1 protein localizes to the cores of meiotic chromosomes. <span><span class="ref-journal">J Cell Biol. </span>1997;<span class="ref-vol">136</span>:957–967.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2132480/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9060462" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Biol&amp;title=The+yeast+Red1+protein+localizes+to+the+cores+of+meiotic+chromosomes&amp;author=AV+Smith&amp;author=GS+Roeder&amp;volume=136&amp;publication_year=1997&amp;pages=957-967&amp;pmid=9060462&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B58"><span class="citation">Spellman PT, Sherlock G, Zhang MQ, Iyer VR, Anders K, Eisen MB, Brown PO, Botstein D, Futcher B. Comprehensive identification of cell cycle-regulated genes of the yeast <em>Saccharomyces cerevisiae</em> by microarray hybridization. <span><span class="ref-journal">Mol Biol Cell. </span>1998;<span class="ref-vol">9</span>:3273–3297.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC25624/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9843569" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Cell&amp;title=Comprehensive+identification+of+cell+cycle-regulated+genes+of+the+yeast+Saccharomyces+cerevisiae+by+microarray+hybridization&amp;author=PT+Spellman&amp;author=G+Sherlock&amp;author=MQ+Zhang&amp;author=VR+Iyer&amp;author=K+Anders&amp;volume=9&amp;publication_year=1998&amp;pages=3273-3297&amp;pmid=9843569&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B59"><span class="citation">Storlazzi A, Xu L, Schwacha A, Kleckner N. Synaptonemal complex (SC) component Zip1 plays a role in meiotic recombination independent of SC polymerization along the chromosomes. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1996;<span class="ref-vol">93</span>:9043–9048.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC38592/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8799151" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Synaptonemal+complex+(SC)+component+Zip1+plays+a+role+in+meiotic+recombination+independent+of+SC+polymerization+along+the+chromosomes&amp;author=A+Storlazzi&amp;author=L+Xu&amp;author=A+Schwacha&amp;author=N+Kleckner&amp;volume=93&amp;publication_year=1996&amp;pages=9043-9048&amp;pmid=8799151&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B60"><span class="citation">Straight AF, Shou W, Dowd GJ, Turck CW, Deshaies RJ, Johnson AD, Moazed D. Net1, a Sir2-associated nucleolar protein required for rDNA silencing and nucleolar integrity. <span><span class="ref-journal">Cell. </span>1999;<span class="ref-vol">97</span>:245–256.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10219245" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Net1,+a+Sir2-associated+nucleolar+protein+required+for+rDNA+silencing+and+nucleolar+integrity&amp;author=AF+Straight&amp;author=W+Shou&amp;author=GJ+Dowd&amp;author=CW+Turck&amp;author=RJ+Deshaies&amp;volume=97&amp;publication_year=1999&amp;pages=245-256&amp;pmid=10219245&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B61"><span class="citation">Struhl K, Stinchcomb DT, Scherer S, Davis RW. High-frequency transformation of yeast: autonomous replication of hybrid DNA molecules. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1979;<span class="ref-vol">76</span>:1035–1039.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC383183/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/375221" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=High-frequency+transformation+of+yeast:+autonomous+replication+of+hybrid+DNA+molecules&amp;author=K+Struhl&amp;author=DT+Stinchcomb&amp;author=S+Scherer&amp;author=RW+Davis&amp;volume=76&amp;publication_year=1979&amp;pages=1035-1039&amp;pmid=375221&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B62"><span class="citation">Sym M, Engebrecht J, Roeder GS. ZIP1 is a synaptonemal complex protein required for meiotic chromosome synapsis. <span><span class="ref-journal">Cell. </span>1993;<span class="ref-vol">72</span>:365–378.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7916652" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=ZIP1+is+a+synaptonemal+complex+protein+required+for+meiotic+chromosome+synapsis&amp;author=M+Sym&amp;author=J+Engebrecht&amp;author=GS+Roeder&amp;volume=72&amp;publication_year=1993&amp;pages=365-378&amp;pmid=7916652&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B63"><span class="citation">Sym M, Roeder GS. Crossover interference is abolished in the absence of a synaptonemal complex protein. <span><span class="ref-journal">Cell. </span>1994;<span class="ref-vol">79</span>:283–292.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7954796" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Crossover+interference+is+abolished+in+the+absence+of+a+synaptonemal+complex+protein&amp;author=M+Sym&amp;author=GS+Roeder&amp;volume=79&amp;publication_year=1994&amp;pages=283-292&amp;pmid=7954796&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B64"><span class="citation">Sym M, Roeder GS. Zip1-induced changes in synaptonemal complex structure and polycomplex assembly. <span><span class="ref-journal">J Cell Biol. </span>1995;<span class="ref-vol">128</span>:455–466.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2199901/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7860625" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Biol&amp;title=Zip1-induced+changes+in+synaptonemal+complex+structure+and+polycomplex+assembly&amp;author=M+Sym&amp;author=GS+Roeder&amp;volume=128&amp;publication_year=1995&amp;pages=455-466&amp;pmid=7860625&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B65"><span class="citation">Thompson DA, Stahl FW. Genetic control of recombination partner preference in yeast meiosis: isolation and characterization of mutants elevated for meiotic unequal sister-chromatid recombination. <span><span class="ref-journal">Genetics. </span>1999;<span class="ref-vol">153</span>:621–641.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1460802/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10511544" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genetic+control+of+recombination+partner+preference+in+yeast+meiosis:+isolation+and+characterization+of+mutants+elevated+for+meiotic+unequal+sister-chromatid+recombination&amp;author=DA+Thompson&amp;author=FW+Stahl&amp;volume=153&amp;publication_year=1999&amp;pages=621-641&amp;pmid=10511544&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B66"><span class="citation">Tung K-S, Roeder GS. Meiotic chromosome morphology and behavior in <em>zip1</em> mutants of <em>Saccharomyces cerevisiae</em>. <span><span class="ref-journal">Genetics. </span>1998;<span class="ref-vol">149</span>:817–832.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1460213/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9611194" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Meiotic+chromosome+morphology+and+behavior+in+zip1+mutants+of+Saccharomyces+cerevisiae&amp;author=K-S+Tung&amp;author=GS+Roeder&amp;volume=149&amp;publication_year=1998&amp;pages=817-832&amp;pmid=9611194&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B67"><span class="citation">Visintin R, Hwang ES, Amon A. Cfi1 prevents premature exit from mitosis by anchoring Cdc14 phosphatase in the nucleolus. <span><span class="ref-journal">Nature. </span>1999;<span class="ref-vol">398</span>:818–823.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10235265" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Cfi1+prevents+premature+exit+from+mitosis+by+anchoring+Cdc14+phosphatase+in+the+nucleolus&amp;author=R+Visintin&amp;author=ES+Hwang&amp;author=A+Amon&amp;volume=398&amp;publication_year=1999&amp;pages=818-823&amp;pmid=10235265&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B68"><span class="citation">Weinert T. DNA damage checkpoints update: getting molecular. <span><span class="ref-journal">Curr Opin Genet Dev. </span>1998;<span class="ref-vol">8</span>:185–193.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9610409" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Genet+Dev&amp;title=DNA+damage+checkpoints+update:+getting+molecular&amp;author=T+Weinert&amp;volume=8&amp;publication_year=1998&amp;pages=185-193&amp;pmid=9610409&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B69"><span class="citation">Xu L, Weiner BM, Kleckner N. Meiotic cells monitor the status of the interhomolog recombination complex. <span><span class="ref-journal">Genes Dev. </span>1997;<span class="ref-vol">11</span>:106–118.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9000054" ref="reftype=pubmed&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;title=Meiotic+cells+monitor+the+status+of+the+interhomolog+recombination+complex&amp;author=L+Xu&amp;author=BM+Weiner&amp;author=N+Kleckner&amp;volume=11&amp;publication_year=1997&amp;pages=106-118&amp;pmid=9000054&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=15018&amp;issue-id=451&amp;journal-id=1&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li></ul></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Molecular Biology of the Cell</span> are provided here courtesy of <strong>American Society for Cell Biology</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC15018/pdf/mk003601.pdf" class="int-view">PDF (1015K)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/15018/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/15018/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="15018" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC15018%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="1sNl94qPdQf0T5AA52St7B1dcYguBvNM0UqyFFUC3doxuFcIPnwvLAZHkEjPFYuV">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC15018%2F&amp;text=Role%20for%20the%20Silencing%20Protein%20Dot1%20in%20Meiotic%20Checkpoint%20Control" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC15018%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC15018/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC15018/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/11029058/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC15018/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/11029058/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC15018/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/15018/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC15018/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
