
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.4404d67389b1.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.484f3c634d37.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.859ea76d5b06.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs2.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Conserved plant genes with similarity to mammalian de novo DNA methyltransferases - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939B37B87F61B96500004D48D80A7B5E.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="pnas" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC18343/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC18343/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Proceedings of the National Academy of Sciences of the United States of America" /><meta name="citation_title" content="Conserved plant genes with similarity to mammalian de novo DNA methyltransferases" /><meta name="citation_author" content="Xiaofeng Cao" /><meta name="citation_author_institution" content="Department of Molecular, Cell and Developmental Biology, University of California, Los Angeles, CA 90095-1606; Department of Agronomy, University of Wisconsin, Madison, WI 53706; Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108; and Pioneer Hi-Bred International, Inc., 7300 NW 62nd Avenue, Johnston, IA 50131" /><meta name="citation_author" content="Nathan M. Springer" /><meta name="citation_author_institution" content="Department of Molecular, Cell and Developmental Biology, University of California, Los Angeles, CA 90095-1606; Department of Agronomy, University of Wisconsin, Madison, WI 53706; Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108; and Pioneer Hi-Bred International, Inc., 7300 NW 62nd Avenue, Johnston, IA 50131" /><meta name="citation_author" content="Michael G. Muszynski" /><meta name="citation_author_institution" content="Department of Molecular, Cell and Developmental Biology, University of California, Los Angeles, CA 90095-1606; Department of Agronomy, University of Wisconsin, Madison, WI 53706; Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108; and Pioneer Hi-Bred International, Inc., 7300 NW 62nd Avenue, Johnston, IA 50131" /><meta name="citation_author" content="Ronald L. Phillips" /><meta name="citation_author_institution" content="Department of Molecular, Cell and Developmental Biology, University of California, Los Angeles, CA 90095-1606; Department of Agronomy, University of Wisconsin, Madison, WI 53706; Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108; and Pioneer Hi-Bred International, Inc., 7300 NW 62nd Avenue, Johnston, IA 50131" /><meta name="citation_author" content="Shawn Kaeppler" /><meta name="citation_author_institution" content="Department of Molecular, Cell and Developmental Biology, University of California, Los Angeles, CA 90095-1606; Department of Agronomy, University of Wisconsin, Madison, WI 53706; Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108; and Pioneer Hi-Bred International, Inc., 7300 NW 62nd Avenue, Johnston, IA 50131" /><meta name="citation_author" content="Steven E. Jacobsen" /><meta name="citation_author_institution" content="Department of Molecular, Cell and Developmental Biology, University of California, Los Angeles, CA 90095-1606; Department of Agronomy, University of Wisconsin, Madison, WI 53706; Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108; and Pioneer Hi-Bred International, Inc., 7300 NW 62nd Avenue, Johnston, IA 50131" /><meta name="citation_publication_date" content="2000/04/04" /><meta name="citation_issue" content="9" /><meta name="citation_volume" content="97" /><meta name="citation_firstpage" content="4979" /><meta name="citation_doi" content="10.1073/pnas.97.9.4979" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC18343/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC18343/" /><meta name="citation_pmid" content="10781108" /><meta name="DC.Title" content="Conserved plant genes with similarity to mammalian de novo DNA methyltransferases" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="National Academy of Sciences" /><meta name="DC.Contributor" content="Xiaofeng Cao" /><meta name="DC.Contributor" content="Nathan M. Springer" /><meta name="DC.Contributor" content="Michael G. Muszynski" /><meta name="DC.Contributor" content="Ronald L. Phillips" /><meta name="DC.Contributor" content="Shawn Kaeppler" /><meta name="DC.Contributor" content="Steven E. Jacobsen" /><meta name="DC.Date" content="2000 Apr 25" /><meta name="DC.Identifier" content="10.1073/pnas.97.9.4979" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Conserved plant genes with similarity to mammalian de novo DNA methyltransferases" /><meta property="og:type" content="article" /><meta property="og:description" content="DNA methylation plays a critical role in controlling states of gene activity in most eukaryotic organisms, and it is essential for proper growth and development. Patterns of methylation are established by de novo methyltransferases and maintained by maintenance ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC18343/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        The PMC website is updating on October 15, 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC18343/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/520/">v.97(9); 2000 Apr 25</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC18343
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC18343/pdf/pq004979.pdf" class="int-view">PDF (253K)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/18343/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/18343/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="18343" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC18343%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="hC8oVmdNibLOc91fvNwWBsIrAcOpbs7wBOkfXSrL0tNDVNdxlhUkm5SwUI7CnLte">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC18343%2F&amp;text=Conserved%20plant%20genes%20with%20similarity%20to%20mammalian%20de%20novo%20DNA%20methyltransferases" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC18343%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC18343/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC18343/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/10781108/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC18343/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/10781108/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC18343/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/18343/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/520/">v.97(9); 2000 Apr 25</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC18343
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><a href="//doi.org/10.1073%2Fpnas.97.9.4979" title="Link to Publisher's site" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"><img src="/corehtml/pmc/pmcgifs/logo-pnas.png" alt="Logo of pnas"><!-- --></img></a></div>
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Proc Natl Acad Sci U S A.</span> 2000 Apr 25; 97(9): 4979–4984. </div><div class="part2">  <span class="doi"><span>doi: </span><a href="//doi.org/10.1073%2Fpnas.97.9.4979" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1073/pnas.97.9.4979</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC18343</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/10781108">10781108</a></div></div></div><h1 class="content-title">Conserved plant genes with similarity to mammalian <em>de novo</em> DNA methyltransferases</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Cao%20X%5BAuthor%5D" class="affpopup" co-rid="_co_idm140626402428096" co-class="co-affbox">Xiaofeng Cao</a>,<sup>&#x02020;</sup><sup>&#x02021;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Springer%20NM%5BAuthor%5D" class="affpopup" co-rid="_co_idm140626407017408" co-class="co-affbox">Nathan M. Springer</a>,<sup>&#x02021;</sup><sup>&#x000a7;</sup><sup>&#x000b6;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Muszynski%20MG%5BAuthor%5D" class="affpopup" co-rid="_co_idm140626404670864" co-class="co-affbox">Michael G. Muszynski</a>,<sup>&#x02016;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Phillips%20RL%5BAuthor%5D" class="affpopup" co-rid="_co_idm140626404668352" co-class="co-affbox">Ronald L. Phillips</a>,<sup>&#x000b6;</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kaeppler%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm140626388868608" co-class="co-affbox">Shawn Kaeppler</a>,<sup>&#x000a7;</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jacobsen%20SE%5BAuthor%5D" class="affpopup" co-rid="_co_idm140626388866096" co-class="co-affbox">Steven E. Jacobsen</a><sup>&#x02020;</sup><sup>&#x0002a;&#x0002a;</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140626402428096"><h3 class="no_margin">Xiaofeng Cao</h3><p><sup>&#x02020;</sup>Department of Molecular, Cell and Developmental Biology, University of California, Los Angeles, CA 90095-1606; <sup>&#x000a7;</sup>Department of Agronomy, University of Wisconsin, Madison, WI 53706; <sup>&#x000b6;</sup>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108; and <sup>&#x02016;</sup>Pioneer Hi-Bred International, Inc., 7300 NW 62nd Avenue, Johnston, IA 50131</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Cao%20X%5BAuthor%5D">Xiaofeng Cao</a></div></div><div id="_co_idm140626407017408"><h3 class="no_margin">Nathan M. Springer</h3><p><sup>&#x02020;</sup>Department of Molecular, Cell and Developmental Biology, University of California, Los Angeles, CA 90095-1606; <sup>&#x000a7;</sup>Department of Agronomy, University of Wisconsin, Madison, WI 53706; <sup>&#x000b6;</sup>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108; and <sup>&#x02016;</sup>Pioneer Hi-Bred International, Inc., 7300 NW 62nd Avenue, Johnston, IA 50131</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Springer%20NM%5BAuthor%5D">Nathan M. Springer</a></div></div><div id="_co_idm140626404670864"><h3 class="no_margin">Michael G. Muszynski</h3><p><sup>&#x02020;</sup>Department of Molecular, Cell and Developmental Biology, University of California, Los Angeles, CA 90095-1606; <sup>&#x000a7;</sup>Department of Agronomy, University of Wisconsin, Madison, WI 53706; <sup>&#x000b6;</sup>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108; and <sup>&#x02016;</sup>Pioneer Hi-Bred International, Inc., 7300 NW 62nd Avenue, Johnston, IA 50131</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Muszynski%20MG%5BAuthor%5D">Michael G. Muszynski</a></div></div><div id="_co_idm140626404668352"><h3 class="no_margin">Ronald L. Phillips</h3><p><sup>&#x02020;</sup>Department of Molecular, Cell and Developmental Biology, University of California, Los Angeles, CA 90095-1606; <sup>&#x000a7;</sup>Department of Agronomy, University of Wisconsin, Madison, WI 53706; <sup>&#x000b6;</sup>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108; and <sup>&#x02016;</sup>Pioneer Hi-Bred International, Inc., 7300 NW 62nd Avenue, Johnston, IA 50131</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Phillips%20RL%5BAuthor%5D">Ronald L. Phillips</a></div></div><div id="_co_idm140626388868608"><h3 class="no_margin">Shawn Kaeppler</h3><p><sup>&#x02020;</sup>Department of Molecular, Cell and Developmental Biology, University of California, Los Angeles, CA 90095-1606; <sup>&#x000a7;</sup>Department of Agronomy, University of Wisconsin, Madison, WI 53706; <sup>&#x000b6;</sup>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108; and <sup>&#x02016;</sup>Pioneer Hi-Bred International, Inc., 7300 NW 62nd Avenue, Johnston, IA 50131</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kaeppler%20S%5BAuthor%5D">Shawn Kaeppler</a></div></div><div id="_co_idm140626388866096"><h3 class="no_margin">Steven E. Jacobsen</h3><p><sup>&#x02020;</sup>Department of Molecular, Cell and Developmental Biology, University of California, Los Angeles, CA 90095-1606; <sup>&#x000a7;</sup>Department of Agronomy, University of Wisconsin, Madison, WI 53706; <sup>&#x000b6;</sup>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108; and <sup>&#x02016;</sup>Pioneer Hi-Bred International, Inc., 7300 NW 62nd Avenue, Johnston, IA 50131</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jacobsen%20SE%5BAuthor%5D">Steven E. Jacobsen</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140626405021072_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140626405021072_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140626405021072_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140626405021072_ai" style="display:none"><div class="fm-affl" id="N0x9b87268.0x8c2dac8"><sup>&#x02020;</sup>Department of Molecular, Cell and Developmental Biology, University of California, Los Angeles, CA 90095-1606; <sup>&#x000a7;</sup>Department of Agronomy, University of Wisconsin, Madison, WI 53706; <sup>&#x000b6;</sup>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108; and <sup>&#x02016;</sup>Pioneer Hi-Bred International, Inc., 7300 NW 62nd Avenue, Johnston, IA 50131</div><div id="FN152"><sup>&#x02021;</sup>X.C. and N.M.S. contributed equally to this work.</div><div id="FN160"><sup>&#x0002a;&#x0002a;</sup>To whom reprint requests should be addressed: E-mail: <a href="mailto:dev@null" data-email="ude.alcu@nesbocaj" class="oemail">ude.alcu@nesbocaj</a>.</div><div id="fn-a.g.b.g.c">Contributed by Ronald L. Phillips</div></div><div class="fm-article-notes hide half_rhythm" id="idm140626405021072_an" style="display:none"><div class="fm-pubdate half_rhythm">Accepted 2000 Feb 25.</div></div><div class="permissions half_rhythm hide" id="idm140626405021072_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  &#x000a9; 2000, The National Academy of Sciences</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="abstract-a.g.b.p" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.g.b.ptitle" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p class="p p-first-last">DNA methylation plays a critical role in controlling states of gene activity in most eukaryotic organisms, and it is essential for proper growth and development. Patterns of methylation are established by <em>de novo</em> methyltransferases and maintained by maintenance methyltransferase activities. The Dnmt3 family of <em>de novo</em> DNA methyltransferases has recently been characterized in animals. Here we describe DNA methyltransferase genes from both <em>Arabidopsis</em> and maize that show a high level of sequence similarity to <em>Dnmt3</em>, suggesting that they encode plant <em>de novo</em> methyltransferases. Relative to all known eukaryotic methyltransferases, these plant proteins contain a novel arrangement of the motifs required for DNA methyltransferase catalytic activity. The N termini of these methyltransferases contain a series of ubiquitin-associated (UBA) domains. UBA domains are found in several ubiquitin pathway proteins and in DNA repair enzymes such as Rad23, and they may be involved in ubiquitin binding. The presence of UBA domains provides a possible link between DNA methylation and ubiquitin/proteasome pathways.</p></div></div><div id="body-a.h" class="tsec sec"><h2 class="headless nomenu"></h2><p class="p p-first">Methylation of the C5 position of cytosine is the most common covalent modification of DNA in higher plants and animals. This methylation is usually associated with transcriptional gene silencing, or so-called epigenetic gene inactivation. There are several examples of epigenetic silencing in the plant kingdom, including paramutation in maize (<a href="#B1" rid="B1" class=" bibr popnode">1</a>), <em>PAI</em> (<a href="#B2" rid="B2" class=" bibr popnode">2</a>) and <em>SUPERMAN</em> (<a href="#B3" rid="B3" class=" bibr popnode">3</a>) gene silencing in <em>Arabidopsis</em>, and transgene silencing in many plants species (<a href="#B4" rid="B4" class=" bibr popnode">4</a>). Genetic experiments in <em>Arabidopsis</em> have shown that proper DNA methylation levels are required for normal development (<a href="#B5" rid="B5" class=" bibr popnode">5</a>, <a href="#B6" rid="B6" class=" bibr popnode">6</a>). In animal systems, DNA methylation plays a prominent role in allele-specific gene expression that occurs during parental genomic imprinting and X chromosome inactivation (<a href="#B7" rid="B7" class=" bibr popnode">7</a>, <a href="#B8" rid="B8" class=" bibr popnode">8</a>). Methylation is also important in the regulation of genomic parasites such as transposable elements, retrotransposons, and retroviruses (<a href="#B9" rid="B9" class=" bibr popnode">9</a>, <a href="#B10" rid="B10" class=" bibr popnode">10</a>).</p><p>The presence of 5-methylcytosine in genomic DNA is the result of enzymatic activity of the C5 DNA methyltransferases, which catalyze the transfer of a methyl group from <em>S</em>-adenosyl-<span class="small-caps">l</span>-methionine (AdoMet). All known C5 DNA methyltransferases are characterized by the presence of several conserved motifs in the region of the protein involved in catalysis (<a href="#B11" rid="B11" class=" bibr popnode">11</a>). This conservation suggests that cytosine methyltransferases share a common evolutionary history.</p><p>There are two major types of DNA methyltransferase activities, maintenance and <em>de novo</em>. The methylation of hemimethylated symmetrical sequences (CpG and CpXpG) after DNA replication is maintenance methylation. This results in stable patterns of methylation that are maintained throughout development or, in many cases, between generations. Methylation that occurs at previously unmethylated cytosines is known as <em>de novo</em> methylation. For symmetric sites, <em>de novo</em> methylation need occur only once, after which methylation can be preserved by maintenance activity. However, for maintenance of methylation at asymmetric sites (cytosines in contexts other than CpG and CpXpG) <em>de novo</em> methylation must occur continually.</p><p>The eukaryotic DNA methyltransferases can be grouped into at least four distinct classes based on sequence homology and function, the Dnmt1&#x0002f;MET1 class, the Dnmt2 class, the CMT class, and Dnmt3 class.</p><p>The Dnmt1&#x0002f;MET1 class enzymes act primarily as maintenance methyltransferases. Dnmt1 was originally isolated from mouse (<a href="#B12" rid="B12" class=" bibr popnode">12</a>) and a Dnmt1 homolog (MET1) was subsequently found in <em>Arabidopsis</em> (<a href="#B13" rid="B13" class=" bibr popnode">13</a>). Biochemical studies with mammalian Dnmt1 and a pea MET1-like protein have shown a high level of methyltransferase activity on hemimethylated substrates relative to unmethylated substrates (<a href="#B14" rid="B14" class=" bibr popnode">14</a>&#x02013;<a href="#B16" rid="B16" class=" bibr popnode">16</a>). Genetic evidence indicates that Dnmt1 and MET1 are the predominant maintenance methyltransferases in mouse and <em>Arabidopsis</em>, respectively. Mice harboring loss-of-function <em>Dnmt1</em> mutations show only one-third the normal level of genomic methylation, and these mice die after 9 days of development (<a href="#B17" rid="B17" class=" bibr popnode">17</a>). However, nullizygous <em>Dnmt1</em> cells retain <em>de novo</em> methylation activity (<a href="#B18" rid="B18" class=" bibr popnode">18</a>). In <em>Arabidopsis,</em> transgenic plants carrying antisense <em>MET1</em> RNA constructs have a 90&#x00025; decrease in overall DNA methylation levels (<a href="#B5" rid="B5" class=" bibr popnode">5</a>, <a href="#B19" rid="B19" class=" bibr popnode">19</a>). These antisense <em>MET1</em> plants are viable but display a number of specific developmental abnormalities. Two of these abnormalities have been studied in detail and are caused by dense ectopic hypermethylation at both symmetric and asymmetric sites within the floral regulatory genes <em>SUPERMAN</em> and <em>AGAMOUS</em> (<a href="#B3" rid="B3" class=" bibr popnode">3</a>, <a href="#B20" rid="B20" class=" bibr popnode">20</a>, <a href="#B21" rid="B21" class=" bibr popnode">21</a>). Therefore, despite an overall reduction in genomic methylation, the antisense-<em>MET1</em> plants retain <em>de novo</em> methyltransferase activity.</p><p>The Dnmt2 class of DNA methyltransferases, found in mammals (Dnmt2), fission yeast (PMT1), and <em>Drosophila melanogaster</em> (DmMT2) does not appear to play a significant role in establishing or maintaining DNA methylation patterns. These proteins do not show significant <em>in vitro</em> activity, and loss-of-function <em>Dnmt2</em> mutations do not show any reduction in the amount of overall DNA methylation, or a reduction in <em>de novo</em> methylation (<a href="#B22" rid="B22" class=" bibr popnode">22</a>&#x02013;<a href="#B26" rid="B26" class=" bibr popnode">26</a>). As neither <em>Drosophila</em> nor fission yeast contain detectable amounts of cytosine methylation in their genomes, the function of these Dnmt2 methyltransferase-like proteins is unclear.</p><p>The chromomethylases (CMTs) represent a class of DNA methyltransferases that have so far been found only in plants. The distinguishing feature of CMTs is the presence of a chromodomain embedded between catalytic motifs I and IV (<a href="#B27" rid="B27" class=" bibr popnode">27</a>, <a href="#B28" rid="B28" class=" bibr popnode">28</a>). A loss-of-function mutation in a maize <em>CMT</em>-like gene, <em>ZMET2</em>, specifically reduces CpXpG methylation (Charles Papa, N.S., and S.K., unpublished results). This observation provides an explanation for the fact that plants contain a high level of CpXpG methylation in their genomes relative to animals (<a href="#B29" rid="B29" class=" bibr popnode">29</a>), and it suggests that the CMTs may act as a specialized type of plant specific maintenance methyltransferase.</p><p>Recently the Dnmt3 class of DNA methyltransferases was identified in mouse, human, and zebra fish (<a href="#B16" rid="B16" class=" bibr popnode">16</a>, <a href="#B30" rid="B30" class=" bibr popnode">30</a>). Several lines of evidence suggest that the Dnmt3 proteins act as <em>de novo</em> methyltransferases. Recombinant Dnmt3a and Dnmt3b enzymes from mouse displayed <em>de novo</em> activity when tested on unmethylated DNA templates <em>in vitro</em> (<a href="#B16" rid="B16" class=" bibr popnode">16</a>). Furthermore, expression of Dnmt3a in <em>Drosophila melanogaster</em> caused <em>de novo</em> methylation of its normally unmethylated genome (<a href="#B31" rid="B31" class=" bibr popnode">31</a>). When both <em>Dnmt3a</em> and <em>Dnmt3b</em> were inactivated by gene targeting, the resulting embryonic stem (ES) cells and early embryos lacked <em>de novo</em> methylation activity (<a href="#B32" rid="B32" class=" bibr popnode">32</a>). Mutations in the human <em>Dnmt3b</em> gene were found to be the cause of ICF syndrome (immunodeficiency, centromeric instability, and facial anomalies syndrome) (<a href="#B32" rid="B32" class=" bibr popnode">32</a>&#x02013;<a href="#B34" rid="B34" class=" bibr popnode">34</a>). ICF patients show decreased CpG methylation within satellite repeats of the centromeres of chromosomes 1, 9, and 16, coupled with centromeric instability. This suggests that Dnmt3b may be critical for methylation of some types of repetitive DNA.</p><p class="p">In this report, we describe genes in maize and <em>Arabidopsis</em> that display a high degree of sequence similarity to <em>Dnmt3</em>. They display a distinct arrangement of the DNA methyltransferase catalytic motifs, and they contain a series of ubiquitin-associated (UBA) domains in their N termini.</p></div><div id="sec-a.h.j" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.h.jtitle" style="text-transform: uppercase;">Materials and Methods</h2><div id="sec-a.h.j.b" class="sec sec-first"><p></p><h3 id="sec-a.h.j.btitle" class="inline">cDNA Cloning and Rapid Amplification of cDNA Ends (RACE) Analysis. </h3><p class="p p-first-last"><em>Arabidopsis</em> genomic DNA was PCR amplified to generate a hybridization probe corresponding to the TAMU bacterial artificial chromosome (BAC) survey sequence B62154, which was used to screen an <em>Arabidopsis</em> young seedling cDNA library (kindly provided by the Arabidopsis Biological Resource Center, Columbus, OH). Hybridization procedures were as previously described (<a href="#B35" rid="B35" class=" bibr popnode">35</a>) but the washes were done under moderately stringent conditions; 0.1&#x000d7; SSPE&#x0002f;0.5&#x00025; SDS for 15 min at 55&#x000b0;C (1&#x000d7; SSPE = 0.18 M NaCl&#x0002f;10 mM phosphate, pH 7.4&#x0002f;1 mM EDTA). After the initial identification of the maize <em>Dnmt3</em>-like expressed sequence tag (EST) sequence, RACE PCR was performed on Marathon cDNA (CLONTECH), using Advantage2 DNA polymerase (CLONTECH) to clone the full-length <em>ZMET3</em> cDNA sequence. One-week-old Mo17 seedling RNA was used to construct the cDNA. The primers used for RACE were Dmt3F1 (5&#x02032;-ATCCGTATGCCAAGCCTGTGGAGAGC-3&#x02032;), Dmt3F2 (GATGGACTTGACGGCGTGTAAGATCC-3&#x02032;), Zmet3RACE1 (5&#x02032;-GGAGGAAGTGGCAGAGGAGGAGG-3&#x02032;), and Zmet3RACE2 (5&#x02032;-GGAGGCACTGGACGGCGTGG-3&#x02032;).</p></div><div id="sec-a.h.j.c" class="sec"><p></p><h3 id="sec-a.h.j.ctitle" class="inline">Phylogenetic Analysis. </h3><p class="p p-first-last">Alignments were performed with a region of each methyltransferase starting at the conserved catalytic motif I and ending at motif IV (brackets in Fig. <a href="/pmc/articles/PMC18343/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2em;">Fig.1</span></span><span>1</span></a><em>B</em>) by using <span class="small-caps">clustal</span>X 1.8 and default parameters. Since the chromodomain present in the CMT proteins represents a large insertion relative to all of the other methyltransferases, we introduced a deletion in this region so that all of the sequences aligned unambiguously (total length of alignment, 81 amino acids). Fig. <a href="/pmc/articles/PMC18343/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2em;">Fig.3</span></span><span>3</span></a> shows a bootstrap tree based on amino acids (unordered character states). Bootstrapping was done by using heuristic searches in <span class="small-caps">paup</span>* (4.0b2 written by David L. Swofford, Smithsonian Institution, Washington, DC) with 1000 replicates and 10 random additions per replicate. Similar trees were also obtained with alignments in which the chromodomain sequences were not deleted (S.E.J., unpublished results).  </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC18343/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140626408488960"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=18343_pq0800817001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pq0800817001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC18343/bin/pq0800817001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140626408488960"><a target="object" rel="noopener" href="/pmc/articles/PMC18343/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC18343/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p>(<em>A</em>) Schematic diagram of the domain structures of Dnmt3b, DRM2, and Zmet3. Figure is drawn to scale. Shaded boxes show the different motifs present in these proteins, including the PWWP and cysteine rich (C-rich) motifs present in Dnmt3b and the UBA domains present in DRM2 and Zmet3. Roman numerals denote the motifs of the methyltransferase catalytic domains. (<em>B</em>) Alignment of DRM2 of <em>Arabidopsis thaliana</em> with Zmet3 of <em>Zea mays</em> and the methyltransferase catalytic domains of mouse Dnmt3b (GenBank accession no. <span>AF068628</span>) and <em>Danio rerio</em> Zmt3 (Danmt3; accession no. <span>AF135438</span>). Alignments were done in <span class="small-caps">clustal</span>X 1.8 using default parameters and shaded in M<span class="small-caps">ac</span>B<span class="small-caps">ox</span>S<span class="small-caps">hade</span> 1.0.8 (Michael D. Baron, Institute for Animal Health, Surrey, U.K.). Identical residues are shown with a black background and similar residues with a gray background. Dashes show putative nuclear localization sequences that are conserved in the plant proteins. Pound symbols (&#x00023;) show the point of rearrangement of the plant proteins relative to the animal proteins. Here the numbering of the animal methyltransferases begins at amino acid 581 for Dnmt3b and 558 for Danmt3. Conserved catalytic motifs I&#x02013;VI and IX&#x02013;X are marked. Asterisks denote conserved amino acids present in each motif (<a href="#B11" rid="B11" class=" bibr popnode">11</a>). Brackets show the exact region used in the alignments that were used to produce the tree shown in Fig. <a href="/pmc/articles/PMC18343/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2em;">Fig.3</span></span><span>3</span></a>.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC18343/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140626389552240"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=18343_pq0800817003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pq0800817003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC18343/bin/pq0800817003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140626389552240"><a target="object" rel="noopener" href="/pmc/articles/PMC18343/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC18343/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p>Phylogenetic relationships of DNA methyltransferases. Tree shows nodes with &#x0003e;50&#x00025; bootstrap support. Abbreviations and accession numbers, in parentheses, are as follows: MET1, <em>Arabidopsis thaliana</em> maintenance methyltransferase MET1 (<span>P34881</span>); Zmet1, <em>Zea mays</em> MET1-like protein (AF0063403); MusDnmt1, mouse maintenance methyltransferase Dnmt1 (<span>P13864</span>); DanioDnmt1, zebrafish Dnmt-like protein (<span>AF097875</span>); AscMasc2, <em>Ascobolus immersus</em> Masc2 (<span>AF030976</span>); CMT1, <em>Arabidopsis thaliana</em> chromomethylase 1 (<span>AF039372</span>); Zmet2, <em>Zea mays</em> CMT-like protein (<span>AF243043</span>); CMT2, <em>Arabidopsis thaliana</em> chromomethylase 2 (<span>AL021711</span>); AscMasc1, <em>Ascobolus immersus</em> Masc1 (<span>AF025475</span>); BssHII, <em>Bacillus stearothermophilus</em> <em>Bss</em>HII methylase (<span>AF020002</span>); MusDNMT2, mouse DNMT2 (<span>AF045889</span>); PMT, <em>Schizosaccharomyces pombe</em> PMTp1 (<span>P40999</span>); DRM1, <em>Arabidopsis thaliana</em> DRM1 (<span>B62154</span>); DRM2, <em>Arabidopsis thaliana</em> DRM2; Zmet3, <em>Zea mays</em> Zmet3; SoyDRM, soybean EST clone with similarity to DRM and Zmet3 (A1736568); MusDnmt3A, mouse <em>de novo</em> methyltransferase Dnmt3a (<span>AF068625</span>); MusDnmt3B, mouse <em>de novo</em> methyltransferase Dnmt3b (<span>AF068628</span>); DanioDnmt3, zebra fish Dnmt3-like protein (<span>AF135438</span>); MSPR, <em>Bacillus subtilis</em> bacteriophage SPR methyltransferase (<span>P00476</span>); and <em>Hha</em>I, bacterial (<em>Haemophilus haemolyticus</em>) <em>Hha</em>I methyltransferase used as an outgroup (<span>P05102</span>).</p></div></div></div></div><div id="sec-a.h.j.d" class="sec"><p></p><h3 id="sec-a.h.j.dtitle" class="inline">Mapping. </h3><p class="p p-first-last">To map <em>DRM2,</em> the 5&#x02032; 970 base pairs of the <em>DRM2</em> cDNA was PCR amplified, labeled with <sup>32</sup>P, and used to probe an ordered Institut f&#x000fc;r Genbiologische Forschung Berlin BAC filter (kindly provided by the Arabidopsis Biological Resource Center). <em>DRM1</em> was mapped in a similar fashion by using a probe corresponding to the genomic sequence contained in GenBank accession no. <span>B62154</span>.</p></div><div id="sec-a.h.j.e" class="sec sec-last"><p></p><h3 id="sec-a.h.j.etitle" class="inline">RNA Blot Analysis. </h3><p class="p p-first-last">Total RNA from <em>Arabidopsis</em> leaves or inflorescences from 4-week-old plants or whole roots grown in culture was isolated with Tri Reagent (Molecular Research Center, Cincinnati). Thirteen micrograms of total RNA was subjected to electrophoresis on a formaldehyde-containing 0.8&#x00025; agarose gel, blotted onto a Hybond-N<sup>&#x0002b;</sup> (Amersham) membrane, and probed with a full-length <em>DRM2</em> cDNA. Hybridization procedures were as previously described (<a href="#B35" rid="B35" class=" bibr popnode">35</a>) and washes were at high stringency: 0.1&#x000d7; SSPE&#x0002f;0.5&#x00025; SDS for 15 min at 65&#x000b0;C.</p></div></div><div id="sec-a.h.k" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.h.ktitle" style="text-transform: uppercase;">Results</h2><p class="p p-first">We searched <em>Arabidopsis</em> and maize databases for genes similar to <em>Dnmt3. </em>In <em>Arabidopsis</em>, a BAC end sequence (GenBank accession no. <span>B62154</span>) showing similarity to the catalytic domain was found. A probe derived from this BAC end sequence was used to screen a cDNA library at moderate stringency. No cDNA clones corresponding to this sequence were found. However, several clones corresponding to a closely related sequence were isolated. Three of the cDNA clones appeared to be full length, as they contained an in-frame stop codon in the 5&#x02032; untranslated region (GenBank accession no. <span>AF240695</span>). In maize, mouse <em>Dnmt3</em> was used as a query against an EST database provided by Pioneer Hi-Bred International. An EST contig containing the 3&#x02032; 612 base pairs of a maize cDNA was identified. RACE-PCR was used to clone and identify the 5&#x02032; sequence of the mRNA (GenBank accession no. <span>AF242320</span>).</p><p>The predicted proteins from the <em>Arabidopsis</em> and maize cDNA sequences are similar to each other along their entire length (Fig. <a href="/pmc/articles/PMC18343/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.1).</span></span><span>1</span></a>). They exhibit 28&#x00025; amino acid identity in the N-terminal domains and 66&#x00025; identity in the C-terminal catalytic domains. The catalytic domains of these proteins are most similar (an average of 28&#x00025; amino acid identity) to the catalytic domains of the animal Dnmt3 proteins (Fig. <a href="/pmc/articles/PMC18343/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.1</span></span><span>1</span></a><em>B</em>). <span class="small-caps">tblastn</span> searches of the <em>Arabidopsis</em> and maize sequences against the National Center for Biotechnology Information EST database showed that the most closely related mammalian EST sequences are those corresponding to the <em>Dnmt3</em> genes of mouse and human. We did not detect significant similarity between the N-terminal domains of the plant and animal proteins by using <span class="small-caps">blastp</span> or <span class="small-caps">clustal</span>X, and both DRM2 and Zmet3 lack the PWWP and cysteine-rich motifs present in the Dnmt3 methyltransferases (<a href="#B30" rid="B30" class=" bibr popnode">30</a>, <a href="#B33" rid="B33" class=" bibr popnode">33</a>).</p><p>Relative to known eukaryotic methyltransferases, the plant proteins show a novel arrangement of the conserved catalytic motifs. Most methyltransferases, including Dnmt3, contain motifs I, II, III, IV, V, VI, IX, and X from the N terminus to the C terminus of the protein (motifs VII and VIII are not highly conserved and are difficult to distinguish in many methyltransferases). However, both the <em>Arabidopsis</em> and maize sequences display an altered arrangement of these motifs, VI, IX, X, I, II, III, IV, V (Fig. <a href="/pmc/articles/PMC18343/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.1</span></span><span>1</span></a><em>A</em>). The location of the rearrangement can be pinpointed to a region of several amino acids between motifs X and I (Fig. <a href="/pmc/articles/PMC18343/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.1</span></span><span>1</span></a><em>B</em>). Because of this rearrangement, the <em>Arabidopsis</em> proteins have been named the domains rearranged methyltransferases (DRMs). The first observed homolog from BAC sequence <span>B62154</span> is named DRM1 and the sequence reported in Fig. <a href="/pmc/articles/PMC18343/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2em;">Fig.1</span></span><span>1</span></a> is named DRM2. The maize sequence has been named <em>ZMET3</em>, as this represents the third class of methyltransferase to have been isolated from maize.</p><p>A search of other plant ESTs by using <em>DRM</em> and <em>ZMET3</em> as queries revealed the presence of a soybean 3&#x02032; cDNA sequence (accession no. A1736568) which displays a high level of identity to both the <em>Arabidopsis</em> and maize sequences. This partial sequence predicts a polypeptide that encodes the methyltransferase catalytic motifs IX, X, I, II, III, IV, and V, which is the same order seen in both <em>Arabidopsis</em> and maize.</p><p>We sought to determine the possible effect of the motif rearrangement seen in DRM2&#x0002f;Zmet3 on the protein structure, relative to known structures for DNA methyltransferases. Specifically, we looked at the possible implications of the juxtaposition of motifs X and I in the primary sequence. The solved structure of a prokaryotic <em>Hha</em>I methyltransferase (<a href="#B36" rid="B36" class=" bibr popnode">36</a>, <a href="#B37" rid="B37" class=" bibr popnode">37</a>) is shown in Fig. <a href="/pmc/articles/PMC18343/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2em;">Fig.2</span></span><span>2</span></a> with motifs I and X highlighted. These motifs lie parallel to one another in the tertiary structure and are physically associated. Furthermore, the C terminus of <em>Hha</em>I motif X (residue 322, marked with arrow in Fig. <a href="/pmc/articles/PMC18343/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2em;">Fig.2</span></span><span>2</span></a><em>B</em>) and the N terminus of <em>Hha</em>I motif I (residue 9, marked with arrow in Fig. <a href="/pmc/articles/PMC18343/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2em;">Fig.2</span></span><span>2</span></a><em>B</em>) are very close together in the three-dimensional structure. Because these amino acids are directly adjacent to one another in the primary sequence of DRM2 and Zmet3, it is conceivable that, despite the motif rearrangement, the overall fold of the plant proteins is similar to that of <em>Hha</em>I.  </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC18343/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140626409341312"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=18343_pq0800817002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pq0800817002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC18343/bin/pq0800817002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140626409341312"><a target="object" rel="noopener" href="/pmc/articles/PMC18343/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC18343/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p>(<em>A</em>) An alignment of motifs X and I of <em>Hha</em>I with the sequence of DRM2 spanning the point of motif rearrangement. <em>Hha</em>I residues underlined are the last residue of motif X and the first residue of motif I and correspond the positions in <em>B</em> marked by arrows. Alignments were done in <span class="small-caps">clustal</span>X and shaded in <span class="small-caps">boxshade</span>. (<em>B</em>) <span class="small-caps">rasmol</span>-generated image (Roger Sayle, Glaxo Research and Development, Greenford, Middlesex, U.K.) of Protein Database ID 5MHT (ternary structure of <em>Hha</em>I methyltransferase with hemimethylated DNA and <em>S</em>-adenosylhomocysteine) (<a href="#B36" rid="B36" class=" bibr popnode">36</a>, <a href="#B37" rid="B37" class=" bibr popnode">37</a>). DNA helix is shown in black, the majority of <em>Hha</em>I is shaded gray, motif I is red, and motif X is blue. The colored regions correspond to those shown in the alignment in <em>A</em>, which are residues 9&#x02013;32 for motif I and residues 299&#x02013;322 for motif X. Arrows shows the last residue of motif X and first residue of motif I.</p></div></div></div><p>To examine the relationships between the plant methyltransferases described here and other known methyltransferases, we performed alignments with the conserved catalytic motifs I&#x02013;IV (Fig. <a href="/pmc/articles/PMC18343/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.1</span></span><span>1</span></a><em>B</em>), which were then used to generate a phylogenetic tree (Fig. <a href="/pmc/articles/PMC18343/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.3).</span></span><span>3</span></a>). Representatives of four classes of animal and plant DNA methyltransferases were used in the alignments, including enzymes of the Dnmt1&#x0002f;MET1 maintenance methyltransferase class, as well as the Dnmt2, CMT, and Dnmt3 classes. DRM1, DRM2, Zmet3, and the related soybean EST sequence group with a 94&#x00025; bootstrap value to the clade containing the <em>de novo</em> methyltransferase proteins Dnmt3a and Dnmt3b from mammals and zebrafish (<em>Danio rerio</em>).  </p><p>Consistent with their putative function as DNA methyltransferases, both the DRM2 and Zmet3 proteins are predicted by <span class="small-caps">psort</span> (<a href="#B38" rid="B38" class=" bibr popnode">38</a>) to reside in the nucleus and contain conserved nuclear targeting sequences of the simian virus 40 large T antigen type. These lie in the N terminus of the protein (underlined in Fig. <a href="/pmc/articles/PMC18343/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2em;">Fig.1</span></span><span>1</span></a><em>B</em>).</p><p>To determine whether DRM2 or Zmet3 contains any recognizable domains in their N termini, we tested the protein sequences on both the PFAM and SMART (<a href="#B39" rid="B39" class=" bibr popnode">39</a>) protein prediction web servers. Both programs predicted a series of UBA domains in DRM2 (three separate domains) and Zmet3 (two domains) (Fig. <a href="/pmc/articles/PMC18343/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.1</span></span><span>1</span></a><em>A</em>). UBA domains are found in several ubiquitination pathway enzymes, in proteins involved in nucleotide excision repair (such as Rad23), and in some protein kinases (<a href="#B40" rid="B40" class=" bibr popnode">40</a>). The NMR structure of a UBA domain from the human homolog of Rad23 (HHR23A) shows that it folds into a compact three-helix bundle (<a href="#B41" rid="B41" class=" bibr popnode">41</a>). Fig. <a href="/pmc/articles/PMC18343/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2em;">Fig.4</span></span><span>4</span></a><em>A</em> shows an alignment of the DRM2 and Zmet3 UBA domains with those of several other proteins. They contain several conserved residues that are thought to participate in the formation of the hydrophobic core as well as the sharp turn in the loop between the first and second alpha helices. To determine whether the DRM2 and Zmet3 UBA domains are likely to have a structure similar to HHR23A, we tested the sequences in a nearest-neighbor secondary structure prediction program, NNSSP (Fig. <a href="/pmc/articles/PMC18343/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.4</span></span><span>4</span></a><em>B</em>). This algorithm predicted a secondary structure for HHR23A that is largely similar to the known NMR structure, and it predicted that two of the DRM2 UBA domains and one of the Zmet3 UBA domains are likely to have a structure similar to HHR23A (Fig. <a href="/pmc/articles/PMC18343/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.4</span></span><span>4</span></a><em>B</em>). The remaining two UBA domains, DRM2 amino acids 60&#x02013;97 and Zmet3 amino acids 165&#x02013;202, were predicted to contain a &#x003b2;-sheet in the place of the third &#x003b1;-helix predicted in HHR23A.  </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC18343/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140626389352144"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=18343_pq0800817004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pq0800817004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC18343/bin/pq0800817004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140626389352144"><a target="object" rel="noopener" href="/pmc/articles/PMC18343/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC18343/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p>(<em>A</em>) <span class="small-caps">clustal</span>X alignment of the UBA domains predicted by PFAM (<a href="http://pfam.wustl.edu/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http:&#x0002f;&#x0002f;pfam.wustl.edu&#x0002f;</a>) of DRM2 and Zmet3 with those of human p62 (accession no. <span>U46751</span>), yeast Rad23 (accession no. <span>S66117</span>), human Cbl (accession no. <span>A43817</span>), human ubiquitin C-terminal hydrolase 5 (accession no. <span>P45974</span>), <em>Drosophila melanogaster</em> ubiquitin-conjugating enzyme E2 (accession no. <span>P52486</span>), and human HHR23A (accession no. <span>S44443</span>). Below the alignment is the known structure of the second UBA domain of HHR23A, showing the three &#x003b1;-helices (-&#x02009;-&#x02009;-) (<a href="#B41" rid="B41" class=" bibr popnode">41</a>) (Thomas Mueller and Juli Feigon, personal communication). *, Residues making important contributions to the hydrophobic core. &#x00023;, Phe residue connecting the loop between helices &#x003b1;1 and &#x003b1;2 to the C terminus of helix &#x003b1;3. (<em>B</em>) Results of nearest-neighbor secondary structure prediction (NNSSP) on the Baylor College of Medicine Protein Web server (<a href="http://dot.imgen.bcm.tmc.edu:9331/pssprediction/pssp.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http:&#x0002f;&#x0002f;dot.imgen.bcm.tmc.edu:9331&#x0002f;pssprediction&#x0002f;pssp.html</a>) (<a href="#B54" rid="B54" class=" bibr popnode">54</a>), which predicts the presence of &#x003b1;-helices (a) or &#x003b2;-sheets (b). Gaps were introduced in the sequences as in the alignments in <em>A</em>.</p></div></div></div><p>We assayed the complexity of the gene families encoding DRM and Zmet3 type proteins by using Southern blot analysis and <span class="small-caps">blast</span> searching. In both <em>Arabidopsis</em> and maize, Southern blot analysis of genomic DNA detected several hybridizing bands suggesting the presence of small gene families (data not shown). <span class="small-caps">tblastn</span> searches with the full-length DRM2 protein identified three additional sequences with significant similarity within the available <em>Arabidopsis</em> genome sequences (85&#x00025; of the genome was sequenced at the time of this writing). These are accession nos. <span>AB022216</span>, <span>AC012375</span>, and T22J18. All three of these sequences reside in sequenced regions of the genome and appear to encode <em>DRM</em> pseudogenes (S.E.J., unpublished observation). A <span class="small-caps">tblastn</span> search of GenBank using the full-length <em>Zmet3</em> sequence detected a maize EST sequence encoding a related protein (accession no. <span>AI947339</span>). This sequence lacks the highly conserved PC site in motif IV of the catalytic domain, suggesting that it may be a pseudogene.</p><p>The <em>DRM</em> genes were mapped by hybridizing them to a filter containing ordered <em>Arabidopsis</em> BAC clones (<a href="#B42" rid="B42" class=" bibr popnode">42</a>). A <em>DRM2</em> probe hybridized strongly to six clones (F6K20, F17H18, F17N1, F19B7, F27E1, and F19H5), all of which map to an overlapping position on the top of chromosome V between markers mi174 and mi322. <em>DRM1</em> hybridized strongly to a different set of overlapping clones (F17C20, F25G21, F7P21, F1K12, F12A2, F27F12, F14F13, F5A22, F19D5, F9D20, F4K15, F3H2, F1I1, and F8M21) which map to a nearby region of chromosome V. From the physical map the estimated distance between <em>DRM1</em> and <em>DRM2</em> is about 230 kb, suggesting a map distance of approximately 1 centimorgan (details can be found at <a href="http://genome-www3.stanford.edu/cgi-bin/AtDB/Pmap?contig=F24C6-T7-F24L21-Sp6&#x00026;clone=F28E9" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http:&#x0002f;&#x0002f;genome-www3.stanford.edu&#x0002f;cgi-bin&#x0002f;AtDB&#x0002f;Pmap?contig=F24C6-T7-F24L21-Sp6&#x00026;clone=F28E9</a>). These map positions do not correspond to known methylation mutants in <em>Arabidopsis</em>. The <em>DRM</em> probes also detected several weakly hybridizing clones, all of which mapped to two different regions on chromosome I. These groups consist of F6F2, F7M24, F24I22, F24L16, F12N13, F28N2, and F4P4, which map to an overlapping region near the <em>PAI3</em> marker, and F27D22, F11L21, and F11P6, which map to an overlapping region near the <em>PAI1</em> marker. These two regions correspond exactly to the pseudogenes mentioned above, <span>AC012375</span> and T22J18, respectively. Thus, all hybridizing clones can be accounted for by DNA sequences present in the database.</p><p class="p p-last">RNA blot analysis was used to detect expression of <em>DRM2</em> in different tissues. A 2.5-kb transcript was easily detected on blots of total RNA from roots, leaves, or inflorescences (Fig. <a href="/pmc/articles/PMC18343/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.5).</span></span><span>5</span></a>). This finding suggests that <em>DRM2</em> is expressed in most tissues. Probing of the same blot with <em>DRM1</em> did not detect any message, suggesting that <em>DRM1</em> is expressed at a lower level than <em>DRM2</em>.  </p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC18343/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140626391183424"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=18343_pq0800817005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is pq0800817005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC18343/bin/pq0800817005.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140626391183424"><a target="object" rel="noopener" href="/pmc/articles/PMC18343/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC18343/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a7--><div class="caption"><p>RNA blot analysis showing the size and abundance of <em>DRM2</em> in different tissues. RNA from roots, leaves, or inflorescences was blotted and hybridized to a full-length <em>DRM2</em> clone. Ethidium bromide staining of the ribosomal RNA bands is shown below as a loading control.</p></div></div></div></div><div id="sec-a.h.l" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.h.ltitle" style="text-transform: uppercase;">Discussion</h2><div id="sec-a.h.l.b" class="sec sec-first"><p></p><h3 id="sec-a.h.l.btitle" class="inline">Identification of a Distinct Family of DNA Methyltransferases. </h3><p class="p p-first">We describe a distinct type of putative C5 DNA methyltransferase conserved in <em>Arabidopsis</em> and maize. DRM2 and Zmet3 display the highest level of similarity to each other in their C-terminal regions, which contain the DNA methyltransferase catalytic domains. The amino acid identity in these regions is 66&#x00025;, whereas it is only 28&#x00025; in the N-terminal regions. This observation suggests selection for the conservation of methyltransferase function, supporting the hypothesis that Zmet3 and DRM2 are functional DNA methyltransferases. This hypothesis is further supported by the observation that both Zmet3 and DRM2 contain several amino acids thought to be critical for the function of cytosine methyltransferases, including the Phe-Xaa-Gly-Xaa-Gly residues present in motif I involved in <em>S</em>-adenosylmethionine binding, the invariant Pro-Cys dipeptide of the catalytic site at motif IV, and the Glu-Asn-Val residues at motif VI, which interact with the target cytosine (<a href="#B11" rid="B11" class=" bibr popnode">11</a>, <a href="#B36" rid="B36" class=" bibr popnode">36</a>, <a href="#B37" rid="B37" class=" bibr popnode">37</a>, <a href="#B43" rid="B43" class=" bibr popnode">43</a>, <a href="#B44" rid="B44" class=" bibr popnode">44</a>).</p><p>DRM2 and Zmet3 are characterized by a rearrangement of the catalytic methyltransferase motifs. The presence of the same arrangement in <em>Arabidopsis</em>, maize, and soybean indicates that the permutation occurred before the divergence of monocots and dicots. There are at least two processes that could have given rise to structure seen in these genes. The first is a transposition event resulting in a swap between motifs I&#x02013;V and motifs VI&#x02013;X. A second possibility is gene duplication followed by deletions to remove motifs I&#x02013;V of the first gene, the intervening sequence between the two genes, and motifs VI&#x02013;X of the second gene.</p><p>We know of no other examples of eukaryotic genes displaying rearranged DNA methyltransferase motifs. However, there are examples of domain permutations within the bacterial C5 DNA methyltransferases (<a href="#B45" rid="B45" class=" bibr popnode">45</a>). One example is <em>Bss</em>HII in which motifs IX and X precede motif I&#x02013;VIII (<a href="#B46" rid="B46" class=" bibr popnode">46</a>). This example is similar to DRM2 and Zmet3 in the sense that motifs X and I are juxtaposed. In a second case, <em>Aqu</em>I, motifs IX and X are located in a separately encoded subunit (<a href="#B47" rid="B47" class=" bibr popnode">47</a>). The fact that these prokaryotic enzymes maintain DNA methyltransferase activity shows that the usual arrangement of DNA methyltransferase motifs is not necessary for function. Furthermore, the specific permutation present in DRM2 and Zmet3, juxtaposition of motifs I and X in the primary sequence, may have little overall effect on the folding or function of the methyltransferase catalytic domains, because these motifs are found adjacent to each other in the tertiary structures of <em>Hha</em>I (<a href="#B36" rid="B36" class=" bibr popnode">36</a>, <a href="#B37" rid="B37" class=" bibr popnode">37</a>) (Fig. <a href="/pmc/articles/PMC18343/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.2)</span></span><span>2</span></a>) and <em>Hae</em>III (<a href="#B44" rid="B44" class=" bibr popnode">44</a>).</p><p>DRM2 and Zmet3 contain a series of UBA domains in their N termini. The function of the UBA domain is presently unclear. UBA domain-containing proteins show a variety of associations with the ubiquitin pathway. Some are components of the ubiquitination machinery such as ubiquitin C-terminal hydrolases, ubiquitin-conjugating enzymes (E2), and ubiquitin protein ligases (E3) (<a href="#B40" rid="B40" class=" bibr popnode">40</a>). Others are involved in DNA repair, such as Rad23, which is itself ubiquitinated and degraded during the transition between the G<sub>1</sub> and S phases of the cell cycle (<a href="#B48" rid="B48" class=" bibr popnode">48</a>). The UBA-containing protooncoprotein Cbl shows reversible ubiquitination that is associated with its cycling between the plasma membrane and cytosolic fractions of the cell (<a href="#B49" rid="B49" class=" bibr popnode">49</a>, <a href="#B50" rid="B50" class=" bibr popnode">50</a>). Cbl also acts as ubiquitin ligase for receptor protein-tyrosine kinases (<a href="#B50" rid="B50" class=" bibr popnode">50</a>, <a href="#B51" rid="B51" class=" bibr popnode">51</a>).</p><p>The C-terminal UBA domain present in the human Rad23 homolog HHR23A was shown to bind <em>in vitro</em> to the HIV-1 protein Vpr (<a href="#B52" rid="B52" class=" bibr popnode">52</a>). Furthermore, an 80-amino acid region of the p62 protein, which consists largely of a UBA domain (<a href="#B39" rid="B39" class=" bibr popnode">39</a>), was shown to display noncovalent binding to ubiquitin (<a href="#B53" rid="B53" class=" bibr popnode">53</a>). These results suggest that UBA domains may serve as protein interaction interfaces, and that one specific function of the UBA domain is to bind ubiquitin.</p><p class="p p-last">The presence of UBA domains in DRM2 and Zmet3 suggests a link between DNA methylation and ubiquitin&#x0002f;proteasome pathways. One possibility is that, like Rad23, DRM2&#x0002f;Zmet3 proteins vary throughout the cell cycle through ubiquitin-mediated protein degradation. A second possibility is that ubiquitination alters the cellular localization of the DRM2&#x0002f;Zmet3 proteins in response to external signals, the cell cycle, or transposon or retroviral activity. As UBA domains are not found in other classes of methyltransferases or in the mammalian or fish Dnmt3 proteins (S.E.J., unpublished observation), this possible association with the ubiquitin pathway may be restricted to the Dnmt3-like methyltransferases of plants.</p></div><div id="sec-a.h.l.c" class="sec"><p></p><h3 id="sec-a.h.l.ctitle" class="inline">Origins of Eukaryotic DNA Methyltransferases. </h3><p class="p p-first-last">Our phylogenetic analysis suggests that the Dnmt3&#x0002f;DRM2&#x0002f;Zmet3 enzymes form a distinct class of proteins that are closer to each other than they are to other types of methyltransferases, including the Dnmt1&#x0002f;MET1 class, the CMT class, and the Dnmt2 class. This observation suggests that these different types of methyltransferases formed early in eukaryotic evolution, before the divergence of plants and animals, and may therefore share common functions. One possible scenario is that the different classes of methyltransferases could have originated from separate prokaryotic lineages. In particular, Xie <em>et al.</em> (<a href="#B30" rid="B30" class=" bibr popnode">30</a>) have suggested that the <em>Dnmt3</em> class of genes may have evolved from an ancestor related to the <em>Bacillus subtilis</em> bacteriophage SPR methyltransferase, whereas the Dnmt1 class of DNA methyltransferases may have evolved from a separate prokaryotic ancestor. This conclusion is supported by the observation that SPR methyltransferase groups more closely with the Dnmt3&#x0002f;DRM2&#x0002f;Zmet3 sequences than with other eukaryotic methyltransferases (Fig. <a href="/pmc/articles/PMC18343/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">(Fig.3</span></span><span>3</span></a>).</p></div><div id="sec-a.h.l.d" class="sec sec-last"><p></p><h3 id="sec-a.h.l.dtitle" class="inline"><em>De Novo</em> Methylation in Plants. </h3><p class="p p-first-last">Given the relationship of the plant genes to <em>Dnmt3</em>, we propose that DRM2 and Zmet3 act as plant <em>de novo</em> methyltransferases. Several well-characterized examples of <em>de novo</em> methylation occur in plants. One case is the extensive methylation at the <em>SUPERMAN</em> locus in the <em>Arabidopsis clark kent</em> mutants and in plants containing antisense-<em>MET1</em> constructs (<a href="#B3" rid="B3" class=" bibr popnode">3</a>). <em>SUPERMAN</em> remains methylated and silenced in most tissues of the plant, indicating a requirement for <em>de novo</em> methyltransferase activity in somatic cells of the shoot. The expression profile of <em>DRM2</em> fits this requirement, since <em>DRM2</em> RNA is found in all of the major tissues of the plant. The methylation at <em>SUPERMAN</em> is very dense (over 50&#x00025; of the cytosines are methylated), and at mostly asymmetric sites. However, different hypermethylated <em>superman</em> alleles show largely similar patterns of methylation (<a href="#B21" rid="B21" class=" bibr popnode">21</a>). It is difficult to envision a mechanism by which this strange pattern of methylation is inherited. One hypothesis is that <em>SUPERMAN</em> is continually methylated by a <em>de novo</em> methyltransferase system in response to a &#x0201c;seed&#x0201d; of preexisting methylation. In this way, the <em>DRM</em> genes might serve as maintenance methyltransferases for genes containing asymmetric methylation. Other examples of <em>de novo</em> methylation in plants include paramutation at <em>R</em>, transposable element inactivation, and transgene silencing. It seems possible that Zmet3&#x0002f;DRM2-type enzymes could play a major role in these phenomena as well. Biochemical and genetic studies should allow determination of the function of these plant DNA methyltransferases.</p></div></div><div id="ack-a.i.b" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.i.btitle" style="text-transform: uppercase;">Acknowledgments</h2><div class="sec"><p>We thank the Arabidopsis Biological Resource Center for materials, and Mike Frohlich, Thomas Mueller, and Juli Feigon for helpful discussions. This work was supported by National Institutes of Health Grant GM60398 and Jonsson Cancer Center Foundation&#x0002f;UCLA Seed Grant ACS &#x00023;IRG 78-001-21 to S.E.J. Maize research was supported by Pioneer Hi-Bred International, Inc., and the University of Wisconsin Graduate School, and this was carried out jointly in the laboratories of S.K. and R.L.P. Questions and material requests for maize work should be directed to S.K. N.S. was supported by a U.S. Department of Agriculture National Needs Fellowship (98-38420-5832).</p></div></div><div id="glossary-a.i.c" class="tsec sec"><h2 class="head no_bottom_margin" id="glossary-a.i.ctitle" style="text-transform: uppercase;">Abbreviations</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td id="G1">CMT</td><td>chromomethylase</td></tr><tr><td id="G2">ES cells</td><td>embryonic stem cells</td></tr><tr><td id="G3">UBA</td><td>ubiquitin-associated</td></tr><tr><td id="G4">RACE</td><td>rapid amplification of cDNA ends</td></tr><tr><td id="G5">BAC</td><td>bacterial artificial chromosome</td></tr><tr><td id="G6">EST</td><td>expressed sequence tag</td></tr></tbody></table></div></div></div><div id="fn-group-a.i.e" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.i.etitle" style="text-transform: uppercase;">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn-a.i.e.a"><p class="p p-first-last">Data deposition: The sequences reported in this paper have been deposited in the GenBank database (accession nos. <span>AF240695</span> and <span>AF242320</span>).</p></p></div></div><div id="ref-list-a.i.d" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.i.dtitle" style="text-transform: uppercase;">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="citation">Martienssen R A, Richards E J. <span><span class="ref-journal">Curr Opin Genet Dev. </span>1995;<span class="ref-vol">5</span>:234–242.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7613094" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Genet+Dev&amp;author=R+A+Martienssen&amp;author=E+J+Richards&amp;volume=5&amp;publication_year=1995&amp;pages=234-242&amp;pmid=7613094&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="citation">Bender J, Fink G R. <span><span class="ref-journal">Cell. </span>1995;<span class="ref-vol">83</span>:725–734.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8521489" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;author=J+Bender&amp;author=G+R+Fink&amp;volume=83&amp;publication_year=1995&amp;pages=725-734&amp;pmid=8521489&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="citation">Jacobsen S E, Meyerowitz E M. <span><span class="ref-journal">Science. </span>1997;<span class="ref-vol">277</span>:1100–1103.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9262479" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;author=S+E+Jacobsen&amp;author=E+M+Meyerowitz&amp;volume=277&amp;publication_year=1997&amp;pages=1100-1103&amp;pmid=9262479&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="citation">Matzke M A, Matzke A J. <span><span class="ref-journal">Cell Mol Life Sci. </span>1998;<span class="ref-vol">54</span>:94–103.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC11147416/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9487390" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Mol+Life+Sci&amp;author=M+A+Matzke&amp;author=A+J+Matzke&amp;volume=54&amp;publication_year=1998&amp;pages=94-103&amp;pmid=9487390&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="citation">Finnegan E J, Peacock W J, Dennis E S. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1996;<span class="ref-vol">93</span>:8449–8454.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC38691/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8710891" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;author=E+J+Finnegan&amp;author=W+J+Peacock&amp;author=E+S+Dennis&amp;volume=93&amp;publication_year=1996&amp;pages=8449-8454&amp;pmid=8710891&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="citation">Richards E J. <span><span class="ref-journal">Trends Genet. </span>1997;<span class="ref-vol">13</span>:319–323.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9260518" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet&amp;author=E+J+Richards&amp;volume=13&amp;publication_year=1997&amp;pages=319-323&amp;pmid=9260518&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="citation">Li E, Beard C, Jaenisch R. <span><span class="ref-journal">Nature (London) </span>1993;<span class="ref-vol">366</span>:362–365.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8247133" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+(London)&amp;author=E+Li&amp;author=C+Beard&amp;author=R+Jaenisch&amp;volume=366&amp;publication_year=1993&amp;pages=362-365&amp;pmid=8247133&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="citation">Beard C, Li E, Jaenisch R. <span><span class="ref-journal">Genes Dev. </span>1995;<span class="ref-vol">9</span>:2325–2334.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7557385" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;author=C+Beard&amp;author=E+Li&amp;author=R+Jaenisch&amp;volume=9&amp;publication_year=1995&amp;pages=2325-2334&amp;pmid=7557385&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="citation">Walsh C P, Bestor T H. <span><span class="ref-journal">Genes Dev. </span>1999;<span class="ref-vol">13</span>:26–34.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC316374/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9887097" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;author=C+P+Walsh&amp;author=T+H+Bestor&amp;volume=13&amp;publication_year=1999&amp;pages=26-34&amp;pmid=9887097&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="citation">Yoder J A, Walsh C P, Bestor T H. <span><span class="ref-journal">Trends Genet. </span>1997;<span class="ref-vol">13</span>:335–340.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9260521" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet&amp;author=J+A+Yoder&amp;author=C+P+Walsh&amp;author=T+H+Bestor&amp;volume=13&amp;publication_year=1997&amp;pages=335-340&amp;pmid=9260521&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="citation">Cheng X. <span><span class="ref-journal">Annu Rev Biophys Biomol Struct. </span>1995;<span class="ref-vol">24</span>:293–318.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7663118" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Biophys+Biomol+Struct&amp;author=X+Cheng&amp;volume=24&amp;publication_year=1995&amp;pages=293-318&amp;pmid=7663118&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="citation">Bestor T, Laudano A, Mattaliano R, Ingram V. <span><span class="ref-journal">J Mol Biol. </span>1988;<span class="ref-vol">203</span>:971–983.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/3210246" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;author=T+Bestor&amp;author=A+Laudano&amp;author=R+Mattaliano&amp;author=V+Ingram&amp;volume=203&amp;publication_year=1988&amp;pages=971-983&amp;pmid=3210246&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="citation">Finnegan E J, Dennis E S. <span><span class="ref-journal">Nucleic Acids Res. </span>1993;<span class="ref-vol">21</span>:2383–2388.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC309536/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8389441" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;author=E+J+Finnegan&amp;author=E+S+Dennis&amp;volume=21&amp;publication_year=1993&amp;pages=2383-2388&amp;pmid=8389441&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="citation">Pradhan S, Cummings M, Roberts R J, Adams R L. <span><span class="ref-journal">Nucleic Acids Res. </span>1998;<span class="ref-vol">26</span>:1214–1222.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC147397/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9469828" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;author=S+Pradhan&amp;author=M+Cummings&amp;author=R+J+Roberts&amp;author=R+L+Adams&amp;volume=26&amp;publication_year=1998&amp;pages=1214-1222&amp;pmid=9469828&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="citation">Bestor T H. <span><span class="ref-journal">EMBO J. </span>1992;<span class="ref-vol">11</span>:2611–2617.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC556736/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1628623" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J&amp;author=T+H+Bestor&amp;volume=11&amp;publication_year=1992&amp;pages=2611-2617&amp;pmid=1628623&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="citation">Okano M, Xie S, Li E. <span><span class="ref-journal">Nat Genet. </span>1998;<span class="ref-vol">19</span>:219–220.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9662389" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;author=M+Okano&amp;author=S+Xie&amp;author=E+Li&amp;volume=19&amp;publication_year=1998&amp;pages=219-220&amp;pmid=9662389&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="citation">Li E, Bestor T H, Jaenisch R. <span><span class="ref-journal">Cell. </span>1992;<span class="ref-vol">69</span>:915–926.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1606615" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;author=E+Li&amp;author=T+H+Bestor&amp;author=R+Jaenisch&amp;volume=69&amp;publication_year=1992&amp;pages=915-926&amp;pmid=1606615&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="citation">Lei H, Oh S P, Okano M, Juttermann R, Goss K A, Jaenisch R, Li E. <span><span class="ref-journal">Development (Cambridge, UK) </span>1996;<span class="ref-vol">122</span>:3195–3205.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8898232" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Development+(Cambridge,+UK)&amp;author=H+Lei&amp;author=S+P+Oh&amp;author=M+Okano&amp;author=R+Juttermann&amp;author=K+A+Goss&amp;volume=122&amp;publication_year=1996&amp;pages=3195-3205&amp;pmid=8898232&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="citation">Ronemus M J, Galbiati M, Ticknor C, Chen J, Dellaporta S L. <span><span class="ref-journal">Science. </span>1996;<span class="ref-vol">273</span>:654–657.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8662558" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;author=M+J+Ronemus&amp;author=M+Galbiati&amp;author=C+Ticknor&amp;author=J+Chen&amp;author=S+L+Dellaporta&amp;volume=273&amp;publication_year=1996&amp;pages=654-657&amp;pmid=8662558&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="citation">Jacobsen S E. <span><span class="ref-journal">Curr Biol. </span>1999;<span class="ref-vol">9</span>:R617–R619.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10469582" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Biol&amp;author=S+E+Jacobsen&amp;volume=9&amp;publication_year=1999&amp;pages=R617-R619&amp;pmid=10469582&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="citation">Jacobsen S E, Sakai H, Finnegan E J, Cao X, Meyerowitz E M. <span><span class="ref-journal">Curr Biol. </span>2000;<span class="ref-vol">10</span>:179–186.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10704409" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Biol&amp;author=S+E+Jacobsen&amp;author=H+Sakai&amp;author=E+J+Finnegan&amp;author=X+Cao&amp;author=E+M+Meyerowitz&amp;volume=10&amp;publication_year=2000&amp;pages=179-186&amp;pmid=10704409&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="citation">Pinarbasi E, Elliott J, Hornby D P. <span><span class="ref-journal">J Mol Biol. </span>1996;<span class="ref-vol">257</span>:804–813.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8636983" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;author=E+Pinarbasi&amp;author=J+Elliott&amp;author=D+P+Hornby&amp;volume=257&amp;publication_year=1996&amp;pages=804-813&amp;pmid=8636983&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="citation">Wilkinson C R, Bartlett R, Nurse P, Bird A P. <span><span class="ref-journal">Nucleic Acids Res. </span>1995;<span class="ref-vol">23</span>:203–210.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC306655/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7862522" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;author=C+R+Wilkinson&amp;author=R+Bartlett&amp;author=P+Nurse&amp;author=A+P+Bird&amp;volume=23&amp;publication_year=1995&amp;pages=203-210&amp;pmid=7862522&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="citation">Okano M, Xie S, Li E. <span><span class="ref-journal">Nucleic Acids Res. </span>1998;<span class="ref-vol">26</span>:2536–2540.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC147598/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9592134" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;author=M+Okano&amp;author=S+Xie&amp;author=E+Li&amp;volume=26&amp;publication_year=1998&amp;pages=2536-2540&amp;pmid=9592134&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="citation">Yoder J A, Bestor T H. <span><span class="ref-journal">Hum Mol Genet. </span>1998;<span class="ref-vol">7</span>:279–284.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9425235" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;author=J+A+Yoder&amp;author=T+H+Bestor&amp;volume=7&amp;publication_year=1998&amp;pages=279-284&amp;pmid=9425235&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="citation">Hung M S, Karthikeyan N, Huang B, Koo H C, Kiger J, Shen C J. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1999;<span class="ref-vol">96</span>:11940–11945.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC18391/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10518555" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;author=M+S+Hung&amp;author=N+Karthikeyan&amp;author=B+Huang&amp;author=H+C+Koo&amp;author=J+Kiger&amp;volume=96&amp;publication_year=1999&amp;pages=11940-11945&amp;pmid=10518555&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="citation">Rose T M, Schultz E R, Henikoff J G, Pietrokovski S, McCallum C M, Henikoff S. <span><span class="ref-journal">Nucleic Acids Res. </span>1998;<span class="ref-vol">26</span>:1628–1635.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC147464/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9512532" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;author=T+M+Rose&amp;author=E+R+Schultz&amp;author=J+G+Henikoff&amp;author=S+Pietrokovski&amp;author=C+M+McCallum&amp;volume=26&amp;publication_year=1998&amp;pages=1628-1635&amp;pmid=9512532&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="citation">Henikoff S, Comai L. <span><span class="ref-journal">Genetics. </span>1998;<span class="ref-vol">149</span>:307–318.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1460135/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9584105" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;author=S+Henikoff&amp;author=L+Comai&amp;volume=149&amp;publication_year=1998&amp;pages=307-318&amp;pmid=9584105&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="citation">Gruenbaum Y, Naveh-Many T, Cedar H, Razin A. <span><span class="ref-journal">Nature (London) </span>1981;<span class="ref-vol">292</span>:860–862.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/6267477" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+(London)&amp;author=Y+Gruenbaum&amp;author=T+Naveh-Many&amp;author=H+Cedar&amp;author=A+Razin&amp;volume=292&amp;publication_year=1981&amp;pages=860-862&amp;pmid=6267477&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="citation">Xie S, Wang Z, Okano M, Nogami M, Li Y, He W W, Okumura K, Li E. <span><span class="ref-journal">Gene. </span>1999;<span class="ref-vol">236</span>:87–95.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10433969" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;author=S+Xie&amp;author=Z+Wang&amp;author=M+Okano&amp;author=M+Nogami&amp;author=Y+Li&amp;volume=236&amp;publication_year=1999&amp;pages=87-95&amp;pmid=10433969&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="citation">Lyko F, Ramsahoye B H, Kashevsky H, Tudor M, Mastrangelo M A, Orr-Weaver T L, Jaenisch R. <span><span class="ref-journal">Nat Genet. </span>1999;<span class="ref-vol">23</span>:363–366.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10545955" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;author=F+Lyko&amp;author=B+H+Ramsahoye&amp;author=H+Kashevsky&amp;author=M+Tudor&amp;author=M+A+Mastrangelo&amp;volume=23&amp;publication_year=1999&amp;pages=363-366&amp;pmid=10545955&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="citation">Okano M, Bell D W, Haber D A, Li E. <span><span class="ref-journal">Cell. </span>1999;<span class="ref-vol">99</span>:247–257.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10555141" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;author=M+Okano&amp;author=D+W+Bell&amp;author=D+A+Haber&amp;author=E+Li&amp;volume=99&amp;publication_year=1999&amp;pages=247-257&amp;pmid=10555141&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="citation">Xu G L, Bestor T H, Bourc'his D, Hsieh C L, Tommerup N, Bugge M, Hulten M, Qu X Y, Russo J J, Viegas-Pequignot E. <span><span class="ref-journal">Nature (London) </span>1999;<span class="ref-vol">402</span>:187–191.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10647011" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+(London)&amp;author=G+L+Xu&amp;author=T+H+Bestor&amp;author=D+Bourc'his&amp;author=C+L+Hsieh&amp;author=N+Tommerup&amp;volume=402&amp;publication_year=1999&amp;pages=187-191&amp;pmid=10647011&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="citation">Hansen R S, Wijmenga C, Luo P, Stanek A M, Canfield T K, Weemaes C M, Gartler S M. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1999;<span class="ref-vol">96</span>:14412–11417.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC24450/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10588719" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;author=R+S+Hansen&amp;author=C+Wijmenga&amp;author=P+Luo&amp;author=A+M+Stanek&amp;author=T+K+Canfield&amp;volume=96&amp;publication_year=1999&amp;pages=14412-11417&amp;pmid=10588719&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="citation">Chang C, Bowman J L, DeJohn A W, Lander E S, Meyerowitz E M. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1988;<span class="ref-vol">85</span>:6856–6860.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC282077/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2901107" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;author=C+Chang&amp;author=J+L+Bowman&amp;author=A+W+DeJohn&amp;author=E+S+Lander&amp;author=E+M+Meyerowitz&amp;volume=85&amp;publication_year=1988&amp;pages=6856-6860&amp;pmid=2901107&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="citation">Cheng X, Kumar S, Posfai J, Pflugrath J W, Roberts R J. <span><span class="ref-journal">Cell. </span>1993;<span class="ref-vol">74</span>:299–307.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8343957" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;author=X+Cheng&amp;author=S+Kumar&amp;author=J+Posfai&amp;author=J+W+Pflugrath&amp;author=R+J+Roberts&amp;volume=74&amp;publication_year=1993&amp;pages=299-307&amp;pmid=8343957&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37">37. <span class="citation">Klimasauskas S, Kumar S, Roberts R J, Cheng X. <span><span class="ref-journal">Cell. </span>1994;<span class="ref-vol">76</span>:357–369.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8293469" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;author=S+Klimasauskas&amp;author=S+Kumar&amp;author=R+J+Roberts&amp;author=X+Cheng&amp;volume=76&amp;publication_year=1994&amp;pages=357-369&amp;pmid=8293469&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38">38. <span class="citation">Nakai K, Kanehisa M. <span><span class="ref-journal">Genomics. </span>1992;<span class="ref-vol">14</span>:897–911.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7134799/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1478671" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;author=K+Nakai&amp;author=M+Kanehisa&amp;volume=14&amp;publication_year=1992&amp;pages=897-911&amp;pmid=1478671&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39">39. <span class="citation">Schultz J, Milpetz F, Bork P, Ponting C P. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1998;<span class="ref-vol">95</span>:5857–5864.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC34487/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9600884" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;author=J+Schultz&amp;author=F+Milpetz&amp;author=P+Bork&amp;author=C+P+Ponting&amp;volume=95&amp;publication_year=1998&amp;pages=5857-5864&amp;pmid=9600884&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40">40. <span class="citation">Hofmann K, Bucher P. <span><span class="ref-journal">Trends Biochem Sci. </span>1996;<span class="ref-vol">21</span>:172–173.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8871400" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Biochem+Sci&amp;author=K+Hofmann&amp;author=P+Bucher&amp;volume=21&amp;publication_year=1996&amp;pages=172-173&amp;pmid=8871400&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B41">41. <span class="citation">Dieckmann T, Withers-Ward E S, Jarosinski M A, Liu C F, Chen I S, Feigon J. <span><span class="ref-journal">Nat Struct Biol. </span>1998;<span class="ref-vol">5</span>:1042–1047.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9846873" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Struct+Biol&amp;author=T+Dieckmann&amp;author=E+S+Withers-Ward&amp;author=M+A+Jarosinski&amp;author=C+F+Liu&amp;author=I+S+Chen&amp;volume=5&amp;publication_year=1998&amp;pages=1042-1047&amp;pmid=9846873&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B42">42. <span class="citation">Mozo T, Dewar K, Dunn P, Ecker J R, Fischer S, Kloska S, Lehrach H, Marra M, Martienssen R, Meier-Ewert S, Altmann T. <span><span class="ref-journal">Nat Genet. </span>1999;<span class="ref-vol">22</span>:271–275.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10391215" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;author=T+Mozo&amp;author=K+Dewar&amp;author=P+Dunn&amp;author=J+R+Ecker&amp;author=S+Fischer&amp;volume=22&amp;publication_year=1999&amp;pages=271-275&amp;pmid=10391215&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B43">43. <span class="citation">Posfai J, Bhagwat A S, Posfai G, Roberts R J. <span><span class="ref-journal">Nucleic Acids Res. </span>1989;<span class="ref-vol">17</span>:2421–2435.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC317633/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2717398" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;author=J+Posfai&amp;author=A+S+Bhagwat&amp;author=G+Posfai&amp;author=R+J+Roberts&amp;volume=17&amp;publication_year=1989&amp;pages=2421-2435&amp;pmid=2717398&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B44">44. <span class="citation">Reinisch K M, Chen L, Verdine G L, Lipscomb W N. <span><span class="ref-journal">Cell. </span>1995;<span class="ref-vol">82</span>:143–153.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7606780" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;author=K+M+Reinisch&amp;author=L+Chen&amp;author=G+L+Verdine&amp;author=W+N+Lipscomb&amp;volume=82&amp;publication_year=1995&amp;pages=143-153&amp;pmid=7606780&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B45">45. <span class="citation">Jeltsch A. <span><span class="ref-journal">J Mol Evol. </span>1999;<span class="ref-vol">49</span>:161–164.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10368444" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Evol&amp;author=A+Jeltsch&amp;volume=49&amp;publication_year=1999&amp;pages=161-164&amp;pmid=10368444&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B46">46. <span class="citation">Xu S, Xiao J, Posfai J, Maunus R, Benner J., 2nd <span><span class="ref-journal">Nucleic Acids Res. </span>1997;<span class="ref-vol">25</span>:3991–3994.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC147014/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9321648" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;author=S+Xu&amp;author=J+Xiao&amp;author=J+Posfai&amp;author=R+Maunus&amp;author=J+Benner&amp;volume=25&amp;publication_year=1997&amp;pages=3991-3994&amp;pmid=9321648&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B47">47. <span class="citation">Karreman C, de Waard A. <span><span class="ref-journal">J Bacteriol. </span>1990;<span class="ref-vol">172</span>:266–272.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC208427/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/2104605" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;author=C+Karreman&amp;author=A+de+Waard&amp;volume=172&amp;publication_year=1990&amp;pages=266-272&amp;pmid=2104605&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B48">48. <span class="citation">Schauber C, Chen L, Tongaonkar P, Vega I, Lambertson D, Potts W, Madura K. <span><span class="ref-journal">Nature (London) </span>1998;<span class="ref-vol">391</span>:715–718.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9490418" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+(London)&amp;author=C+Schauber&amp;author=L+Chen&amp;author=P+Tongaonkar&amp;author=I+Vega&amp;author=D+Lambertson&amp;volume=391&amp;publication_year=1998&amp;pages=715-718&amp;pmid=9490418&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B49">49. <span class="citation">Wang Y, Yeung Y G, Langdon W Y, Stanley E R. <span><span class="ref-journal">J Biol Chem. </span>1996;<span class="ref-vol">271</span>:17–20.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8550554" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;author=Y+Wang&amp;author=Y+G+Yeung&amp;author=W+Y+Langdon&amp;author=E+R+Stanley&amp;volume=271&amp;publication_year=1996&amp;pages=17-20&amp;pmid=8550554&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B50">50. <span class="citation">Lee P S, Wang Y, Dominguez M G, Yeung Y G, Murphy M A, Bowtell D D, Stanley E R. <span><span class="ref-journal">EMBO J. </span>1999;<span class="ref-vol">18</span>:3616–3628.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1171440/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10393178" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J&amp;author=P+S+Lee&amp;author=Y+Wang&amp;author=M+G+Dominguez&amp;author=Y+G+Yeung&amp;author=M+A+Murphy&amp;volume=18&amp;publication_year=1999&amp;pages=3616-3628&amp;pmid=10393178&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B51">51. <span class="citation">Joazeiro C A, Wing S S, Huang H, Leverson J D, Hunter T, Liu Y C. <span><span class="ref-journal">Science. </span>1999;<span class="ref-vol">286</span>:309–312.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10514377" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;author=C+A+Joazeiro&amp;author=S+S+Wing&amp;author=H+Huang&amp;author=J+D+Leverson&amp;author=T+Hunter&amp;volume=286&amp;publication_year=1999&amp;pages=309-312&amp;pmid=10514377&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B52">52. <span class="citation">Withers-Ward E S, Jowett J B, Stewart S A, Xie Y M, Garfinkel A, Shibagaki Y, Chow S A, Shah N, Hanaoka F, Sawitz D G, et al. <span><span class="ref-journal">J Virol. </span>1997;<span class="ref-vol">71</span>:9732–9742.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC230283/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9371639" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Virol&amp;author=E+S+Withers-Ward&amp;author=J+B+Jowett&amp;author=S+A+Stewart&amp;author=Y+M+Xie&amp;author=A+Garfinkel&amp;volume=71&amp;publication_year=1997&amp;pages=9732-9742&amp;pmid=9371639&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B53">53. <span class="citation">Vadlamudi R K, Joung I, Strominger J L, Shin J. <span><span class="ref-journal">J Biol Chem. </span>1996;<span class="ref-vol">271</span>:20235–20237.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8702753" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;author=R+K+Vadlamudi&amp;author=I+Joung&amp;author=J+L+Strominger&amp;author=J+Shin&amp;volume=271&amp;publication_year=1996&amp;pages=20235-20237&amp;pmid=8702753&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B54">54. <span class="citation">Salamov A A, Solovyev V V. <span><span class="ref-journal">J Mol Biol. </span>1995;<span class="ref-vol">247</span>:11–15.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7897654" ref="reftype=pubmed&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;author=A+A+Salamov&amp;author=V+V+Solovyev&amp;volume=247&amp;publication_year=1995&amp;pages=11-15&amp;pmid=7897654&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=18343&amp;issue-id=520&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Proceedings of the National Academy of Sciences of the United States of America</span> are provided here courtesy of <strong>National Academy of Sciences</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC18343/pdf/pq004979.pdf" class="int-view">PDF (253K)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/18343/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/18343/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="18343" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC18343%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="hC8oVmdNibLOc91fvNwWBsIrAcOpbs7wBOkfXSrL0tNDVNdxlhUkm5SwUI7CnLte">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC18343%2F&amp;text=Conserved%20plant%20genes%20with%20similarity%20to%20mammalian%20de%20novo%20DNA%20methyltransferases" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC18343%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC18343/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC18343/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/10781108/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC18343/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/10781108/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC18343/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/18343/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC18343/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
